%% JWS example using Elsevier elsart classes, adapted by Tim 
%% Finin, finin@cs.umbc.edu

%% comment out one of these to get preprint vs. journal format

%\documentclass{elsart}       %% one column for review, preprint
\documentclass{elsart3p}    %% two columns for publication

\usepackage{graphicx,amssymb}

\usepackage{moreverb}
\usepackage{url}

\newcommand{\protege}{Prot\'{e}g\'{e}}

\renewcommand\floatpagefraction{.2}
\makeatletter
\def\elsartstyle{%
   \def\normalsize{\@setfontsize\normalsize\@xiipt{14.5}}
   \def\small{\@setfontsize\small\@xipt{13.6}}
   \let\footnotesize=\small
   \def\large{\@setfontsize\large\@xivpt{18}}
   \def\Large{\@setfontsize\Large\@xviipt{22}}
   \skip\@mpfootins = 18\p@ \@plus 2\p@
   \normalsize
}
\@ifundefined{square}{}{\let\Box\square}
\makeatother

\def\sourcetabsize{4}
\newenvironment{sourcestyle}{\begin{scriptsize}}{\end{scriptsize}}
\def\sourceinput#1{\par\begin{sourcestyle}\verbatimtabinput[\sourcetabsize]{#1}\end{sourcestyle}\par}



\begin{document}

\begin{frontmatter}

\title{\textbf{MIREOT: the Minimum Information to Reference an External Ontology Term}}

%% Here are the  AUTHORS.  Note the separating commas
%% with no and.

%\author[tfl]{M\'elanie Courtot\corauthref{C}},
%\ead{mcourtot@gmail.com}
%\author[ucsd]{William Bug}, 
%\author[ebi]{James Malone},
%\author[ebi,IMBI]{Daniel Schober},
%\author[tfl]{Ryan R. Brinkman},
%\author[sc]{Alan Ruttenberg\corauthref{C}}
%\ead{alanruttenberg@gmail.com}


\author[institute1]{author1\corauthref{C}},
\ead{email1@mail.com}
\author[institute2]{author2}, 
\author[institute3]{author3}, 
\author[institute4]{author4}, 
\author[institute5]{author5}, 
\author[institute6]{author6\corauthref{C}}
\ead{email2@mail.com}


%% these specify any 'notes' for the title and authors,
%% including 'thanks' that acknowledge support, etc. and an
%% indication of who the corresponding author is if there's
%% more than one author.

\corauth[C]{Corresponding authors.}

\medskip

%% Here are the addresses refereenced for the authors.  If
%% all of the authors are from the same institution you need
%% not have the lables for the author and address commands.


\address[institute1]{xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx}
\address[institute2]{xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx}
\address[institute3]{xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx}
\address[institute4]{xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx}
\address[institute5]{xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx}
\address[institute6]{xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx}
%\address[tfl]{Terry Fox Laboratory, British Columbia Cancer Research Center, Vancouver, BC, Canada}
%\address[ucsd]{National Center for Microscopy Imaging Research, UCSD, CA, USA}
%\address[new]{School of Computing Science, Newcastle University, Newcastle upon Tyne, UK}
%\address[cisban]{CISBAN and School of Computing Science, Newcastle University, Newcastle upon Tyne, UK}
%\address[ebi]{The European Bioinformatics Institute, Cambridge, CB101SD, UK}
%\address[sc]{Science Commons, Cambridge, MA, USA}


\end{frontmatter}



\section*{Abstract}
\emph{
While the Web Ontology Language (OWL) provides a mechanism to import ontologies, this mechanism is not always suitable.
%FG: Abstracts tend not to have references, moved to intro
First, given the current state of editing tools and the issues they have working with large ontologies, direct OWL imports are not practical for day-to-day development.
Second, the other ontologies used may be under active development and not aligned with the chosen development methodology. Importing heterogenous ontologies in their entirety may lead to inconsistencies or unintended inferences.
In this paper we propose a set of guidelines for importing required terms from an external artefact into a target ontology.
We describe the guidelines, their implementation, present some examples of application, and outline future work and extensions.}

%DS: Maybe more straight to the point: "First, due to the inability to load only parts of e.g. large ontologies, direct OWL imports are not practical for day-to-day development.

%DS:Reword abstract a bit,should be differrent from wording in intro.


\section*{Introduction}
\label{intro}
The Web Ontology Language (OWL \cite{RefWorks:1506}) provides a built-in mechanism to import ontologies (\emph{owl:imports}).
The \emph{owl:imports} mechanism has proved insufficient for the needs of the authors during development of the Ontology for Biomedical Investigations (OBI \ref{*****}), a large integrative ontology for the description of life-science and clinical investigations.
%AL: The ``in addition'' does not make sense here - this is the first actual reason that it is insufficient
Limitations in the currently available tools and reasoners can make the implementation of the \emph{owl:imports} mechanism problematic.
For example, most current OWL tools can neither load nor reason over very large ontologies, such as the NCBI Taxonomy \cite{RefWorks:1502} or the Foundational Model of Anatomy \cite{RefWorks:1558}, making direct OWL imports of such ontologies impractical.


%DS: Is the statement that NCIT can't be reasoned over with \protege\ correct? o.k. its large but what do they use to reason over it? They do reasoning that is for sure. maybe also state, e.g. in the conclusion section, that this problem is likely to occure in artefacts that are not 'properly modularized'. E.g. if there would be clearer guidelines on how to delineate a domain, such artefacts would be modularized into smaller 'importable' artefacts? Just a thought.
%AL: Perhaps they reason without \protege\ - just directly use the Pellet or Fact++ API? In which case, we should talk about that too. Is there a citation describing the problems reasoning with large ontologies? That's what we need - this previous sentence is an unfounded statement at the moment.

Ontologies are typically dynamic knowledge resources, and must be not only maintained but constantly updated.
%AL: The above sentence does not mean much. What is a dynamic knowledge resource?
This is evident within the bio-ontology domain, where the codification of knowledge must keep pace with scientific knowledge discovery.
The flux of knowledge within an ontology can also present issues when trying to import or integrate different resources.
The individual resources may have been constructed using different desing principles, which may not align.
These issues of dynamic development, and different design principles could lead to inconsistencies or unintended inferences, when integrated.

%AL: Some sentences were not on their own line. have changed that.

To address these issues, we have developed a set of guidelines for importing classes from multiple ontology resources.

%DS: "a more involved"? 

Our solution is a new standard we call the Minimum Information to Represent an External Ontology Term (MIREOT).
MIREOT provides guidelines on importing selected terms without the overhead of importing the complete ontology from which the terms derive. 
%DS: I am glad its not called Miroot (O=other instead of E=external)    ;-)

MIREOT was created to aid the development of the Ontology for Biomedical Investigations (OBI  \cite{RefWorks:1507}).
OBI uses the Basic Formal Ontology (BFO \cite{RefWorks:1557}) as upper-level ontology and is part of the OBO Foundry \cite{RefWorks:1472}. 
One of the fundamental principles of the OBO Foundry is to reuse, where sensible, existing ontology resources, therefore avoiding duplication of effort and ensuring orthogonality.

%DS: So far the Foundry has made little effort to work out an ontology module delineation mechanism. There will always be borders with overlapping content. There had been steps in the right direction at the NCBO though.

MIREOT describes a methodology for the import of external terms from ontologies not yet using BFO, using other OWL flavours or even differrent syntax formats such as OBO.

\section*{Policy}

In deciding upon a minimum unit of import, our first step was to consider the practices of other ontologies.
The practice of the Gene Ontology (GO \cite{RefWorks:79}) is that the intended meanings of classes remain stable.
The repair or reorganization of an ontology should not change the intended meaning of individual terms, but rather provide a better expression of the logical relations between them.
When a term's definition changes meaning, the term is deprecated rather than creating a situation where a term's semantics changes \cite{RefWorks:1560}.
We can therefore consider a term as stable, in isolation from the rest of the ontology, and use terms (i.e. classes) a basic unit of import. 

%AL: It seems from the way the sentences are worded that we're still talking about GO above. If so, we need to now have a sentence that says how it can be valid that we apply these statements to other ontologies that may have different deprecation policies! Is this the same for all OBO ontologies? If so, what about non-OBO ontologies?

The minimum amount of information needed to \textit{reference} an external class is the ontology URI and the term's URI.
Generally, these items remain stable and can be used to unambiguously reference the external class from within the importing target ontology.
The minimum amount of information to \textit{integrate} this class is its position in the hierarchy, specifically the URI of the parent class from the target ontology.

% AL: Shouldn't this parent class also be a URI?

Taken together, the following minimal set is enough to consistently reference an external term:
\begin{description}
 \item[source ontology URI] The logical URI for the ontology containing the term(s) to be imported. %DS: This is the core of the paper, don't just mention it in brackets, add def and example? %AL: This is the logical and not the physical URI, correct?
 \item[source term URI] The logical URI for the specific term to import. %DS: def and example?
 \item[Target ontology superclass URI] The logical URI for the new parent class in the target ontology for the source term to be imported. %DS: maby make it independent from OBI: own ontologies superclass
 \end{description} 

%AL: Agree with DS. Put it in a description of some sort. Have done this as an example. Also, we're using class and term interchangeably here - stick with one!

To ease development of the target ontology we also recommend, although do not require, additional metadata about the imported classes such as their label and definition.

%AL: What extra metadata, specifically? That information should go here.

%DS: Maybe say somewhere what the differrences between imorts and mappings are (Look at Chris Mungalls OBO mapping page) ? Also maybe discuss at the end that an equal approch can be used to import property hiererchies, if necessary (they are usually much smaller).

%AL: Yes, what DS says here is really important. After all, the reviewers will definitely be wondering why we don't just make use of Prompt, the \protege\ plugin, to do some class-level mergings and be done with it? Perhaps should go in the discussion. Will add a comment there too.


\section*{Implementation}

A practical implementation of the MIREOT standard has been performed in the context of the OBI project, and involved a two-step process:

\begin{enumerate}
\item gather the minimum information for external terms, and
\item use this minimum information to fetch additional elements, like labels and definitions.
\end{enumerate}

%DS: mention somewhere that we rely on scripts. If we are very honest we would have to admit that an ontology group using MIREOT needs a talented programmer for curation. Also, not the same, but related approach exists supported by the \protege\ UMLS tab plugin, which allows to import terms from UMLS into an own artefact (see http://\protege\wiki.stanford.edu/index.php/UMLS_Tab).

Once the classes to import have been identified by the curators, the first step is to gather their corresponding minimum information set.
This set is stored in a file that we call \emph{external.owl}, either by creating it, or by appending the new information to the pre-existing one if some terms were already imported.
This file can be edited by hand, but to ease the process of importing external terms, we provide a perl script that can be used in command line to append to the file. This step could be made more user friendly in the future by implementing a \protege\ plugin.

This perl script, add-to-external.pl, takes as arguments the identifiers of the term to be imported and its parent in the OBI hierarchy.

%DS: Making this a \protege\ plugin equal to the UMLS tab would definitely increase user compliance.
%AL: Agreed - intergrate its functionality within Prompt, or your own tab...
%AL: if we talk about scripts, we should point out where they can be found on the sf svn server.

A mapping mechanism between the prefix used in the identifier and the source ontology URI is built in the script.

%DS: Stick to one name, e.g. "Source ontology URI" vs "External ontology URI"(which you have above)...

Curators therefore need only specify the ID of the term to import and its superclass' ID in the target ontology.
For example:


\begin{sourcestyle}%
\begin{verbatimtab}[\sourcetabsize]
    perl add-to-external.pl CHEBI:23367 IAO:0000018
\end{verbatimtab}%
\end{sourcestyle}
will add the class molecular entities (CHEBI:23367) as subclass of the class material entity (IAO:0000018), and set the source ontology URI as \url{http://purl.org/obo/owl/CHEBI}.

Once the minimum information has been collected  and stored in the \emph{external.owl} file, we can use it in order to get additional elements programatically.
%DS: Add examples for such additional info


In order to do so, we created a set of SPARQL \cite{RefWorks:1531} CONSTRUCT queries that we use as template. 
\begin{figure}[t]
\begin{sourcestyle}%
\verbatiminput{./figs/sparql.txt} 
\end{sourcestyle}
\caption{Template SPARQL query. For convenience, we use alias:preferredTerm and
alias:definition to reference our annotations properties IAO\_0000111 and IAO\_0000115 respectively}
\label{fig:sparql}
\end{figure}
These queries specify which extra information about the class to gather, such as the definition and preferred label, and how to map these into the corresponding OBI annotation properties. 
%DS: This can only make sense if its ensured that the meaning (formal axiomatic definition in the target ontology) has not changed. One just has to be careful here I guess.

For example, in the current OWL rendering of OBO files, definitions are individuals and the rdfs:label of that individual records the text of the definition.
The query (figure \ref{fig:sparql}) will map the rdfs:label of the oboInOwl:Definition instance into the value of the obi:definition property.
Within the OBI implementation of MIREOT, only annotation properties which map directly to our own metadata are mapped: new properties are not created.
Additional metadata is integrated in accordance with the agreed minimal metadata policy ( Add URL as long minimal metadata is not out), \emph{e.g.}, curation status annotation property, definition editor or definition source.
The term is directly imported from the external resource, with the status and definition as defined by the outside resource. %AL: What? You were just talking about additional metadata and here is a sentence about the term again - probably belongs elsewhere.

A script iterates through the minimum information stored in \emph{external.owl}.
Depending on the source ontology, it then selects the correct SPARQL template and substitutes the relevant ID.
The queries are executed against the Neurocommons SPARQL endpoint\cite{RefWorks:1540}

This supplementary information, which is prone to change as the source ontologies evolve, is stored in a second file, \emph{externalDerived.owl} and generated via a script. 
This file can be removed on a regular basis (e.g. for a slim release), and rebuilt via script based on \emph{external.owl}, allowing us to keep imported information up-to-date.
%AL: You must describe what a slim release is - it's an undefined term here.

The two files, \emph{external.owl} and \emph{externalDerived.owl}, are then imported by \emph{obi.owl}, providing the necessary information to OBI editors while at the same time keeping it isolated from the core OBI ontology.


\subsection*{Use case one - Cell class}

As an example, we replaced the OBI class cell with that from the OBO Foundry Cell Type (CL) ontology (cf figure \ref{fig:cell}).
The Cell Type Ontology \cite{RefWorks:1559} is part of the OBO foundry effort, and we would like to use the cell class as defined by this resource, instead of creating our own redundant class.
%DS: State what native format CO is (OBO?).

\begin{figure}[t]
\centering \includegraphics*[width=1\columnwidth]{./figs/cell}
\caption{The ``cell" class in the Cell Type Ontology. ``cell by class" and ``cell by organism" are examples of classes we would rather not import, because....}
\label{fig:cell}
\end{figure}

Following the MIREOT guidelines, 
we identify the minimum information required in this case:  
%DS: Who is we? Maybe elaborate on the implied precondition, that there is a release team or Mapping team that curates this bit?

\begin{enumerate}
\item the URI of the term cell: \url{http://purl.org/obo/owl/CL#CL_0000000}  
\item the ontology from which the term is imported: \url{http://purl.org/obo/owl/CL} 
\item the position of ``cell" in the OBI hierarchy: as a subclass of material entity
\end{enumerate}
%DS: Follow bullet list as introduced above, use same names for MIREOT entities throughout.

The imported ``cell" class can be used withim axiomatisations as would be any other OBI class: for example,using Manchester owl syntax, the process ``electroporation" is defined as:

\begin{sourcestyle}%
\begin{verbatimtab}[\sourcetabsize]
     is_a cell permeabilzation
     has_specified_input some cell
     has_specified_output some 
	    (cell and has_quality some electroporated))
     utilizes_device some power supply

\end{verbatimtab}%
\end{sourcestyle}


It can also be subclassed, either by other imported classes or by OBI classes.

\subsection*{Use case two - taxonomic information}

It is expected that in most cases we will want to import information as described in the cell use case above, i.e. a simple one-to-one mapping towards an external class. However in some cases we might want or need more than that, and the MIREOT mechanism has been devised to be flexible and allow to store other types of information as deemed necessary.

OBI currently uses the NCBI taxonomy (REF?) for its mammalian species terms. 
%DS: Mentiona what part of it(as we do NOT import all)

Consider the scenario in which we have two experiments, one in human and one in mouse. The files are annotated with the classes human and mouse from OBI, which are in turn individually mapped from the NCBI taxonomy.

We can easily imagine that somebody would want to have a query like ``give me all experiments in mammals". In this case, we would need to know that human and mouse are subclasses (even indirect) of mammals in the NCBI taxonomy.
Therefore, when mapping towards an NCBI term, it is needed to get the class itself and all its superclasses up to the root of the NCBI taxonomy.
In this case, we use a specific SPARQL query, which retrieves all direct superclasses up to one of a set of upper-level classes in the taxonomy.% (cf figure \ref{fig:ncbi}).
As per the mechanism described above, the minimum information about the mapped class (\emph{e.g.}, Mus musculus) is defined in \emph{external.owl}, whereas the additional information (rank information - genus, kingdom, phylum, etc.) is stored in \emph{ externalDerived.owl}. 

%\begin{figure}[t]
%\centering \includegraphics*[width=1\columnwidth]{./figs/ncbi}
%\caption{The ``Mus musculus" class in the OBI hierarchy (shown up to the Mammalia rank)}
%\label{fig:ncbi}
%\end{figure}



\section*{Discussion}

The MIREOT standard is a trade-off between complete consistency checking and heavyweight importing versus lightweight importing but partial consistency checking.

We are aware of and accept that by copying only parts of an ontology there is the risk that inferences drawn may be incomplete or incorrect: correct inference using the external classes is only guaranteed if the full ontologies are imported.
When deciding to import an external term we review the textual definition and, if needed, talk with the original editor.
As we are importing from OBO Foundry ontologies we have a community process for monitoring change, a shared understanding of the basics of our domain, and the intention to eventually share the same upper-level ontology. 
%DS: ... in theory at least  ;-)
Therefore, we expect that terms will be deprecated if there is a significant change in meaning, and are flexible enough to adjust and update our import of terms as the other ontologies start enhancing their logical definitions.

%AL: Reviewers will probably ask why are the slots/relationships not being imported too? This should go in the discussion.
%AL: The reviewers will definitely be wondering why we don't just make use of Prompt, the \protege\ plugin, to do some class-level mergings and be done with it?
%However, as the NCBI taxonomy currently uses only subclass relations, we are guaranteed to obtain the desired module in this case.
%DS: desired module?

An other consideration using this approach is the status of assertions made on external terms.
In adding axioms such as the subclass axiom when importing the external term, the aim is to only assert true statements about the terms.
We can also add extra restrictions to these imported classes: for example in OBI, cell is the bearer of the role reagent role or specimen role. 
These additional restriction should be stored in the importing ontology file: the \emph{external.owl} and \emph{externalDerived.owl} are meant to include only the imported information.
%DS: Maybe elaborate on the justification for where restrictions get stored, and what the consequences would be?

We anticipate that some of these statements may migrate to the source ontologies at some point in the future; a fruit of the collaborative nature of OBO Foundry ontology development. 

\section*{Future work}
Even though we aimed at providing an easy mechanism for our editors to use the MIREOT standard, we still rely on command-line scripts.
Ideally, a Prot\'eg\'e\cite{RefWorks:1501} plugin could be developed to improve the interaction between the curators and the tool.
%DS: Ah here you have it... so ignore previous comment about this, but maybe mention UMLS Plugin anyway.

In the future, we also expect to provide an option in the OBI distribution that replaces \emph{external.owl} with \emph{imports.owl}, a file of imports statements generated by extracting the ontology URIs mentioned in \emph{external.owl}.
%Other options are possible, for instance having the script that generates additional information use software that extracts module of the external ontology.

The MIREOT standard is currently being implemented by other ontologies,
%DS: Which ones beside IAO? Is it really already a standard?
and we ultimately hope that combined feedback will allow us to perfect the mechanism.


\section*{Acknowledgements}

In memory of our friend and colleague William Bug, Ontological Engineer. 

%added Dirk, Oliver and Larisa who joined recently
The OBI consortium is (in alphabetical order): Ryan Brinkman, Bill Bug, Helen Causton, Kevin Clancy, Christian Cocos, M\'elanie Courtot, Dirk Derom, Eric Deutsch, Liju Fan, Dawn Field, Jennifer Fostel, Gilberto Fragoso, Frank Gibson, Tanya Gray, Jason Greenbaum, Pierre Grenon, Jeff Grethe, Yongqun He, Mervi Heiskanen, Tina Hernandez-Boussard, Allyson Lister, James Malone, Elisabetta Manduchi, Luisa Montecchi, Norman Morrison, Chris Mungall, Helen Parkinson, Bjoern Peters, Matthew Pocock, Philippe Rocca-Serra, Daniel Rubin, Alan Ruttenberg, Susanna-Assunta Sansone, Richard Scheuermann, Daniel Schober, Barry Smith, Larisa Soldatova, Holger Stenzhorn, Chris Stoeckert, Chris Taylor, John Westbrook,  Joe White, Trish Whetzel, Stefan Wiemann, Jie Zheng. 
%The author’s work is partially supported by funding from the NIH(R01EB005034), the EC EMERALD project(LSHG-CT-2006-037686)
%, the BBSRC(BB/C008200/1), the EU NoE NuGO(NoE 503630), the CARMEN project EPSRC(EP/E002331/1)
%, and the Michael Smith Foundation for Health Research.

\bibliographystyle{unsrt}   

\bibliography{20090218-RefWorks}


\end{document}