<html><head><title>file:///Users/alanr/repos/obi/branchDevelopment/trunk/obil.owl</title></head><style type='text/css'> ul { list-style-type: none; margin-left: 10px; padding-left:10px;} </style><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Role.owl</h3><h2>Ontology: http://obi.sourceforge.net/ontology/OBI.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/external/protege-dc.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/external/ro.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/external/ro_bfo_bridge.owl<br><b>protegeowl:defaultLanguage</b>: en<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/DataTransformation.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Biomaterial.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Plan.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/InstrumentAndPart.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/bfo/bfo.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/DigitalEntityPlus.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Quality.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Relations.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/external/protege.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/TheRest.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Obsolete.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/ProtocolApplication.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Role.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/AnnotationProperty.owl<br><b>owl:imports</b>: file:/Users/alanr/repos/obi/branchDevelopment/trunk/OBI-Function.owl<br><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Quality.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/DigitalEntityPlus.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Obsolete.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/OBI-Function.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/DataTransformation.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/external/protege.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/bfo/bfo.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/TheRest.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/AnnotationProperty.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Plan.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/external/protege-dc.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/external/ro_bfo_bridge.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/external/ro.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Relations.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/InstrumentAndPart.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/ProtocolApplication.owl</h3><h3>Ontology: file:/Users/alanr/repos/obi/branchDevelopment/trunk/Biomaterial.owl</h3><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/Plan.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 114 $<br><h2>Ontology: http://purl.org/obo/owl/ro_bfo_bridge</h2><h3><i>Ontology Annotations</i></h3><b>obo:hasDate</b>: 14:07:2006 00:00<br><b>obo:savedBy</b>: cjm<br><b>obo:hasDefaultNamespace</b>: relationship<br><b>rdfs:comment</b>: Domain and range constraints for OBO_REL, in terms of BFO<br><h3>Ontology: http://protege.stanford.edu/plugins/owl/dc/protege-dc.owl</h3><h3>Ontology: http://purl.org/dc/elements/1.1/</h3><h2>Ontology: http://purl.org/obo/owl/ro</h2><h3><i>Ontology Annotations</i></h3><b>obo:hasDate</b>: 14:07:2006 00:00<br><b>obo:savedBy</b>: cjm<br><b>obo:hasDefaultNamespace</b>: relationship<br><b>rdfs:comment</b>: <p>This ontology contains logical relations to be shared across the different OBO ontologies. Each OBO ontology is free to extend this set with relations specific to the biology within that ontology</p><div class="notes"><p>In the definitions, variables c, d, ..., p, q, ... range over instances of continuants and processes respectively. C, D, ..., P, Q, ... range over the corresponding classes</p><p> For a discussion of instance-level relations and also of additional axioms needed to infer transitivity and other properties of the relations listed above see <a href="http://genomebiology.com/2005/6/5/R46">http://genomebiology.com/2005/6/5/R46</a> or the OBO relations page hosted at <a href="http://obo.sourceforge.net/relationship">http://obo.sourceforge.net/relationship</a></div><br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/Relations.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 80 $<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/DigitalEntityPlus.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 107 $<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/Quality.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 81 $<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/InstrumentAndPart.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 114 $<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/Role.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 114 $<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/TheRest.owl</h2><h3><i>Ontology Annotations</i></h3><b>dc:creator</b>: Alan Ruttenberg<br><b>dc:subject</b>: An ontology for the annotation of biomedical and functional genomics experiments.<br><b>dc:creator</b>: Daniel Rubin<br><b>dc:format</b>: OWL-DL<br><b>dc:creator</b>: Norman Morrison<br><b>dc:creator</b>: Richard Bruskiewich<br><b>dc:creator</b>: Daniel Schober<br><b>dc:creator</b>: Susanna-Assunta Sansone<br><b>dc:creator</b>: Stefan Wiemann<br><b>dc:date</b>: April 27, 2007<br><b>dc:creator</b>: Helen C. Causton<br><b>dc:creator</b>: Chris Taylor<br><b>dc:contributor</b>: Advisors for this project come from the IFOMIS group, Saarbruecken and from the Co-ODE group in Manchester<br><b>dc:creator</b>: Ryan R. Brinkman<br><b>dc:rights</b>: free, no license required.<br><b>dc:creator</b>: Dawn Field<br><b>dc:creator</b>: Richard Scheuermann<br><b>dc:creator</b>: Bjoern Peters<br><b>dc:creator</b>: Andrew Wood<br><b>dc:creator</b>: Liju Fan<br><b>dc:creator</b>: Luisa Montecchi<br><b>dc:creator</b>: Allyson Lister<br><b>dc:creator</b>: Jennifer Fostel<br><b>dc:creator</b>: Frank Gibson<br><b>dc:creator</b>: Bill Bug<br><b>owl:versionInfo</b>: 0.6.7<br><b>dc:creator</b>: Mervi Heiskanen<br><b>dc:creator</b>: Jeffrey Grethe<br><b>protegeowl:defaultLanguage</b>: en<br><b>dc:creator</b>: Helen Parkinson<br><b>dc:creator</b>: Joe White<br><b>dc:creator</b>: Kevin Clancy<br><b>dc:title</b>: Ontology for Biomedical Investigation<br><b>dc:creator</b>: Robert Stevens<br><b>rdfs:comment</b>: draft version<br><b>dc:creator</b>: Trish Whetzel<br><b>dc:creator</b>: Philippe Rocca-Serra<br><b>dc:creator</b>: Chris Stoeckert<br><b>dc:creator</b>: Gilberto Fragoso<br><b>dc:creator</b>: Tanya Gray<br><b>dc:creator</b>: Eric Deutsch<br><b>dc:description</b>: The Ontology for Biomedical Investigations (OBI) is build in a collaborative, international effort and will serve as a resource for annotating biomedical investigations, including the study design, protocols and instrumentation used, the data generated and the types of analysis performed on the data. This ontology arose from the Functional Genomics Investigation Ontology (FuGO) and will contain both terms that are common to all biomedical investigations, including functional genomics investigations and those that are more domain specific.<br><b>dc:creator</b>: Barry Smith<br><b>dc:creator</b>: Tina Hernandez-Boussard<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/AnnotationProperty.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 94 $<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/Biomaterial.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 114 $<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/DataTransformation.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 104 $<br><h3>Ontology: http://protege.stanford.edu/plugins/owl/protege</h3><h2>Ontology: http://www.ifomis.org/bfo/1.0</h2><h3><i>Ontology Annotations</i></h3><b>dc:source</b>: Pierre Grenon: "Nuts in BFO's Nutshell: Revisions to the Bi-categorial Axiomatization of BFO"<br><b>dc:publisher</b>: Institute for Formal Ontology and Medical Information Science (IFOMIS)<br><b>dc:source</b>: Pierre Grenon: "BFO in a Nutshell: A Bi-categorial Axiomatization of BFO and Comparison with DOLCE"<br><b>dc:source</b>: Barry Smith and Pierre Grenon: "The Cornucopia of Formal Ontological Relations"<br><b>dc:identifier</b>: http://www.ifomis.org/bfo/1.0<br><b>dc:creator</b>: Holger Stenzhorn<br><b>dc:title</b>: Basic Formal Ontology (BFO)<br><b>owl:imports</b>: http://protege.stanford.edu/plugins/owl/dc/protege-dc.owl<br><b>dc:source</b>: Pierre Grenon and Barry Smith: "SNAP and SPAN: Towards Geospatial Dynamics"<br><b>dc:format</b>: application/rdf+xml<br><b>dc:language</b>: en<br><b>owl:versionInfo</b>: 1.0<br><b>dc:source</b>: Barry Smith: "Basic Tools of Formal Ontology"<br><b>dc:date</b>: #<JAVA-OBJECT com.hp.hpl.jena.datatypes.xsd.XSDDateTime {A7179A}><br><b>dc:source</b>: Barry Smith: "Against Fantology"<br><b>dc:source</b>: Pierre Grenon: "Spatio-temporality in Basic Formal Ontology: SNAP and SPAN, Upper-Level Ontology, and Framework for Formalization"<br><b>dc:source</b>: Pierre Grenon, Barry Smith and Louis Goldberg: "Biodynamic Ontology: Applying BFO in the Biomedical Domain"<br><b>dc:contributor</b>: Andrew Spear<br><b>dc:source</b>: Barry Smith: "Beyond Concepts: Ontology as Reality Representation"<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/ProtocolApplication.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 114 $<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/Obsolete.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 81 $<br><h2>Ontology: http://obi.sourceforge.net/ontology/OBI/OBI-Function.owl</h2><h3><i>Ontology Annotations</i></h3><b>owl:versionInfo</b>: $Revision: 100 $<br><b>dc:creator</b>: Bill Bug<br><hr><h3><i>Class Tree</i></h3><ul><a href='#item1'>Thing</a><ul style='border-left: thin solid #999999'>
<li><a href='#item2'>DbXref</a></li><li><a href='#item3'>sex_effect</a></li><li><a href='#item4'>Subset</a></li><li><a href='#item5'>DIRECTED-BINARY-RELATION</a></li><li><a href='#item6'>Definition</a></li><li><a href='#item7'>SynonymType</a></li><li><a href='#item8'>PAL-CONSTRAINT</a></li><a href='#item9'>to_be_fixed</a><ul style='border-left: thin solid #999999'>
<a href='#item10'>state</a><ul style='border-left: thin solid #999999'>
<li><a href='#item11'>ex-vivo_state</a></li><li><a href='#item12'>in-vitro_state</a></li><li><a href='#item13'>in-vivo_state</a></li></ul>
</ul>
<a href='#item14'>entity</a><ul style='border-left: thin solid #999999'>
<a href='#item15'>continuant</a><ul style='border-left: thin solid #999999'>
<a href='#item16'>specifically_dependent_continuant</a><ul style='border-left: thin solid #999999'>
<a href='#item17'>quality</a><ul style='border-left: thin solid #999999'>
<li><a href='#item18'>radioactive</a></li><a href='#item19'>PATO_quality</a><ul style='border-left: thin solid #999999'>
<li><a href='#item20'>frozen_state</a></li><li><a href='#item21'>solid_state</a></li><li><a href='#item22'>liquid_state</a></li><li><a href='#item23'>gaseous_state</a></li></ul>
</ul>
<a href='#item24'>realizable_entity</a><ul style='border-left: thin solid #999999'>
<a href='#item25'>disposition</a><ul style='border-left: thin solid #999999'>
<li><a href='#item26'>disease</a></li></ul>
<a href='#item27'>function</a><ul style='border-left: thin solid #999999'>
<li><a href='#item28'>GO_molecular_function</a></li><a href='#item29'>obi_function</a><ul style='border-left: thin solid #999999'>
<li><a href='#item30'>reagent_function</a></li><a href='#item31'>labware_function</a><ul style='border-left: thin solid #999999'>
<li><a href='#item32'>dialysis_tubing_function</a></li></ul>
<a href='#item33'>instrument_function</a><ul style='border-left: thin solid #999999'>
<li><a href='#item34'>analytical_instrument_function</a></li></ul>
<li><a href='#item35'>instrument_part_function</a></li><li><a href='#item36'>instrument_platform_function</a></li></ul>
<li><a href='#item37'>Instrument_function</a></li></ul>
<li><a href='#item38'>intervention_study</a></li><a href='#item39'>role</a><ul style='border-left: thin solid #999999'>
<a href='#item40'>role_in_protocol_application</a><ul style='border-left: thin solid #999999'>
<a href='#item41'>specimen_role</a><ul style='border-left: thin solid #999999'>
<li><a href='#item42'>chromium_release_assay_effector_role</a></li></ul>
<li><a href='#item43'>replicate_role</a></li><a href='#item44'>control_role</a><ul style='border-left: thin solid #999999'>
<a href='#item45'>negative_control_role</a><ul style='border-left: thin solid #999999'>
<li><a href='#item46'>placebo_role</a></li></ul>
<li><a href='#item47'>reference_role</a></li><li><a href='#item48'>positive_control_role</a></li></ul>
<a href='#item49'>subject_role</a><ul style='border-left: thin solid #999999'>
<li><a href='#item50'>patient_role</a></li></ul>
<a href='#item51'>reagent_role</a><ul style='border-left: thin solid #999999'>
<li><a href='#item52'>detector_role</a></li><li><a href='#item53'>chromium_release_assay_target_role</a></li><a href='#item54'>reporter_role</a><ul style='border-left: thin solid #999999'>
<li><a href='#item55'>label_role</a></li></ul>
</ul>
<a href='#item56'>output_role</a><ul style='border-left: thin solid #999999'>
<li><a href='#item57'>dependent_variable</a></li></ul>
<a href='#item58'>input_role</a><ul style='border-left: thin solid #999999'>
<li><a href='#item59'>independent_variable</a></li></ul>
</ul>
<a href='#item60'>investigation_administration_role</a><ul style='border-left: thin solid #999999'>
<li><a href='#item61'>investigator</a></li><li><a href='#item62'>funder</a></li></ul>
<li><a href='#item63'>treated_role</a></li><li><a href='#item64'>author_role</a></li></ul>
<a href='#item65'>concretized_information_entity</a><ul style='border-left: thin solid #999999'>
<li><a href='#item66'>concretized_protocol</a></li></ul>
</ul>
</ul>
<a href='#item67'>independent_continuant</a><ul style='border-left: thin solid #999999'>
<li><a href='#item68'>object_boundary</a></li><a href='#item69'>object_aggregate</a><ul style='border-left: thin solid #999999'>
<a href='#item70'>sample_population</a><ul style='border-left: thin solid #999999'>
<li><a href='#item71'>pool_of_specimens</a></li></ul>
<a href='#item72'>platform</a><ul style='border-left: thin solid #999999'>
<li><a href='#item73'>liquid_chromatography_mass_spectrometry_platform</a></li><li><a href='#item74'>microarray_platform</a></li></ul>
<li><a href='#item75'>population</a></li><li><a href='#item76'>site_sample</a></li></ul>
<li><a href='#item77'>fiat_object_part</a></li><a href='#item78'>object</a><ul style='border-left: thin solid #999999'>
<a href='#item79'>artefact_object</a><ul style='border-left: thin solid #999999'>
<a href='#item80'>cardinal_part_of_instrument</a><ul style='border-left: thin solid #999999'>
<a href='#item81'>part_of_electronic_device</a><ul style='border-left: thin solid #999999'>
<a href='#item82'>voltage_amplfier</a><ul style='border-left: thin solid #999999'>
<li><a href='#item83'>preamplifier</a></li><li><a href='#item84'>logarithmic_voltage_amplifier</a></li></ul>
<a href='#item85'>photodetector</a><ul style='border-left: thin solid #999999'>
<li><a href='#item86'>photomultiplier_tube</a></li><a href='#item87'>photodiode</a><ul style='border-left: thin solid #999999'>
<li><a href='#item88'>avalanche_photdiode</a></li></ul>
</ul>
<li><a href='#item89'>analog_to_digital_converter</a></li></ul>
<a href='#item90'>cardinal_part_of_liquid_chromatography_instrument</a><ul style='border-left: thin solid #999999'>
<li><a href='#item91'>?column_chromatography_solvent</a></li><li><a href='#item92'>chromatography_column</a></li><li><a href='#item93'>column_chromatography_matrix</a></li><li><a href='#item94'>pump_valve_switch</a></li></ul>
<a href='#item95'>cardinal_part_of_fluidic_system</a><ul style='border-left: thin solid #999999'>
<li><a href='#item96'>differential_pressure_gauge</a></li><li><a href='#item97'>sheath_tank</a></li><li><a href='#item98'>charge_plate</a></li><li><a href='#item99'>?interrogation_point</a></li><li><a href='#item100'>fluid_pressure_regulator</a></li><li><a href='#item101'>piezo_electric_crystal</a></li><li><a href='#item102'>plate_loader</a></li><li><a href='#item103'>?sheath_fluid</a></li><li><a href='#item104'>?sheath_propellent</a></li><a href='#item105'>particle_delivery_vessel</a><ul style='border-left: thin solid #999999'>
<li><a href='#item106'>multi-well_plate</a></li><li><a href='#item107'>sample_tube</a></li></ul>
<a href='#item108'>flow_cell</a><ul style='border-left: thin solid #999999'>
<li><a href='#item109'>?jet_in_air_flow_chamber</a></li><li><a href='#item110'>quartz_cuvette_flow_chamber</a></li></ul>
<li><a href='#item111'>waste_tank</a></li><li><a href='#item112'>syringe_pump</a></li><li><a href='#item113'>?collection_tube</a></li><li><a href='#item114'>?sample_fluid</a></li></ul>
</ul>
<a href='#item115'>_collected_relations</a><ul style='border-left: thin solid #999999'>
<li><a href='#item116'>?instrument_setting</a></li><li><a href='#item117'>?instrument_configuration</a></li></ul>
<a href='#item118'>device</a><ul style='border-left: thin solid #999999'>
<a href='#item119'>labware</a><ul style='border-left: thin solid #999999'>
<li><a href='#item120'>paper_filter</a></li><li><a href='#item121'>filtration_instrument</a></li><li><a href='#item122'>polystyrene_tubes</a></li><li><a href='#item123'>supernatant_collection_system_harvesting_frames</a></li><li><a href='#item124'>microtitre_plate</a></li></ul>
<a href='#item125'>instrument</a><ul style='border-left: thin solid #999999'>
<li><a href='#item126'>computer</a></li><li><a href='#item127'>heating_block</a></li><li><a href='#item128'>sonicator</a></li><li><a href='#item129'>vacuum_dryer</a></li><li><a href='#item130'>waterbath</a></li><a href='#item131'>optical_subsystem</a><ul style='border-left: thin solid #999999'>
<a href='#item132'>light_source</a><ul style='border-left: thin solid #999999'>
<a href='#item133'>laser</a><ul style='border-left: thin solid #999999'>
<a href='#item134'>gas_laser</a><ul style='border-left: thin solid #999999'>
<a href='#item135'>metal_vapor_laser</a><ul style='border-left: thin solid #999999'>
<li><a href='#item136'>helium_cadmium_ion_laser</a></li></ul>
<a href='#item137'>ion_laser</a><ul style='border-left: thin solid #999999'>
<li><a href='#item138'>argon_ion_laser</a></li><li><a href='#item139'>mixed_argon-krypton_gas_laser</a></li><li><a href='#item140'>krypton_ion_laser</a></li><li><a href='#item141'>helium_neon_ion_laser</a></li></ul>
</ul>
<a href='#item142'>solid_state_laser</a><ul style='border-left: thin solid #999999'>
<li><a href='#item143'>neodymium-YAG_laser</a></li></ul>
<li><a href='#item144'>diode_laser</a></li><li><a href='#item145'>dye_laser</a></li></ul>
<li><a href='#item146'>arc_lamp</a></li></ul>
<a href='#item147'>optical_filter</a><ul style='border-left: thin solid #999999'>
<li><a href='#item148'>long_pass_filter</a></li><li><a href='#item149'>dichroic_mirror</a></li><li><a href='#item150'>band_pass_filter</a></li><li><a href='#item151'>dichroic_filter</a></li><li><a href='#item152'>short_pass_filter</a></li></ul>
<li><a href='#item153'>obscuration_bar</a></li></ul>
<li><a href='#item154'>microarray_wash_station</a></li><li><a href='#item155'>oligonucleotide_synthesizer</a></li><li><a href='#item156'>homogenizer</a></li><li><a href='#item157'>array_scanner</a></li><li><a href='#item158'>gamma_counter</a></li><li><a href='#item159'>incubator</a></li><a href='#item160'>?flow_cytometer</a><ul style='border-left: thin solid #999999'>
<a href='#item161'>flow_cytometer_analyzer</a><ul style='border-left: thin solid #999999'>
<li><a href='#item162'>MACS_Quant</a></li><li><a href='#item163'>Guava_EasyCyte_Plus</a></li><li><a href='#item164'>Bactiflow</a></li><li><a href='#item165'>A10-Analyzer</a></li><li><a href='#item166'>LactoScope</a></li><li><a href='#item167'>FACS_Scan</a></li><li><a href='#item168'>Guava_PCA</a></li><li><a href='#item169'>CyFlow_SL3</a></li><li><a href='#item170'>CYFlow_ML</a></li><li><a href='#item171'>FACS_Caliber</a></li><li><a href='#item172'>CyFlow_SL</a></li><li><a href='#item173'>ImageStream</a></li><li><a href='#item174'>Cell_Lab_Quanta_SC</a></li><li><a href='#item175'>Luminex_200</a></li><li><a href='#item176'>Luminex_100</a></li><li><a href='#item177'>Cyan</a></li><li><a href='#item178'>FC_500</a></li><li><a href='#item179'>FACS_Canto2</a></li><li><a href='#item180'>A40-MiniFCM</a></li><li><a href='#item181'>Somacount</a></li><li><a href='#item182'>inFlux_Analyzer</a></li><li><a href='#item183'>CytoSub</a></li><li><a href='#item184'>CytoBuoy</a></li><li><a href='#item185'>SomaScope</a></li><li><a href='#item186'>CytoSence</a></li><li><a href='#item187'>Guava_PCA-96</a></li><li><a href='#item188'>Guava_EasyCyte_Mini</a></li><li><a href='#item189'>FACS_Canto</a></li><a href='#item190'>flow_cytometer_sorter</a><ul style='border-left: thin solid #999999'>
<li><a href='#item191'>BioSorter2000</a></li><li><a href='#item192'>CyFlow_Space</a></li><li><a href='#item193'>MoFlo</a></li><li><a href='#item194'>FACSvantage</a></li><li><a href='#item195'>Reflection</a></li><li><a href='#item196'>BioSorter500</a></li><li><a href='#item197'>BioSorter1000</a></li><li><a href='#item198'>BioSorter250</a></li><li><a href='#item199'>FACSAria</a></li><li><a href='#item200'>Influx_Cell_Sorter</a></li><li><a href='#item201'>LSR2</a></li></ul>
</ul>
</ul>
<li><a href='#item202'>manufacturer_of_flow_cytometer</a></li><li><a href='#item203'>mass_spectrometry_instrument</a></li><li><a href='#item204'>thermal_cycler</a></li><li><a href='#item205'>spectrophotometer</a></li><li><a href='#item206'>vortexer</a></li><li><a href='#item207'>DNA_sequencer</a></li><li><a href='#item208'>hybridization_chamber</a></li><li><a href='#item209'>?liquid_chromatography_instrument</a></li><li><a href='#item210'>crosslinking_instrument</a></li><li><a href='#item211'>?liquid_handler</a></li><li><a href='#item212'>hybridization_station</a></li><li><a href='#item213'>arrayer</a></li><li><a href='#item214'>centrifuge</a></li></ul>
</ul>
<li><a href='#item215'>physical_document</a></li></ul>
<a href='#item216'>biomaterial_entity</a><ul style='border-left: thin solid #999999'>
<li><a href='#item217'>substance</a></li><a href='#item218'>toBeDiscussed</a><ul style='border-left: thin solid #999999'>
<li><a href='#item219'>individual</a></li></ul>
<li><a href='#item220'>immortal cell</a></li><li><a href='#item221'>organism_status</a></li><li><a href='#item222'>biosource_provider</a></li><a href='#item223'>molecular_mixture</a><ul style='border-left: thin solid #999999'>
<a href='#item224'>liquid chemical solution</a><ul style='border-left: thin solid #999999'>
<li><a href='#item225'>PBS</a></li></ul>
<li><a href='#item226'>suspension</a></li><li><a href='#item227'>emulsion</a></li><li><a href='#item228'>solution</a></li></ul>
<a href='#item229'>NaturalBiomaterialEntity</a><ul style='border-left: thin solid #999999'>
<a href='#item230'>organism</a><ul style='border-left: thin solid #999999'>
<li><a href='#item231'>eukaryote</a></li><li><a href='#item232'>virus</a></li><li><a href='#item233'>viroid</a></li><li><a href='#item234'>bacterium</a></li><li><a href='#item235'>virus</a></li><li><a href='#item236'>archae_bacterium</a></li></ul>
<a href='#item237'>population</a><ul style='border-left: thin solid #999999'>
<li><a href='#item238'>cohort</a></li></ul>
<li><a href='#item239'>serotype</a></li><li><a href='#item240'>strain</a></li><li><a href='#item241'>unicellular organism</a></li><li><a href='#item242'>ecotype</a></li><a href='#item243'>cell</a><ul style='border-left: thin solid #999999'>
<li><a href='#item244'>Epstein_Barr_virus_transformed_b_cell</a></li><li><a href='#item245'>cytotoxic_t-cell</a></li></ul>
<li><a href='#item246'>organism</a></li><li><a href='#item247'>biological_macromolecule</a></li><a href='#item248'>organism_part</a><ul style='border-left: thin solid #999999'>
<li><a href='#item249'>serum</a></li><li><a href='#item250'>plasma</a></li></ul>
<li><a href='#item251'>in_vivo_biomaterial</a></li></ul>
<a href='#item252'>entity_of_organismal_origin</a><ul style='border-left: thin solid #999999'>
<li><a href='#item253'>cell_component</a></li><li><a href='#item254'>multicellular_entity_of_organismal_origin</a></li><li><a href='#item255'>cell_of_organismal_origin</a></li><a href='#item256'>cell_line</a><ul style='border-left: thin solid #999999'>
<li><a href='#item257'>immortalized_cell_line</a></li></ul>
</ul>
<li><a href='#item258'>fragment derived from protein</a></li><li><a href='#item259'>xenograft</a></li><a href='#item260'>ChEBI_objects</a><ul style='border-left: thin solid #999999'>
<li><a href='#item261'>chromium_51</a></li><li><a href='#item262'>Triton X-100</a></li></ul>
<li><a href='#item263'>artificially engineered biomaterial</a></li><li><a href='#item264'>cultivar</a></li><li><a href='#item265'>chimera</a></li><li><a href='#item266'>cell derived from multicellular organism</a></li><a href='#item267'>ExperimentalBiomateralEntity</a><ul style='border-left: thin solid #999999'>
<li><a href='#item268'>organ_section</a></li><li><a href='#item269'>display_library</a></li><li><a href='#item270'>cdna_library</a></li><li><a href='#item271'>cell_pellet</a></li><li><a href='#item272'>cell culture</a></li><li><a href='#item273'>cell_culture_supernatant</a></li><li><a href='#item274'>lysate</a></li><li><a href='#item275'>library</a></li><li><a href='#item276'>whole_mount</a></li><li><a href='#item277'>transgenic_organism</a></li><li><a href='#item278'>Bronchial_Aveolar_Lavage [BAL]</a></li><li><a href='#item279'>synthetic polypeptide</a></li><li><a href='#item280'>peptide_construct</a></li></ul>
<li><a href='#item281'>differentiation_state</a></li><a href='#item282'>geneticInformationContent</a><ul style='border-left: thin solid #999999'>
<li><a href='#item283'>diplotype</a></li><li><a href='#item284'>genotype</a></li><li><a href='#item285'>haplotype</a></li><li><a href='#item286'>allele</a></li></ul>
<li><a href='#item287'>genetic_modification</a></li><li><a href='#item288'>supernatant</a></li></ul>
</ul>
<li><a href='#item289'>site</a></li></ul>
<a href='#item290'>generically_dependent_continuant</a><ul style='border-left: thin solid #999999'>
<a href='#item291'>non-realizable_information_entity</a><ul style='border-left: thin solid #999999'>
<a href='#item292'>information_content_entity</a><ul style='border-left: thin solid #999999'>
<a href='#item293'>data_display_element</a><ul style='border-left: thin solid #999999'>
<li><a href='#item294'>tabular_data_report</a></li><li><a href='#item295'>figure</a></li></ul>
<a href='#item296'>cardinal_part_of_value</a><ul style='border-left: thin solid #999999'>
<a href='#item297'>cardinal_part_of_qualitative_value</a><ul style='border-left: thin solid #999999'>
<li><a href='#item298'>intensity</a></li></ul>
<a href='#item299'>cardinal_part_of_quantitative_value</a><ul style='border-left: thin solid #999999'>
<li><a href='#item300'>unit</a></li><li><a href='#item301'>magnitude</a></li></ul>
</ul>
<a href='#item302'>narrative_object</a><ul style='border-left: thin solid #999999'>
<li><a href='#item303'>conclusion</a></li><li><a href='#item304'>objective</a></li><li><a href='#item305'>diagnosis</a></li><li><a href='#item306'>patent</a></li><li><a href='#item307'>hypothesis</a></li><li><a href='#item308'>journal_article</a></li><li><a href='#item309'>clinical_finding</a></li><a href='#item310'>report</a><ul style='border-left: thin solid #999999'>
<li><a href='#item311'>investigation_record</a></li></ul>
</ul>
<a href='#item312'>data_object</a><ul style='border-left: thin solid #999999'>
<li><a href='#item313'>image</a></li><a href='#item314'>value</a><ul style='border-left: thin solid #999999'>
<a href='#item315'>quantitative_value</a><ul style='border-left: thin solid #999999'>
<a href='#item316'>quantitative_confidence_value</a><ul style='border-left: thin solid #999999'>
<li><a href='#item317'>standard_error</a></li><li><a href='#item318'>P_value</a></li></ul>
</ul>
<li><a href='#item319'>qualitative_value</a></li></ul>
<li><a href='#item320'>data_set</a></li></ul>
</ul>
</ul>
<a href='#item321'>information_entity</a><ul style='border-left: thin solid #999999'>
<li><a href='#item322'>algorithm</a></li><li><a href='#item323'>data_format_specification</a></li><a href='#item324'>study_design</a><ul style='border-left: thin solid #999999'>
<a href='#item325'>balanced_incomplete_block_design</a><ul style='border-left: thin solid #999999'>
<li><a href='#item326'>reference_design</a></li><li><a href='#item327'>loop_design</a></li></ul>
<li><a href='#item328'>time_series_design</a></li><li><a href='#item329'>intervention_study</a></li><li><a href='#item330'>parallel group_design (syn:Independent measures design)</a></li><li><a href='#item331'>self_vs_self_design</a></li><a href='#item332'>sequential_design</a><ul style='border-left: thin solid #999999'>
<a href='#item333'>cross_over_design</a><ul style='border-left: thin solid #999999'>
<li><a href='#item334'>n-to-1_design</a></li></ul>
</ul>
<a href='#item335'>factorial_design</a><ul style='border-left: thin solid #999999'>
<li><a href='#item336'>2x2_factorial_design</a></li><li><a href='#item337'>fractional_factorial_design</a></li></ul>
<li><a href='#item338'>dye_swap_design</a></li><a href='#item339'>randomized_complete_block_design</a><ul style='border-left: thin solid #999999'>
<li><a href='#item340'>graeco_latin_square_design</a></li><li><a href='#item341'>latin_square_design</a></li><li><a href='#item342'>hyper_graeco_latin_square_design</a></li></ul>
<li><a href='#item343'>clinical_study_design</a></li><li><a href='#item344'>matched_pairs_design</a></li><li><a href='#item345'>replicate_design</a></li></ul>
<li><a href='#item346'>investigation_design</a></li><li><a href='#item347'>protocol</a></li><a href='#item348'>instruction_trigger</a><ul style='border-left: thin solid #999999'>
<a href='#item349'>rule</a><ul style='border-left: thin solid #999999'>
<li><a href='#item350'>stopping_rule</a></li><a href='#item351'>compliance_rule</a><ul style='border-left: thin solid #999999'>
<a href='#item352'>standard_compliance_rule</a><ul style='border-left: thin solid #999999'>
<li><a href='#item353'>ethical_standard_compliance_rule</a></li></ul>
</ul>
<a href='#item354'>eligibility_rule</a><ul style='border-left: thin solid #999999'>
<li><a href='#item355'>inclusion_rule</a></li><li><a href='#item356'>exclusion_rule</a></li></ul>
</ul>
<li><a href='#item357'>time_trigger</a></li><li><a href='#item358'>adverse_event_trigger</a></li></ul>
</ul>
<a href='#item359'>digital_entity</a><ul style='border-left: thin solid #999999'>
<li><a href='#item360'>binary_executable</a></li><li><a href='#item361'>digital_document</a></li><li><a href='#item362'>source_code_module</a></li></ul>
</ul>
<a href='#item363'>spatial_region</a><ul style='border-left: thin solid #999999'>
<li><a href='#item364'>two_dimensional_region</a></li><li><a href='#item365'>three_dimensional_region</a></li><li><a href='#item366'>zero_dimensional_region</a></li><li><a href='#item367'>one_dimensional_region</a></li></ul>
</ul>
<a href='#item368'>occurrent</a><ul style='border-left: thin solid #999999'>
<a href='#item369'>spatiotemporal_region</a><ul style='border-left: thin solid #999999'>
<a href='#item370'>connected_spatiotemporal_region</a><ul style='border-left: thin solid #999999'>
<li><a href='#item371'>spatiotemporal_interval</a></li><li><a href='#item372'>spatiotemporal_instant</a></li></ul>
<li><a href='#item373'>scattered_spatiotemporal_region</a></li></ul>
<a href='#item374'>processual_entity</a><ul style='border-left: thin solid #999999'>
<li><a href='#item375'>process_aggregate</a></li><li><a href='#item376'>processual_context</a></li><a href='#item377'>process</a><ul style='border-left: thin solid #999999'>
<a href='#item378'>protocol_application</a><ul style='border-left: thin solid #999999'>
<a href='#item379'>material_transformation</a><ul style='border-left: thin solid #999999'>
<li><a href='#item380'>Killing</a></li><a href='#item381'>Material Separation</a><ul style='border-left: thin solid #999999'>
<a href='#item382'>Separating Based On Characteristic</a><ul style='border-left: thin solid #999999'>
<a href='#item383'>Separating Based On Quality</a><ul style='border-left: thin solid #999999'>
<li><a href='#item384'>separating_based_on_size</a></li></ul>
<li><a href='#item385'>Resection</a></li></ul>
<li><a href='#item386'>Material Portioning</a></li><li><a href='#item387'>Taking Sample From Organism</a></li></ul>
<a href='#item388'>Material Exposure</a><ul style='border-left: thin solid #999999'>
<li><a href='#item389'>Acclimatization</a></li></ul>
<a href='#item390'>material_combination</a><ul style='border-left: thin solid #999999'>
<li><a href='#item391'>administering_substance_in_vivo</a></li><a href='#item392'>adding_reagent_to_substance</a><ul style='border-left: thin solid #999999'>
<a href='#item393'>administering_substance_to_cell culture</a><ul style='border-left: thin solid #999999'>
<li><a href='#item394'>cell_chromium_labeling</a></li></ul>
</ul>
<li><a href='#item395'>cell_co-culture</a></li><li><a href='#item396'>material_pooling</a></li></ul>
</ul>
<a href='#item397'>PA_to_be_reviewed</a><ul style='border-left: thin solid #999999'>
<li><a href='#item398'>DNA_extraction</a></li><li><a href='#item399'>imaging_assay</a></li><li><a href='#item400'>clinical_chemistry_assay</a></li><li><a href='#item401'>mass_spectrometry_assay</a></li><li><a href='#item402'>?sample-population_formation_process</a></li><li><a href='#item403'>nucleic_acid_hybridization_assay</a></li><li><a href='#item404'>DNA_sequencing_assay</a></li><li><a href='#item405'>observation_assay</a></li><li><a href='#item406'>nuclear_magnetic_resonance_assay</a></li><li><a href='#item407'>sample_preparation_for_assay</a></li></ul>
<a href='#item408'>data_transformation</a><ul style='border-left: thin solid #999999'>
<a href='#item409'>mathematical_function</a><ul style='border-left: thin solid #999999'>
<li><a href='#item410'>logarithm</a></li><li><a href='#item411'>sine</a></li><li><a href='#item412'>exponential_function</a></li><li><a href='#item413'>cosine</a></li></ul>
<a href='#item414'>unclassified</a><ul style='border-left: thin solid #999999'>
<li><a href='#item415'>newman_global_modularity_calculation</a></li><li><a href='#item416'>intra-subgraph_connectivity_calculation</a></li><li><a href='#item417'>edge_weighting</a></li></ul>
<a href='#item418'>filtering</a><ul style='border-left: thin solid #999999'>
<li><a href='#item419'>structural_filtering</a></li><li><a href='#item420'>low_intensity_filter</a></li><li><a href='#item421'>edge_filtering</a></li></ul>
<a href='#item422'>statistical_error</a><ul style='border-left: thin solid #999999'>
<li><a href='#item423'>type_II_error</a></li><li><a href='#item424'>type_I_error</a></li></ul>
<a href='#item425'>probabilistic_algorithm</a><ul style='border-left: thin solid #999999'>
<li><a href='#item426'>expectation_maximization</a></li></ul>
<a href='#item427'>partitioning_method</a><ul style='border-left: thin solid #999999'>
<a href='#item428'>classification_method</a><ul style='border-left: thin solid #999999'>
<li><a href='#item429'>mhc_binding_prediction</a></li><li><a href='#item430'>b_cell_epitope_prediction</a></li><li><a href='#item431'>t_cell_epitope_prediction</a></li></ul>
<a href='#item432'>clustering_method</a><ul style='border-left: thin solid #999999'>
<a href='#item433'>hierarchical_clustering</a><ul style='border-left: thin solid #999999'>
<li><a href='#item434'>average_linkage</a></li><li><a href='#item435'>simple_linkage</a></li><li><a href='#item436'>complete_linkage</a></li></ul>
<li><a href='#item437'>k-means_clustering</a></li></ul>
<li><a href='#item438'>modular_decomposition</a></li></ul>
<li><a href='#item439'>data_normalization</a></li><a href='#item440'>averaging_method</a><ul style='border-left: thin solid #999999'>
<li><a href='#item441'>dye_swap_merge</a></li><li><a href='#item442'>moving_average</a></li><li><a href='#item443'>replicate_analysis</a></li></ul>
<a href='#item444'>statistical_significance_method</a><ul style='border-left: thin solid #999999'>
<li><a href='#item445'>differentially_expressed</a></li><li><a href='#item446'>p-value</a></li></ul>
<li><a href='#item447'>network-independent_data_superimposition</a></li><a href='#item448'>data_reduction_method</a><ul style='border-left: thin solid #999999'>
<li><a href='#item449'>principal_component_analysis</a></li></ul>
<a href='#item450'>mathematical_transformation</a><ul style='border-left: thin solid #999999'>
<li><a href='#item451'>split-scale_transformation</a></li><li><a href='#item452'>logistic-log_transformation</a></li><li><a href='#item453'>linear_combination</a></li><li><a href='#item454'>log-log_transformation</a></li><li><a href='#item455'>box-cox_transformation</a></li><li><a href='#item456'>quadratic_transformation</a></li><li><a href='#item457'>hyperlog_transformation</a></li><li><a href='#item458'>linear_transformation</a></li><li><a href='#item459'>logit-log_transformation</a></li><li><a href='#item460'>biexponential_transformation</a></li><li><a href='#item461'>logicle_transformation</a></li><li><a href='#item462'>nonlinear_transformation</a></li><li><a href='#item463'>scaling</a></li></ul>
<a href='#item464'>normalization_method</a><ul style='border-left: thin solid #999999'>
<li><a href='#item465'>loess_group_normalization</a></li><li><a href='#item466'>mean_log_normalization</a></li><li><a href='#item467'>quantile_normalization</a></li><li><a href='#item468'>linlog_transformation</a></li><li><a href='#item469'>total_intensity_normalization_single</a></li><li><a href='#item470'>arsinh_transformation</a></li><li><a href='#item471'>loess_scaled_group_normalization</a></li><li><a href='#item472'>median_log_normalization</a></li><li><a href='#item473'>loess_global_normalization</a></li><li><a href='#item474'>total_intensity_normalization_paired</a></li></ul>
<a href='#item475'>data_imputation_method</a><ul style='border-left: thin solid #999999'>
<li><a href='#item476'>centroid_mass_spectrun</a></li><li><a href='#item477'>characteristic_path_length_calculation</a></li><li><a href='#item478'>continuum_mass_spectrum</a></li></ul>
</ul>
<a href='#item479'>assay</a><ul style='border-left: thin solid #999999'>
<li><a href='#item480'>radioactivity_detection</a></li><li><a href='#item481'>clinical_diagnosis</a></li><li><a href='#item482'>chromium_release_assay</a></li><li><a href='#item483'>tumor_grading</a></li></ul>
</ul>
<a href='#item484'>enrollment</a><ul style='border-left: thin solid #999999'>
<li><a href='#item485'>Enrolling Human Subjects</a></li></ul>
<a href='#item486'>investigation</a><ul style='border-left: thin solid #999999'>
<li><a href='#item487'>hypothesis_generating_investigation</a></li><li><a href='#item488'>hypothesis_driven_investigation</a></li></ul>
<a href='#item489'>Acquisition</a><ul style='border-left: thin solid #999999'>
<li><a href='#item490'>Data Collection</a></li><li><a href='#item491'>Environmental Material Collection</a></li><li><a href='#item492'>Object Acquisition</a></li></ul>
<li><a href='#item493'>Group Assignment</a></li><li><a href='#item494'>disease_course</a></li><li><a href='#item495'>role_assignment</a></li></ul>
<li><a href='#item496'>process_boundary</a></li><a href='#item497'>fiat_process_part</a><ul style='border-left: thin solid #999999'>
<li><a href='#item498'>parameter_value_application</a></li><li><a href='#item499'>disease_stage</a></li><li><a href='#item500'>unplanned_process_part</a></li></ul>
</ul>
<a href='#item501'>temporal_region</a><ul style='border-left: thin solid #999999'>
<li><a href='#item502'>scattered_temporal_region</a></li><a href='#item503'>connected_temporal_region</a><ul style='border-left: thin solid #999999'>
<li><a href='#item504'>temporal_instant</a></li><li><a href='#item505'>temporal_interval</a></li></ul>
</ul>
</ul>
</ul>
<li><a href='#item506'>ObsoleteClass</a></li><li><a href='#item507'>Synonym</a></li></ul>
</ul><hr><h3><i>Property Tree</i></h3><ul><li><i>AnnotationProperty</i></li><ul style='border-left: thin solid #999999'><li><a href='#item507'>hasRelatedSynonym</a></li>
<li><a href='#item508'>source</a></li>
<li><a href='#item509'>type</a></li>
<li><a href='#item510'>label</a></li>
<li><a href='#item511'>hasSynonymType</a></li>
<li><a href='#item512'>title</a></li>
<li><a href='#item513'>maxExclusive</a></li>
<li><a href='#item514'>defaultLanguage</a></li>
<li><a href='#item515'>publisher</a></li>
<li><a href='#item516'>hasDefaultNamespace</a></li>
<li><a href='#item517'>isDefinedBy</a></li>
<li><a href='#item518'>minLength</a></li>
<li><a href='#item519'>minInclusive</a></li>
<li><a href='#item520'>subclassesDisjoint</a></li>
<li><a href='#item521'>format</a></li>
<li><a href='#item522'>versionInfo</a></li>
<li><a href='#item523'>hasURI</a></li>
<li><a href='#item524'>totalDigits</a></li>
<li><a href='#item525'>allowedParent</a></li>
<li><a href='#item526'>maxInclusive</a></li>
<li><a href='#item527'>seeAlso</a></li>
<li><a href='#item528'>identifier</a></li>
<li><a href='#item529'>readOnly</a></li>
<li><a href='#item530'>hasAlternativeId</a></li>
<li><a href='#item531'>alternative_term</a></li>
<li><a href='#item532'>definition_citation</a></li>
<li><a href='#item533'>preferred_term</a></li>
<li><a href='#item534'>minExclusive</a></li>
<li><a href='#item535'>hasNarrowSynonym</a></li>
<li><a href='#item536'>usedLanguage</a></li>
<li><a href='#item537'>language</a></li>
<li><a href='#item538'>branch</a></li>
<li><a href='#item539'>subject</a></li>
<li><a href='#item540'>abstract</a></li>
<li><a href='#item541'>probeClass</a></li>
<li><a href='#item542'>fractionDigits</a></li>
<li><a href='#item543'>todoPrefix</a></li>
<li><a href='#item544'>hasDate</a></li>
<li><a href='#item545'>comment</a></li>
<li><a href='#item546'>date</a></li>
<li><a href='#item547'>maxLength</a></li>
<li><a href='#item548'>coverage</a></li>
<li><a href='#item549'>relation</a></li>
<li><a href='#item550'>hasDbXref</a></li>
<li><a href='#item551'>replacedBy</a></li>
<li><a href='#item552'>savedBy</a></li>
<li><a href='#item553'>pattern</a></li>
<li><a href='#item554'>example_of_usage</a></li>
<li><a href='#item555'>excludedTest</a></li>
<li><a href='#item556'>inSubset</a></li>
<li><a href='#item557'>length</a></li>
<li><a href='#item558'>definition</a></li>
<li><a href='#item559'>editor_note</a></li>
<li><a href='#item560'>alternative_term_citation</a></li>
<li><a href='#item561'>todoProperty</a></li>
<li><a href='#item562'>hasOBONamespace</a></li>
<li><a href='#item563'>base</a></li>
<li><a href='#item564'>hasDefinition</a></li>
<li><a href='#item565'>description</a></li>
<li><a href='#item566'>contributor</a></li>
<li><a href='#item567'>curation_status</a></li>
<li><a href='#item568'>consider</a></li>
<li><a href='#item569'>creator</a></li>
<li><a href='#item570'>hasExactSynonym</a></li>
<li><a href='#item571'>hasVersion</a></li>
<li><a href='#item572'>hasBroadSynonym</a></li>
<li><a href='#item573'>alternative_term_tag</a></li>
<li><a href='#item574'>rights</a></li>
<li><a href='#item575'>hasSubset</a></li>
<li><a href='#item576'>definition_editor</a></li>
</ul><li><i>DatatypeProperty</i></li><ul style='border-left: thin solid #999999'><li><a href='#item577'>PAL-NAME</a></li>
<li><a href='#item578'>PAL-STATEMENT</a></li>
<li><a href='#item579'>PAL-DESCRIPTION</a></li>
<li><a href='#item580'>PAL-RANGE</a></li>
</ul><li><i>ObjectProperty</i></li><ul style='border-left: thin solid #999999'><li><a href='#item581'>FROM</a></li>
<li><a href='#item582'>is_realized_as</a></li>
<li><a href='#item583'>relationship</a></li><ul style='border-left: thin solid #999999'><li><a href='#item584'>location_of</a></li>
<li><a href='#item585'>participates_in</a></li><ul style='border-left: thin solid #999999'><li><a href='#item586'>is_input_of</a></li>
<li><a href='#item587'>is_reagent_in</a></li>
<li><a href='#item588'>is_output_of</a></li>
<li><a href='#item589'>agent_in</a></li>
<li><a href='#item590'>is_instrument_for</a></li>
</ul>
<li><a href='#item591'>derives_from</a></li>
<li><a href='#item592'>part_of</a></li><ul style='border-left: thin solid #999999'><li><a href='#item593'>proper_part_of</a></li>
<li><a href='#item594'>integral_part_of</a></li>
</ul>
<li><a href='#item595'>has_participant</a></li><ul style='border-left: thin solid #999999'><li><a href='#item596'>has_output</a></li>
<li><a href='#item597'>has_input</a></li>
<li><a href='#item598'>utilizes_reagent</a></li>
<li><a href='#item599'>has_agent</a></li>
<li><a href='#item600'>utilizes_instrument</a></li>
</ul>
<li><a href='#item601'>derived_into</a></li>
<li><a href='#item602'>preceded_by</a></li>
<li><a href='#item603'>inputs_data</a></li>
<li><a href='#item604'>outputs_data</a></li>
<li><a href='#item605'>has_part</a></li><ul style='border-left: thin solid #999999'><li><a href='#item606'>has_proper_part</a></li>
<li><a href='#item607'>has_integral_part</a></li>
</ul>
<li><a href='#item608'>contains</a></li>
<li><a href='#item609'>transformation_of</a></li>
<li><a href='#item610'>located_in</a></li>
<li><a href='#item611'>contained_in</a></li>
<li><a href='#item612'>adjacent_to</a></li>
<li><a href='#item613'>precedes</a></li>
</ul>
<li><a href='#item614'>is_realization_of</a></li>
<li><a href='#item615'>is_concretization_of</a></li>
<li><a href='#item616'>is_proxy_for</a></li>
<li><a href='#item617'>transformed_into</a></li>
<li><a href='#item618'>SLOT-CONSTRAINTS</a></li>
<li><a href='#item619'>is_concretized_as</a></li>
<li><a href='#item620'>ObsoleteProperty</a></li>
<li><a href='#item621'>has_role</a></li>
<li><a href='#item622'>TO</a></li>
<li><a href='#item623'>has_quality</a></li>
</ul></ul><hr><h3><i>Class Descriptions</i></h3><a name='item1'><html><div style="width:800"><b>Class: http://www.w3.org/2002/07/owl#Thing<br></b><i></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item2'><html><div style="width:800"><b>Class: http://www.geneontology.org/formats/oboInOwl#DbXref<br></b><i></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item3'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_33<br></b><b><i>sex_effect</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A variable in an experiment that is designed to look at differences between organisms of different genders, mating types etc.</div></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item4'><html><div style="width:800"><b>Class: http://www.geneontology.org/formats/oboInOwl#Subset<br></b><i></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item5'><html><div style="width:800"><b>Class: http://protege.stanford.edu/plugins/owl/protege#DIRECTED-BINARY-RELATION<br></b><i></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item6'><html><div style="width:800"><b>Class: http://www.geneontology.org/formats/oboInOwl#Definition<br></b><i></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item7'><html><div style="width:800"><b>Class: http://www.geneontology.org/formats/oboInOwl#SynonymType<br></b><i></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item8'><html><div style="width:800"><b>Class: http://protege.stanford.edu/plugins/owl/protege#PAL-CONSTRAINT<br></b><i></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item9'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_334<br></b><b><i>to_be_fixed</i></b><i><br></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item10'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_137<br></b><b><i>state</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">a state is a quality indicating the condition in which an entity exists, e.g. immortalized, frozen, cellular_state (in-vivo, in-vitro), macromolecular state, 'ectopic in-vivo', 'denatured or native for protein'. definition_source: OBI.</div></i><font color="#000080">to_be_fixed</font><br><i>definition</i>: a state is a quality indicating the condition in which an entity exists, e.g. immortalized, frozen, cellular_state (in-vivo, in-vitro), macromolecular state, 'ectopic in-vivo', 'denatured or native for protein'. definition_source: OBI.<br></div></html><hr>
<a name='item11'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_259<br></b><b><i>ex-vivo_state</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An ex-vivo state is a state in which the entity is in an artificial environment outside the living organism. definition_source: http://wordnet.princeton.edu/perl/webwn?s=in%20vitro, Jan 3, 2007.</div></i><font color="#000080">state</font><br><i>editor_note</i>: rem0: (AL) One of in-vitro state/ex-vivo state should be deleted, and made a synonym of the remaining term.<br><i>definition</i>: An ex-vivo state is a state in which the entity is in an artificial environment outside the living organism. definition_source: http://wordnet.princeton.edu/perl/webwn?s=in%20vitro, Jan 3, 2007.<br></div></html><hr>
<a name='item12'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_130<br></b><b><i>in-vitro_state</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An in-vitro state is a state in which the entity is in an artificial environment outside the living organism. definition_source: http://wordnet.princeton.edu/perl/webwn?s=in%20vitro, Jan 3, 2007.</div></i><font color="#000080">state</font><br><i>definition</i>: An in-vitro state is a state in which the entity is in an artificial environment outside the living organism. definition_source: http://wordnet.princeton.edu/perl/webwn?s=in%20vitro, Jan 3, 2007.<br></div></html><hr>
<a name='item13'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_138<br></b><b><i>in-vivo_state</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An in-vivo state is a state in which the entity remains part of the living organism. definition_source: OBI.</div></i><font color="#000080">state</font><br><i>definition</i>: An in-vivo state is a state in which the entity remains part of the living organism. definition_source: OBI.<br></div></html><hr>
<a name='item14'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0#Entity<br></b><b><i>entity</i></b><i><br></i><font color="#000080">(equivalent-class&nbsp;(<i>or</i>&nbsp;continuant&nbsp;occurrent))</font><br></div></html><hr>
<a name='item15'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#Continuant<br></b><b><i>continuant</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: An entity that exists in full at any time in which it exists at all, persists through time while maintaining its identity and has no temporal parts.</div></i><font color="#000080">(equivalent-class<br>&nbsp;(<i>or</i>&nbsp;specifically_dependent_continuant&nbsp;independent_continuant&nbsp;spatial_region))</font><br></div></html><hr>
<a name='item16'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#SpecificallyDependentContinuant<br></b><b><i>specifically_dependent_continuant</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A continuant that inheres in or is borne by other entities.</div></i><font color="#000080">(equivalent-class&nbsp;(<i>or</i>&nbsp;quality&nbsp;realizable_entity))</font><br></div></html><hr>
<a name='item17'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#Quality<br></b><b><i>quality</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A dependent continuant that is exhibited if it inheres in an entity or entities at all (a categorical property).</div></i><font color="#000080">(<i>and</i>&nbsp;specifically_dependent_continuant&nbsp;entity<br>&nbsp;(<i>or</i>&nbsp;spatial_region&nbsp;realizable_entity&nbsp;independent_continuant&nbsp;quality)<br>&nbsp;(<i>disjoint-with</i>&nbsp;realizable_entity&nbsp;occurrent))</font><br></div></html><hr>
<a name='item18'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000011<br></b><b><i>radioactive</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">move to dependent.owl</div></i><font color="#000080">quality</font><br></div></html><hr>
<a name='item19'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_302<br></b><b><i>PATO_quality</i></b><i><br></i><font color="#000080">quality</font><br></div></html><hr>
<a name='item20'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_139<br></b><b><i>frozen_state</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A frozen state is a state in which the entity is kept below its freezing point. definition_source: OBI.</div></i><font color="#000080">PATO_quality</font><br><i>definition</i>: A frozen state is a state in which the entity is kept below its freezing point. definition_source: OBI.<br></div></html><hr>
<a name='item21'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_190<br></b><b><i>solid_state</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A solid state is a state in which the entity has no tendency to flow under moderate stress; resists forces (such as compression) which tend to deform it; and retains a definite size and shape. definition_source: http://wordnet.princeton.edu/perl/webwn?s=solid%20state, Jan 3, 2007.</div></i><font color="#000080">PATO_quality</font><br><i>definition</i>: A solid state is a state in which the entity has no tendency to flow under moderate stress; resists forces (such as compression) which tend to deform it; and retains a definite size and shape. definition_source: http://wordnet.princeton.edu/perl/webwn?s=solid%20state, Jan 3, 2007.<br></div></html><hr>
<a name='item22'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_189<br></b><b><i>liquid_state</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A liquid state is a state in which the entity has no fixed shape but a fixed volume, generally at a temperature between its freezing and boiling points. definition_source: OBI.</div></i><font color="#000080">PATO_quality</font><br><i>definition</i>: A liquid state is a state in which the entity has no fixed shape but a fixed volume, generally at a temperature between its freezing and boiling points. definition_source: OBI.<br></div></html><hr>
<a name='item23'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_191<br></b><b><i>gaseous_state</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A gaseous state is a state in which the entity is distinguished from the solid and liquid states by: relatively low density and viscosity; relatively great expansion and contraction with changes in pressure and temperature; the ability to diffuse readily; and the spontaneous tendency to become distributed uniformly throughout any container. definition_source: http://wordnet.princeton.edu/perl/webwn?s=gaseous%20state, Jan 3, 2007.</div></i><font color="#000080">PATO_quality</font><br><i>definition</i>: A gaseous state is a state in which the entity is distinguished from the solid and liquid states by: relatively low density and viscosity; relatively great expansion and contraction with changes in pressure and temperature; the ability to diffuse readily; and the spontaneous tendency to become distributed uniformly throughout any container. definition_source: http://wordnet.princeton.edu/perl/webwn?s=gaseous%20state, Jan 3, 2007.<br></div></html><hr>
<a name='item24'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#RealizableEntity<br></b><b><i>realizable_entity</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A dependent continuant that inheres in continuants and are not exhibited in full at every time in which it inheres in an entity or group of entities. The exhibition or actualization of a realizable entity is a particular manifestation, functioning or process that occurs under certain circumstances.</div></i><font color="#000080">(<i>and</i>&nbsp;specifically_dependent_continuant&nbsp;(<i>or</i>&nbsp;disposition&nbsp;function&nbsp;role)<br>&nbsp;entity&nbsp;(<i>or</i>&nbsp;spatial_region&nbsp;realizable_entity&nbsp;independent_continuant&nbsp;quality)<br>&nbsp;(<i>disjoint-with</i>&nbsp;quality&nbsp;occurrent))</font><br></div></html><hr>
<a name='item25'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#Disposition<br></b><b><i>disposition</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A realizable entity that essentially causes a specific process or transformation in the object in which it inheres, under specific circumstances and in conjunction with the laws of nature. A general formula for dispositions is: X (object) has the disposition D to (transform, initiate a process) R under conditions C.</div></i><font color="#000080">(<i>and</i>&nbsp;(<i>not</i>&nbsp;role)&nbsp;realizable_entity)</font><br></div></html><hr>
<a name='item26'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_155<br></b><b><i>disease</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Place_holder for disease ontology</div></i><font color="#000080">disposition</font><br><i>editor_note</i>: rem1: BP - placed into quality. needs to be discussed<br><i>editor_note</i>: rem0: BS did not view disease as a process, and indeed it is now a child of dependent continuant.<br><i>definition</i>: Place_holder for disease ontology<br></div></html><hr>
<a name='item27'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#Function<br></b><b><i>function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A realizable entity the manifestation of which is an essentialy end-directed activity of a continuant entity in virtue of that continuant entity being a specific kind of entity in the kind or kinds of contexts that it is made for.</div></i><font color="#000080">(<i>and</i>&nbsp;realizable_entity&nbsp;(<i>disjoint-with</i>&nbsp;role&nbsp;disposition))</font><br></div></html><hr>
<a name='item28'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_282<br></b><b><i>GO_molecular_function</i></b><i><br></i><font color="#000080">function</font><br></div></html><hr>
<a name='item29'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_900000<br></b><b><i>obi_function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A function inhering in an biomedical investigation-related entity where the necessary material continuants, processes, roles, and biophysical qualities have been identitified and there inter-relations in this functional context have been specified.</div></i><font color="#000080">function</font><br><i>definition_editor</i>: Bill Bug<br><i>editor_note</i>: Still debating whether Assay_function, Assessment_function (sub-class of Assay_function), and Algorithm_function have any meaning in a BFO context, where function appears to be intended to inhere only in independent material continuants (BB:2007-07-10).<br><i>definition_citation</i>: #<JAVA-OBJECT com.hp.hpl.jena.datatypes.BaseDatatype$TypedValue {586A09}><br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: E.g. reagent function, instrument function, instrument part function, etc.<br><i>preferred_term</i>: obi_function<br><i>definition</i>: A function inhering in an biomedical investigation-related entity where the necessary material continuants, processes, roles, and biophysical qualities have been identitified and there inter-relations in this functional context have been specified.<br></div></html><hr>
<a name='item30'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_900001<br></b><b><i>reagent_function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A function inhering a molecule, set of molecules, cell, collection of cells, or tissue  enabling it to participate in critical material transformation in the context of a specific investigation plan.</div></i><font color="#000080">obi_function</font><br><i>definition_editor</i>: Bill Bug<br><i>definition_citation</i>: #<JAVA-OBJECT com.hp.hpl.jena.datatypes.BaseDatatype$TypedValue {268810}><br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: E.g. the function of Cy5 coupled-ligands to enable separation of cells in a Fluorescence-Activated Cell Sorter (FACS) by virtue of cells possessing specific receptors with an specific affinity to bind the coupled ligand thereby causing only those cells to fluoresce when illuminated with laser light frequency that specifically excites Cy5, triggering relays and redirecting the solution flow to a separate effluent container.<br><i>preferred_term</i>: reagent_function<br><i>definition</i>: A function inhering a molecule, set of molecules, cell, collection of cells, or tissue  enabling it to participate in critical material transformation in the context of a specific investigation plan.<br></div></html><hr>
<a name='item31'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_900005<br></b><b><i>labware_function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A function inhering in a type of labware enabling it to participate in critical process in the context of a specific investigation plan.</div></i><font color="#000080">obi_function</font><br><i>definition_editor</i>: Bill Bug<br><i>editor_note</i>: How does this relate to an object such as the enzyme-coated plate used in an ELISA protocol application?  The plate itself is a type of labware, but the enzyme is a reagent.(BB:2007-07-10).<br><i>definition_citation</i>: #<JAVA-OBJECT com.hp.hpl.jena.datatypes.BaseDatatype$TypedValue {7A1787}><br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: E.g. the function of semi-permeable dialysis tubing to separate solutes based on size by virtue of restricted diffusion and osmosis; the function of a plastic 25 ml centrifuge tube to act as an inert storage vessel for a material sample.<br><i>preferred_term</i>: labware_function<br><i>definition</i>: A function inhering in a type of labware enabling it to participate in critical process in the context of a specific investigation plan.<br></div></html><hr>
<a name='item32'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_900007<br></b><b><i>dialysis_tubing_function</i></b><i><br></i><font color="#000080">labware_function</font><br><i>definition_citation</i>: #<JAVA-OBJECT com.hp.hpl.jena.datatypes.BaseDatatype$TypedValue {DC08D}><br><i>preferred_term</i>: dialysis_tubing_function<br></div></html><hr>
<a name='item33'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_900002<br></b><b><i>instrument_function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A function inhering an instrument enabling it to participate in critical process in the context of a specific investigation plan.</div></i><font color="#000080">obi_function</font><br><i>definition_editor</i>: Bill Bug<br><i>definition_citation</i>: #<JAVA-OBJECT com.hp.hpl.jena.datatypes.BaseDatatype$TypedValue {FD18DA}><br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: E.g. the function of a analytical centrifuge to sediment very large material objects out of a liquid suspension.<br><i>preferred_term</i>: instrument_function<br><i>definition</i>: A function inhering an instrument enabling it to participate in critical process in the context of a specific investigation plan.<br></div></html><hr>
<a name='item34'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_900006<br></b><b><i>analytical_instrument_function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument function where the process in which the instrument participates results in a quality measurement of the material under examination which is critical in the context of a specific investigation plan.</div></i><font color="#000080">instrument_function</font><br><i>definition_editor</i>: Bill Bug<br><i>definition_citation</i>: #<JAVA-OBJECT com.hp.hpl.jena.datatypes.BaseDatatype$TypedValue {C07DA1}><br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: E.g. the function of a spectrophotometer to provide a measure of the relative absorbance of a liquid sample for a particular wavelength (or set of wavelengths) of light.<br><i>preferred_term</i>: analytical_instrument_function<br><i>definition</i>: An instrument function where the process in which the instrument participates results in a quality measurement of the material under examination which is critical in the context of a specific investigation plan.<br></div></html><hr>
<a name='item35'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_900004<br></b><b><i>instrument_part_function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A function inhering the cardinal part of an instrument necessary for that instrument to participate in critical process in the context of a specific investigation plan.</div></i><font color="#000080">obi_function</font><br><i>definition_editor</i>: Bill Bug<br><i>definition_citation</i>: #<JAVA-OBJECT com.hp.hpl.jena.datatypes.BaseDatatype$TypedValue {2EB3FE}><br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: E.g. the function of the electromagnetic lenses in an electron microscope to direct the incident electron beam trajectory so as to systematically raster across a specimen when constructing the overall image; the function of Nomarksi optics to enhance the constrast of specimens viewed in a light microscope based on their ability to refract visible wavelengths of light.<br><i>preferred_term</i>: instrument_part_function<br><i>definition</i>: A function inhering the cardinal part of an instrument necessary for that instrument to participate in critical process in the context of a specific investigation plan.<br></div></html><hr>
<a name='item36'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_900003<br></b><b><i>instrument_platform_function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A function inhering an instrument_platform enabling it to participate in critical process in the context of a specific investigation plan.</div></i><font color="#000080">obi_function</font><br><i>definition_editor</i>: Bill Bug<br><i>definition_citation</i>: #<JAVA-OBJECT com.hp.hpl.jena.datatypes.BaseDatatype$TypedValue {EE5F5E}><br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: E.g. the function of a high pressure liquid chromatagraphic (HPLC) system to separate molecules based on their solubility properties<br><i>preferred_term</i>: instrument_platform_function<br><i>definition</i>: A function inhering an instrument_platform enabling it to participate in critical process in the context of a specific investigation plan.<br></div></html><hr>
<a name='item37'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_303<br></b><b><i>Instrument_function</i></b><i><br></i><font color="#000080">function</font><br></div></html><hr>
<a name='item38'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#intervention_study<br></b><i></i><font color="#000080">(<i>and</i>&nbsp;study_design<br>&nbsp;(is_realized_as&nbsp;<i>some</i><br>&nbsp;&nbsp;(<i>and</i>&nbsp;process<br>&nbsp;&nbsp;&nbsp;(has_part&nbsp;<i>some</i><br>&nbsp;&nbsp;&nbsp;&nbsp;(has_input&nbsp;<i>some</i>&nbsp;(has_role&nbsp;<i>some</i>&nbsp;treated_role))))))</font><br></div></html><hr>
<a name='item39'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#Role<br></b><b><i>role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A realizable entity the manifestation of which brings about some result or end that is not essential to a continuant in virtue of the kind of thing that it is but that can be served or participated in by that kind of continuant in some kinds of natural, social or institutional contexts.</div></i><font color="#000080">(<i>and</i>&nbsp;realizable_entity&nbsp;(<i>disjoint-with</i>&nbsp;disposition&nbsp;function))</font><br></div></html><hr>
<a name='item40'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_343<br></b><b><i>role_in_protocol_application</i></b><i><br></i><font color="#000080">role</font><br></div></html><hr>
<a name='item41'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_254<br></b><b><i>specimen_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A specimen role is an investigation object role where the object is a piece of a material. definition_source: OBI.</div></i><font color="#000080">role_in_protocol_application</font><br><i>editor_note</i>: organism "has_role_which_can_only_be_played_by_biomaterial_entity" some specimen_role in obi before split (LF 040907).<br><i>editor_note</i>: rem: not necessarily played by biomaterial entities, also include material entity<br><i>definition</i>: A specimen role is an investigation object role where the object is a piece of a material. definition_source: OBI.<br></div></html><hr>
<a name='item42'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_9999997<br></b><b><i>chromium_release_assay_effector_role</i></b><i><br></i><font color="#000080">specimen_role</font><br></div></html><hr>
<a name='item43'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_284<br></b><b><i>replicate_role</i></b><i><br></i><font color="#000080">role_in_protocol_application</font><br></div></html><hr>
<a name='item44'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_171<br></b><b><i>control_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A control role is a role which is assigned to an entity that is used as a standard against which other entities can be compared. definition_source: http://wordnet.princeton.edu/perl/webwn?s=control Jan 5, 2007.</div></i><font color="#000080">role_in_protocol_application</font><br><i>editor_note</i>: domain of an object property "is_control_for" in obi before split (LF 040907).<br><i>definition</i>: A control role is a role which is assigned to an entity that is used as a standard against which other entities can be compared. definition_source: http://wordnet.princeton.edu/perl/webwn?s=control Jan 5, 2007.<br></div></html><hr>
<a name='item45'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_173<br></b><b><i>negative_control_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A negative control role is a control role which is assigned to an entity used as a standard for unresponsive or unaffected entities.  definition_source: OBI.</div></i><font color="#000080">control_role</font><br><i>definition</i>: A negative control role is a control role which is assigned to an entity used as a standard for unresponsive or unaffected entities.  definition_source: OBI.<br></div></html><hr>
<a name='item46'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_174<br></b><b><i>placebo_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A placebo role is a negative control role of an entity with no pharmacological effect with respect to the question being examined in the investigation. definition_source: OBI.</div></i><font color="#000080">negative_control_role</font><br><i>definition</i>: A placebo role is a negative control role of an entity with no pharmacological effect with respect to the question being examined in the investigation. definition_source: OBI.<br></div></html><hr>
<a name='item47'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_340<br></b><b><i>reference_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">a reference role is assigned to an entity in a protocol application which has causes a known level of effect in order to establish a scale of different effects</div></i><font color="#000080">control_role</font><br><i>definition_editor</i>: BP<br><i>definition</i>: a reference role is assigned to an entity in a protocol application which has causes a known level of effect in order to establish a scale of different effects<br></div></html><hr>
<a name='item48'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_172<br></b><b><i>positive_control_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A positive control role is a control role which is assigned to an entity used as a standard for responsive or affected entities.   definition_source: OBI.</div></i><font color="#000080">control_role</font><br><i>definition</i>: A positive control role is a control role which is assigned to an entity used as a standard for responsive or affected entities.   definition_source: OBI.<br></div></html><hr>
<a name='item49'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_285<br></b><b><i>subject_role</i></b><i><br></i><font color="#000080">role_in_protocol_application</font><br></div></html><hr>
<a name='item50'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_144<br></b><b><i>patient_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A patient role is an investigation object role where the object participating in the study is an organism. definition_source: OBI.</div></i><font color="#000080">subject_role</font><br><i>definition</i>: A patient role is an investigation object role where the object participating in the study is an organism. definition_source: OBI.<br></div></html><hr>
<a name='item51'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_328<br></b><b><i>reagent_role</i></b><i><br></i><font color="#000080">role_in_protocol_application</font><br></div></html><hr>
<a name='item52'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_323<br></b><b><i>detector_role</i></b><i><br></i><font color="#000080">reagent_role</font><br></div></html><hr>
<a name='item53'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_9999996<br></b><b><i>chromium_release_assay_target_role</i></b><i><br></i><font color="#000080">reagent_role</font><br></div></html><hr>
<a name='item54'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_289<br></b><b><i>reporter_role</i></b><i><br></i><font color="#000080">reagent_role</font><br></div></html><hr>
<a name='item55'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_9999998<br></b><b><i>label_role</i></b><i><br></i><font color="#000080">reporter_role</font><br></div></html><hr>
<a name='item56'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_23<br></b><b><i>output_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An output role is a role which is played by an entity such as data or material at the end of a process. definition_source: OBI.</div></i><font color="#000080">role_in_protocol_application</font><br><i>editor_note</i>: role of data or material played by an entity at the end of a process<br><i>definition</i>: An output role is a role which is played by an entity such as data or material at the end of a process. definition_source: OBI.<br></div></html><hr>
<a name='item57'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_143<br></b><b><i>dependent_variable</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An dependent variable is an investigation variable role which consists of being affected by other investigation variables and often includes being one of the measured outcomes of an investigation. definition_source: OBI.</div></i><font color="#000080">output_role</font><br><i>definition</i>: An dependent variable is an investigation variable role which consists of being affected by other investigation variables and often includes being one of the measured outcomes of an investigation. definition_source: OBI.<br></div></html><hr>
<a name='item58'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_22<br></b><b><i>input_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An input role is a role which is played by an entity such as data or material at the beginning of a process. definition_source: OBI.</div></i><font color="#000080">role_in_protocol_application</font><br><i>editor_note</i>: role of data or material entity played at the start of a process<br><i>definition</i>: An input role is a role which is played by an entity such as data or material at the beginning of a process. definition_source: OBI.<br></div></html><hr>
<a name='item59'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_142<br></b><b><i>independent_variable</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An independent variable is an investigation variable role which consists of being free of effects from other investigation variables and often includes being under the control of the investigator for the purpose of establishing the effects on other variables. definition_source: OBI.</div></i><font color="#000080">input_role</font><br><i>definition</i>: An independent variable is an investigation variable role which consists of being free of effects from other investigation variables and often includes being under the control of the investigator for the purpose of establishing the effects on other variables. definition_source: OBI.<br></div></html><hr>
<a name='item60'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_176<br></b><b><i>investigation_administration_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A group membership role is an investigation object role where the the object participating in the study is associated to other entities to form a collection. definition_source: OBI.</div></i><font color="#000080">role</font><br><i>definition</i>: A group membership role is an investigation object role where the the object participating in the study is associated to other entities to form a collection. definition_source: OBI.<br></div></html><hr>
<a name='item61'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_288<br></b><b><i>investigator</i></b><i><br></i><font color="#000080">investigation_administration_role</font><br></div></html><hr>
<a name='item62'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_332<br></b><b><i>funder</i></b><i><br></i><font color="#000080">investigation_administration_role</font><br></div></html><hr>
<a name='item63'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#treated_role<br></b><i></i><font color="#000080">role</font><br></div></html><hr>
<a name='item64'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_357<br></b><b><i>author_role</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An author_role is a role attributed to someone creating an information_object</div></i><font color="#000080">role</font><br><i>definition_editor</i>: BP<br><i>definition</i>: An author_role is a role attributed to someone creating an information_object<br></div></html><hr>
<a name='item65'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#concretized_information_entity<br></b><i></i><font color="#000080">realizable_entity</font><br></div></html><hr>
<a name='item66'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#concretized_protocol<br></b><i></i><font color="#000080">concretized_information_entity</font><br></div></html><hr>
<a name='item67'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#IndependentContinuant<br></b><b><i>independent_continuant</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A continuant that is a bearer of qualities and realizable entities, in which other entities inhere and which itself cannot inhere in anything.</div></i><font color="#000080">(<i>and</i>&nbsp;entity&nbsp;continuant<br>&nbsp;(<i>or</i>&nbsp;fiat_object_part&nbsp;object&nbsp;object_aggregate&nbsp;object_boundary&nbsp;site)<br>&nbsp;(<i>disjoint-with</i>&nbsp;spatial_region&nbsp;specifically_dependent_continuant&nbsp;occurrent&nbsp;quality<br>&nbsp;&nbsp;realizable_entity))</font><br></div></html><hr>
<a name='item68'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#ObjectBoundary<br></b><b><i>object_boundary</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Comment: Boundaries are theoretically difficult entities to account for, however the intuitive notion of a physical boundary as a surface of some sort (whether inside or outside of a thing) will generally serve as a good guide for the use of this universal.</div></i><font color="#000080">(<i>and</i>&nbsp;independent_continuant<br>&nbsp;(<i>disjoint-with</i>&nbsp;object_aggregate&nbsp;site&nbsp;fiat_object_part&nbsp;object))</font><br></div></html><hr>
<a name='item69'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#ObjectAggregate<br></b><b><i>object_aggregate</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: An independent continuant that is a mereological sum of separate objects and possesses non-connected boundaries.</div></i><font color="#000080">(<i>and</i>&nbsp;independent_continuant<br>&nbsp;(<i>disjoint-with</i>&nbsp;site&nbsp;fiat_object_part&nbsp;object&nbsp;object_boundary))</font><br></div></html><hr>
<a name='item70'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_264<br></b><b><i>sample_population</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A sample population is an object aggregate that is selected from the population, e.g. the fish in the net that were sampled from the lake, the people that responded to the call for volunteers. definition_source: OBI.</div></i><font color="#000080">object_aggregate</font><br><i>definition</i>: A sample population is an object aggregate that is selected from the population, e.g. the fish in the net that were sampled from the lake, the people that responded to the call for volunteers. definition_source: OBI.<br></div></html><hr>
<a name='item71'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_265<br></b><b><i>pool_of_specimens</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A pool of specimens is a sample population that is a set of specimens which have been gathered from one or more sample_populations, obtained by the physical process of mixing individual specimens, e.g.  mixing the DNA collected from the individual fish. definition_source: OBI.</div></i><font color="#000080">sample_population</font><br><i>editor_note</i>: check with advisors as to how to represent multiple instances of any class? a set of specimens which have been gathered from one or more sample_populations, obtained by the physical process of mixing individual specimens, e.g.  mixing the DNA collected from the individual fish<br><i>definition</i>: A pool of specimens is a sample population that is a set of specimens which have been gathered from one or more sample_populations, obtained by the physical process of mixing individual specimens, e.g.  mixing the DNA collected from the individual fish. definition_source: OBI.<br></div></html><hr>
<a name='item72'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_50<br></b><b><i>platform</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A platform is an object_aggregate that is the set of instruments, software and reagents needed to perform a protocol. definition_source: OBI.</div></i><font color="#000080">object_aggregate</font><br><i>editor_note</i>: rem1: We decided at the Philly workshop that consumables do not include reagents.<br><i>definition</i>: A platform is an object_aggregate that is the set of instruments, software and reagents needed to perform a protocol. definition_source: OBI.<br></div></html><hr>
<a name='item73'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_51<br></b><b><i>liquid_chromatography_mass_spectrometry_platform</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A liquid chromatography mass spectrometry platform is a platform that is the collection of instrument, software and reagents needed to perform a liquid chromatography mass spectrometry protocol. definition_source: OBI.</div></i><font color="#000080">platform</font><br><i>definition</i>: A liquid chromatography mass spectrometry platform is a platform that is the collection of instrument, software and reagents needed to perform a liquid chromatography mass spectrometry protocol. definition_source: OBI.<br></div></html><hr>
<a name='item74'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_52<br></b><b><i>microarray_platform</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A microarray platform is a platform that contains the instruments, software and reagents needed to perform a microarray protocol. definition_source: OBI.</div></i><font color="#000080">platform</font><br><i>definition</i>: A microarray platform is a platform that contains the instruments, software and reagents needed to perform a microarray protocol. definition_source: OBI.<br></div></html><hr>
<a name='item75'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_181<br></b><b><i>population</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A population is an object aggregate in which the entities have common qualities. definition_source: OBI.</div></i><font color="#000080">object_aggregate</font><br><i>editor_note</i>: rem1: collection somehow always involve a selection process<br><i>definition</i>: A population is an object aggregate in which the entities have common qualities. definition_source: OBI.<br></div></html><hr>
<a name='item76'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_256<br></b><b><i>site_sample</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An environmental sample is an object that contains an array of objects that may or may not contain a biomaterial. definition_source: OBI.</div></i><font color="#000080">object_aggregate</font><br><i>editor_note</i>: rem1: this has replaced 'collection of organism' previously used<br><i>example_of_usage</i>: A dirt sample containing different populations of bacteria. A sample of martian water. A sample of gut flora.<br><i>definition</i>: An environmental sample is an object that contains an array of objects that may or may not contain a biomaterial. definition_source: OBI.<br></div></html><hr>
<a name='item77'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#FiatObjectPart<br></b><b><i>fiat_object_part</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: An independent continuant that is part of an object but is not demarcated by any physical discontinuities.</div></i><font color="#000080">(<i>and</i>&nbsp;independent_continuant<br>&nbsp;(<i>disjoint-with</i>&nbsp;object_aggregate&nbsp;site&nbsp;object&nbsp;object_boundary))</font><br></div></html><hr>
<a name='item78'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#Object<br></b><b><i>object</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Examples: an organism, a heart, a chair, a lung, an apple</div></i><font color="#000080">(<i>and</i>&nbsp;independent_continuant<br>&nbsp;(<i>disjoint-with</i>&nbsp;object_aggregate&nbsp;site&nbsp;fiat_object_part&nbsp;object_boundary))</font><br></div></html><hr>
<a name='item79'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_47<br></b><b><i>artefact_object</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An artefact_object is an object that is produced or shaped by human craft, especially a tool, weapon, or ornament of archaeological or historical interest.</div></i><font color="#000080">object</font><br><i>definition_editor</i>: Frank Gibson<br><i>definition_citation</i>: http://www.answers.com/artefact<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: Examples include the siblings of device and other objects such as gel matrices; used in gel electrophoresis, filter paper, parafilm and buffer solutions.<br><i>preferred_term</i>: artefact_object<br><i>definition</i>: An artefact_object is an object that is produced or shaped by human craft, especially a tool, weapon, or ornament of archaeological or historical interest.<br></div></html><hr>
<a name='item80'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_36<br></b><b><i>cardinal_part_of_instrument</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cardinal part of instrument is an object that is a principal part of the instrument. definition_source: OBI.</div></i><font color="#000080">artefact_object</font><br><i>editor_note</i>: Unclarity if this needs to be cardinal_part_of or just part_of, also look into x_subsystems (which also imply part_ofs)<br><i>branch</i>: Instrument<br><i>definition</i>: A cardinal part of instrument is an object that is a principal part of the instrument. definition_source: OBI.<br></div></html><hr>
<a name='item81'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400035<br></b><b><i>part_of_electronic_device</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An electronics subsytem is an instrument that is a secondary or subordinate portion of a flow cytometer, that is concerned with or using devices that operate on principles governing the behavior of electrons The electronic subsystem of a flow cytometer recieves light as an input and delivers matrices of data to the user."</div></i><font color="#000080">cardinal_part_of_instrument</font><br><i>definition_editor</i>: John Quinn<br><i>editor_note</i>: DS: Modeling subsytems and cardinal part of is redundant. Will be modeled via part_of relation anyway...<br><i>definition_citation</i>: "http://www.reference.com/search?q=electronic%20subsystem&db=web, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: electronic subsystem of a cytometer<br><i>preferred_term</i>: part_of_electronic_device<br><i>definition</i>: "An electronics subsytem is an instrument that is a secondary or subordinate portion of a flow cytometer, that is concerned with or using devices that operate on principles governing the behavior of electrons The electronic subsystem of a flow cytometer recieves light as an input and delivers matrices of data to the user."<br></div></html><hr>
<a name='item82'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400101<br></b><b><i>voltage_amplfier</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A voltage amplifier is part of an electroncs subsystem. It is an appliance or circuit that increases the strength of a weak electrical signal without changing the other characteristics of the signal. The voltage amplifier boosts the signal strength output from the PMT.</div></i><font color="#000080">part_of_electronic_device</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wiktionary.org/wiki/amplifier, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Linear amplifier, log amplifier, microwave amplifier"<br><i>preferred_term</i>: voltage_amplfier<br><i>definition</i>: A voltage amplifier is part of an electroncs subsystem. It is an appliance or circuit that increases the strength of a weak electrical signal without changing the other characteristics of the signal. The voltage amplifier boosts the signal strength output from the PMT.<br></div></html><hr>
<a name='item83'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400087<br></b><b><i>preamplifier</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A preamplifier is part of the electronics subsystem. It converts the current output from its associated detecor to a voltage. The preamplifier is the first stage in alalog electronics signal processing.</div></i><font color="#000080">voltage_amplfier</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Built in preamplifier in a Hamamatsu H9656 PMT<br><i>preferred_term</i>: preamplifier<br><i>definition</i>: A preamplifier is part of the electronics subsystem. It converts the current output from its associated detecor to a voltage. The preamplifier is the first stage in alalog electronics signal processing.<br></div></html><hr>
<a name='item84'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400066<br></b><b><i>logarithmic_voltage_amplifier</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A logarithmic voltage amplifier is an analog electronic circuit that puts out a voltage or current proportioal o the voltage or current at its input, with logarithmic proportionality. In an analog system, the logarithmic voltage amplifier is used to present parameters with a high dynamic range on a mre useful scale."</div></i><font color="#000080">voltage_amplfier</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "HLVA-100 logarithmic voltage amplifier developed by FEMTO Messtechnik, GmbH"<br><i>preferred_term</i>: logarithmic_voltage_amplifier<br><i>definition</i>: "A logarithmic voltage amplifier is an analog electronic circuit that puts out a voltage or current proportioal o the voltage or current at its input, with logarithmic proportionality. In an analog system, the logarithmic voltage amplifier is used to present parameters with a high dynamic range on a mre useful scale."<br></div></html><hr>
<a name='item85'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400082<br></b><b><i>photodetector</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Part of the electronic subsytem. A device used to detect and measure the intensity of radiant energy through photoelectric action. In a cytometer, photodetectors measure either the number of photons of laser light scattered on inpact with a cell (for example), or the flourescence emmited by excitation of a lourescent dye."</div></i><font color="#000080">part_of_electronic_device</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://einstein.stanford.edu/content/glossary/glossary.html, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "A photomultiplier tube, a photo diode"<br><i>preferred_term</i>: photodetector<br><i>definition</i>: "Part of the electronic subsytem. A device used to detect and measure the intensity of radiant energy through photoelectric action. In a cytometer, photodetectors measure either the number of photons of laser light scattered on inpact with a cell (for example), or the flourescence emmited by excitation of a lourescent dye."<br></div></html><hr>
<a name='item86'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400084<br></b><b><i>photomultiplier_tube</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A photodetector, normally in the form of a tube, that uses a photocathode to convert photons into photoelectrons which are then amplified. PMTs are typically used to detect SSC and flourescent parameters. Cytometers have a PMT for each color they can collect."</div></i><font color="#000080">photodetector</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://cyto.mednet.ucla.edu/Protocols/flow.htm, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: R9647 by manufactured by Hamamatsu<br><i>preferred_term</i>: photomultiplier_tube<br><i>definition</i>: "A photodetector, normally in the form of a tube, that uses a photocathode to convert photons into photoelectrons which are then amplified. PMTs are typically used to detect SSC and flourescent parameters. Cytometers have a PMT for each color they can collect."<br></div></html><hr>
<a name='item87'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400083<br></b><b><i>photodiode</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A photodiode is a semiconductor photodector used to detect light and generate an electrical current. Typically used in forward scatter (FSC) detection. The photodiode collects the forward light scatter in a cytometer.</div></i><font color="#000080">photodetector</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://cyto.mednet.ucla.edu/Protocols/flow.htm, 2007-05-10"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Avalanche photodiode<br><i>preferred_term</i>: photodiode<br><i>definition</i>: A photodiode is a semiconductor photodector used to detect light and generate an electrical current. Typically used in forward scatter (FSC) detection. The photodiode collects the forward light scatter in a cytometer.<br></div></html><hr>
<a name='item88'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400011<br></b><b><i>avalanche_photdiode</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An avalanche photodiode is typically used to collect photons emmitted by forward scatter because it is far less sensitive, and less likely o be burned out, than a PMT. A photodiode wih high quantom efficiency and a mechanism for producing gains as high as a few thousand."</div></i><font color="#000080">photodiode</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: C30644E -  InGaAs Avalanche Photodiode<br><i>preferred_term</i>: avalanche_photdiode<br><i>definition</i>: "An avalanche photodiode is typically used to collect photons emmitted by forward scatter because it is far less sensitive, and less likely o be burned out, than a PMT. A photodiode wih high quantom efficiency and a mechanism for producing gains as high as a few thousand."<br></div></html><hr>
<a name='item89'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400007<br></b><b><i>analog_to_digital_converter</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An analog to digital converter is the part of the electronic subsytem used to convert an infinite resolution analog signal to a finite resolution digital signal The analog to digital converter transformed the analog output from the photomultiplier tube to a digital signal for collection.</div></i><font color="#000080">part_of_electronic_device</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wiktionary.org/wiki/Analog-to-Digital_Converter, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: API A2D convertor<br><i>preferred_term</i>: analog_to_digital_converter<br><i>definition</i>: An analog to digital converter is the part of the electronic subsytem used to convert an infinite resolution analog signal to a finite resolution digital signal The analog to digital converter transformed the analog output from the photomultiplier tube to a digital signal for collection.<br></div></html><hr>
<a name='item90'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_37<br></b><b><i>cardinal_part_of_liquid_chromatography_instrument</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cardinal part of a liquid chromatography instrument is a principal part of an instrument that is used to selectively separate substances dissolved in a liquid. definition_source: OBI.</div></i><font color="#000080">cardinal_part_of_instrument</font><br><i>definition</i>: A cardinal part of a liquid chromatography instrument is a principal part of an instrument that is used to selectively separate substances dissolved in a liquid. definition_source: OBI.<br></div></html><hr>
<a name='item91'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_40<br></b><b><i>?column_chromatography_solvent</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A column chromatography solvent is a cardinal part of a liquid chromatography instrument that is a liquid that is used to dissolve the components to be separated in the column chromatography matrix. definition_source: OBI.</div></i><font color="#000080">cardinal_part_of_liquid_chromatography_instrument</font><br><i>editor_note</i>: DS: is this solvent part of the instrument itself ?<br><i>editor_note</i>: syn: mobile_phase<br><i>definition</i>: A column chromatography solvent is a cardinal part of a liquid chromatography instrument that is a liquid that is used to dissolve the components to be separated in the column chromatography matrix. definition_source: OBI.<br></div></html><hr>
<a name='item92'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_38<br></b><b><i>chromatography_column</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A chromatography column is a cardinal part of a liquid chromatography instrument that is a glass tube that contains the separation matrix substance. definition_source: OBI.</div></i><font color="#000080">cardinal_part_of_liquid_chromatography_instrument</font><br><i>definition</i>: A chromatography column is a cardinal part of a liquid chromatography instrument that is a glass tube that contains the separation matrix substance. definition_source: OBI.<br></div></html><hr>
<a name='item93'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_39<br></b><b><i>column_chromatography_matrix</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A column chromatography matrix is a cardinal part of a liquid chromatography instrument that is the stationary phase in the chromatography column that is able to selectively bind materials. definition_source: OBI.</div></i><font color="#000080">cardinal_part_of_liquid_chromatography_instrument</font><br><i>editor_note</i>: syn:stationary_phase<br><i>definition</i>: A column chromatography matrix is a cardinal part of a liquid chromatography instrument that is the stationary phase in the chromatography column that is able to selectively bind materials. definition_source: OBI.<br></div></html><hr>
<a name='item94'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_41<br></b><b><i>pump_valve_switch</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A pump valve switch is a cardinal part of a liquid chromatography instrument that controls the flow. definition_source: OBI.</div></i><font color="#000080">cardinal_part_of_liquid_chromatography_instrument</font><br><i>definition</i>: A pump valve switch is a cardinal part of a liquid chromatography instrument that controls the flow. definition_source: OBI.<br></div></html><hr>
<a name='item95'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400046<br></b><b><i>cardinal_part_of_fluidic_system</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A fluidics system is an insturment that deals with the use of a flowing liquid or gas in various devices, esp. controls, to perform functions usually performed by an electric current in electronic devices. In a flow cytometer a fluidics subsystem transports particles in a fluid stream to the light source for interrogation."</div></i><font color="#000080">cardinal_part_of_instrument</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.reference.com/search?q=fluidics%20and%20Intro%20to%20Flow%20Cytometry%20by%20BD&r=d&db=web, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: fluidic subsystem of a cytometer<br><i>preferred_term</i>: cardinal_part_of_fluidic_system<br><i>definition</i>: "A fluidics system is an insturment that deals with the use of a flowing liquid or gas in various devices, esp. controls, to perform functions usually performed by an electric current in electronic devices. In a flow cytometer a fluidics subsystem transports particles in a fluid stream to the light source for interrogation."<br></div></html><hr>
<a name='item96'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400032<br></b><b><i>differential_pressure_gauge</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Part of the fluidics subsystem. The differential pressure gauge monitors the difference between sample and sheath fluid pressures in systems where pressure is used to force the sample fluid to flow in the center of the sheath fluid. A differential pressure gauge can be used by the operator to make sure that the sample fluid is at a greater pressure than the sheath fluid, which maintains a core of sample fluid."</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: LSR2 differential pressure gauge<br><i>preferred_term</i>: differential_pressure_gauge<br><i>definition</i>: "Part of the fluidics subsystem. The differential pressure gauge monitors the difference between sample and sheath fluid pressures in systems where pressure is used to force the sample fluid to flow in the center of the sheath fluid. A differential pressure gauge can be used by the operator to make sure that the sample fluid is at a greater pressure than the sheath fluid, which maintains a core of sample fluid."<br></div></html><hr>
<a name='item97'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400094<br></b><b><i>sheath_tank</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Part of the fluidics system. The sheath tank is the vessel that holds the sheath fluid at a constant pressure, allowing for it to be pushed into the flow chamber at a constant rate. The sheath tank holds the pressurized sheath fluid."</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: LSR2 sheath tank<br><i>preferred_term</i>: sheath_tank<br><i>definition</i>: "Part of the fluidics system. The sheath tank is the vessel that holds the sheath fluid at a constant pressure, allowing for it to be pushed into the flow chamber at a constant rate. The sheath tank holds the pressurized sheath fluid."<br></div></html><hr>
<a name='item98'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400019<br></b><b><i>charge_plate</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Part of the fluidics subsystem. Charge plates are used or sorters. They create an charged electric field when particles deemed to be desired for further analysis are shaken form the piexo electric crystal. The charged particles are drawn toward the charged plate, and the altered drop location causes the particles to fall into a collection tube. Charge plates enable sorting."</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: LSR2 charge plate<br><i>preferred_term</i>: charge_plate<br><i>definition</i>: "Part of the fluidics subsystem. Charge plates are used or sorters. They create an charged electric field when particles deemed to be desired for further analysis are shaken form the piexo electric crystal. The charged particles are drawn toward the charged plate, and the altered drop location causes the particles to fall into a collection tube. Charge plates enable sorting."<br></div></html><hr>
<a name='item99'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400059<br></b><b><i>?interrogation_point</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Point within the fluidic subsystem at which the laser intersects the stream, illuminating the cells where they emit their fluorescence and scatter the light. The laser irratidates the cell at the interogation point."</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>editor_note</i>: DS: Spatial entity?<br><i>definition_citation</i>: "http://www.flocyte.com/FRTP/Resources/flow_cytometry_glossary.htm, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Interrogation point in BD FACS Calibur<br><i>preferred_term</i>: interrogation_point<br><i>definition</i>: "Point within the fluidic subsystem at which the laser intersects the stream, illuminating the cells where they emit their fluorescence and scatter the light. The laser irratidates the cell at the interogation point."<br></div></html><hr>
<a name='item100'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400045<br></b><b><i>fluid_pressure_regulator</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Part of the fluidic subsystem. The fluid pressure regulator maintains constant pressure within the sheath and or sample lines by filling the lines with enough gas to push the fluid at the desired rate. The gas is usually air, and less freqeuntly nitorgen. In the sheath line, the gas is pushed into the sheath tank. In the sample line the gas is pushed into the collection tube. Fluid pressure regulators maintain great enough pressure to push sample fluid out of the tube and sheath fluid out of the sheath tank."</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: LSR2 fluid pressure regulator<br><i>preferred_term</i>: fluid_pressure_regulator<br><i>definition</i>: "Part of the fluidic subsystem. The fluid pressure regulator maintains constant pressure within the sheath and or sample lines by filling the lines with enough gas to push the fluid at the desired rate. The gas is usually air, and less freqeuntly nitorgen. In the sheath line, the gas is pushed into the sheath tank. In the sample line the gas is pushed into the collection tube. Fluid pressure regulators maintain great enough pressure to push sample fluid out of the tube and sheath fluid out of the sheath tank."<br></div></html><hr>
<a name='item101'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400085<br></b><b><i>piezo_electric_crystal</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Apparatus in the fluidic subsystem of sorters that vibrates to break up the stream coming out of the flow chamber into droplets for sorting. The peizo electric crystal vibrates in a manner that breaks off droplets at regular intervals. Not all droplets contain a cell.</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.flocyte.com/FRTP/Resources/flow_cytometry_glossary.htm, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "quartz piezoelectric crystal, topaz piezoelectric crystal, piezoelectric crystal in a sonar"<br><i>preferred_term</i>: piezo_electric_crystal<br><i>definition</i>: Apparatus in the fluidic subsystem of sorters that vibrates to break up the stream coming out of the flow chamber into droplets for sorting. The peizo electric crystal vibrates in a manner that breaks off droplets at regular intervals. Not all droplets contain a cell.<br></div></html><hr>
<a name='item102'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400086<br></b><b><i>plate_loader</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Part of the fluidics system. A plate loader positions the wells of a multi-well plate under the aspiration tube is a preset order. A plate loader is used for high throughput applications.</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.beckmancoulter.com/literature/Bioresearch/P-10202A.pdf, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: FC 500 plate loader<br><i>preferred_term</i>: plate_loader<br><i>definition</i>: Part of the fluidics system. A plate loader positions the wells of a multi-well plate under the aspiration tube is a preset order. A plate loader is used for high throughput applications.<br></div></html><hr>
<a name='item103'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400092<br></b><b><i>?sheath_fluid</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Part of the fluidics system. Sheath fluid is a flowing medium, usually water, maintained at lower pressure than the sample fluid and used to keep the sample fluid focused and laminar while passing through the flow chamber. The sheath fluid helps maintain laminar flow."</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: DS: Is a fluid an instrument? Or even part of an instrument ?<br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: LSR2 sheath fluid<br><i>preferred_term</i>: sheath_fluid<br><i>definition</i>: "Part of the fluidics system. Sheath fluid is a flowing medium, usually water, maintained at lower pressure than the sample fluid and used to keep the sample fluid focused and laminar while passing through the flow chamber. The sheath fluid helps maintain laminar flow."<br></div></html><hr>
<a name='item104'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400093<br></b><b><i>?sheath_propellent</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Part of the fluidic subsystem. The sheath propellent is a gas, usually air or nitrogen, used to push the sheath fluid into the flow chamber at a constant rate. Sheath propellent is used to push the sheath fluid into the flow chamber."</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>editor_note</i>: DS: Is a propellant an instrument? Or even part of an instrument ? Move to function/role?<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: LSR2 sheath propellent<br><i>preferred_term</i>: sheath_propellent<br><i>definition</i>: "Part of the fluidic subsystem. The sheath propellent is a gas, usually air or nitrogen, used to push the sheath fluid into the flow chamber at a constant rate. Sheath propellent is used to push the sheath fluid into the flow chamber."<br></div></html><hr>
<a name='item105'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400081<br></b><b><i>particle_delivery_vessel</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Part of the fluidics subsystem. A particle delivery vessel is used to introcue either a single sample or multiple samples to a flow cytometer. The most common particle delivery vessel is a sample tube.</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: FC 500 particle delivery vessel<br><i>preferred_term</i>: particle_delivery_vessel<br><i>definition</i>: Part of the fluidics subsystem. A particle delivery vessel is used to introcue either a single sample or multiple samples to a flow cytometer. The most common particle delivery vessel is a sample tube.<br></div></html><hr>
<a name='item106'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400076<br></b><b><i>multi-well_plate</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A partical deliver vessel. A multi-well plate is a vessel that can deliver multiple samples to a flow cytometer in a specified order. It must be used with a plate loader.</div></i><font color="#000080">particle_delivery_vessel</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.nuncbrand.com/page.aspx?ID=301, 2007-05-10"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "96 well plate, 48 multiwell plate"<br><i>preferred_term</i>: multi-well_plate<br><i>definition</i>: A partical deliver vessel. A multi-well plate is a vessel that can deliver multiple samples to a flow cytometer in a specified order. It must be used with a plate loader.<br></div></html><hr>
<a name='item107'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400091<br></b><b><i>sample_tube</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A partical delivery vessel. The sample tube is a vessel in which the sample is introduced to the cytometer. Frequently the tube is placed on the cytometer in such a manner that a seal is formed between the tube and cytometer, and gas is used to create enough pressure to push the sample out of the tube and into the cytometer."</div></i><font color="#000080">particle_delivery_vessel</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: sample tube in a cytometer<br><i>preferred_term</i>: sample_tube<br><i>definition</i>: "A partical delivery vessel. The sample tube is a vessel in which the sample is introduced to the cytometer. Frequently the tube is placed on the cytometer in such a manner that a seal is formed between the tube and cytometer, and gas is used to create enough pressure to push the sample out of the tube and into the cytometer."<br></div></html><hr>
<a name='item108'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400043<br></b><b><i>flow_cell</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Aparatus in the fluidic subsystem where the sheath and sample meet. Can be one of several types; jet-in-air, quartz cuvette, or a hybrid of the two. The sample flows through the center of a fluid column of sheath fluid in the flow cell."</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.flocyte.com/FRTP/Resources/flow_cytometry_glossary.htm, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Biofilm Flow Cell<br><i>preferred_term</i>: flow_cell<br><i>definition</i>: "Aparatus in the fluidic subsystem where the sheath and sample meet. Can be one of several types; jet-in-air, quartz cuvette, or a hybrid of the two. The sample flows through the center of a fluid column of sheath fluid in the flow cell."<br></div></html><hr>
<a name='item109'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400061<br></b><b><i>?jet_in_air_flow_chamber</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A flow cell in which the cells are pinched off as droplets and allowed to drop through the air. The laser intersects with the cell in mid-air. A jet in air flow chamber allows the laser to intersect with the cells without any intermitent media other than air.</div></i><font color="#000080">flow_cell</font><br><i>definition_editor</i>: John Quinn<br><i>editor_note</i>: DS: a chamber is not an instrument, its a spatial entity...<br><i>definition_citation</i>: "http://www.flocyte.com/FRTP/Resources/flow_cytometry_glossary.htm, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Jet-in-air flow chamber in DakoCytomation MoFlo high-speed cell sorter<br><i>preferred_term</i>: jet_in_air_flow_chamber<br><i>definition</i>: A flow cell in which the cells are pinched off as droplets and allowed to drop through the air. The laser intersects with the cell in mid-air. A jet in air flow chamber allows the laser to intersect with the cells without any intermitent media other than air.<br></div></html><hr>
<a name='item110'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400088<br></b><b><i>quartz_cuvette_flow_chamber</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A flow cell in which the laser irradiates the cell as it passes through a quartz cuvette. A quartz cuvette flow chamber can be used to allow the laser to irradiate cells.</div></i><font color="#000080">flow_cell</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.flocyte.com/FRTP/Resources/flow_cytometry_glossary.htm, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "CVF-Q-10 flow chamber, CV-Q-10 flow chamber"<br><i>preferred_term</i>: quartz_cuvette_flow_chamber<br><i>definition</i>: A flow cell in which the laser irradiates the cell as it passes through a quartz cuvette. A quartz cuvette flow chamber can be used to allow the laser to irradiate cells.<br></div></html><hr>
<a name='item111'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400102<br></b><b><i>waste_tank</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Part of the fluidics systems. In analyzers the sheath and sample fluid, once analyzed is dumped into a waste tank. The waste tank is where the fluids passing through the cytometer end up after the analysis is finished."</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: LSR2 waste tank<br><i>preferred_term</i>: waste_tank<br><i>definition</i>: "Part of the fluidics systems. In analyzers the sheath and sample fluid, once analyzed is dumped into a waste tank. The waste tank is where the fluids passing through the cytometer end up after the analysis is finished."<br></div></html><hr>
<a name='item112'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400100<br></b><b><i>syringe_pump</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Part of the fluidics system. A syringe pump can be used to inject the sample fluid and cells into the sheath fluid in the flow chamber. Syringe pumps are useful for creating stable flow rates.</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.answers.com/topic/syringe, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: NE-1000 Single Syringe Pump<br><i>preferred_term</i>: syringe_pump<br><i>definition</i>: Part of the fluidics system. A syringe pump can be used to inject the sample fluid and cells into the sheath fluid in the flow chamber. Syringe pumps are useful for creating stable flow rates.<br></div></html><hr>
<a name='item113'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400020<br></b><b><i>?collection_tube</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Part of the fluidics subsystem. The collection tube is a vessel for capturing cells of interest that have been identified by a sorter. The collection tube is the end location of sorted cells.</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>editor_note</i>: DS: a collection tube is no instrument. This is yet another example of distorted is_a and part_of hierarchy...
part_of fluidic_subsystem<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: LSR2 collection tube<br><i>preferred_term</i>: collection_tube<br><i>definition</i>: Part of the fluidics subsystem. The collection tube is a vessel for capturing cells of interest that have been identified by a sorter. The collection tube is the end location of sorted cells.<br></div></html><hr>
<a name='item114'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400090<br></b><b><i>?sample_fluid</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Part of the fluidics system. The sample fluid is the flowing medium within which cells are transported to the flow chamber in. The sample fluid is frequently water or phosphate buffered saline.</div></i><font color="#000080">cardinal_part_of_fluidic_system</font><br><i>definition_editor</i>: John Quinn<br><i>editor_note</i>: DS: Is a fluid an instrument? Or even part of an instrument ?<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: LSR2 sample fluid<br><i>preferred_term</i>: sample_fluid<br><i>definition</i>: Part of the fluidics system. The sample fluid is the flowing medium within which cells are transported to the flow chamber in. The sample fluid is frequently water or phosphate buffered saline.<br></div></html><hr>
<a name='item115'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400132<br></b><b><i>_collected_relations</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">This is a helper class / bin to collect things that will be modeled as relations...</div></i><font color="#000080">artefact_object</font><br><i>definition</i>: This is a helper class / bin to collect things that will be modeled as relations...<br></div></html><hr>
<a name='item116'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400058<br></b><b><i>?instrument_setting</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A specification of instrument component parameter controls.</div></i><font color="#000080">_collected_relations</font><br><i>definition_editor</i>: Thomas Moloshok<br><i>editor_note</i>: DS: not an instrument<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Adjustment, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: lightsource voltage<br><i>preferred_term</i>: instrument_setting<br><i>definition</i>: A specification of instrument component parameter controls.<br></div></html><hr>
<a name='item117'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400057<br></b><b><i>?instrument_configuration</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A specification of the assemblage of instrument components.</div></i><font color="#000080">_collected_relations</font><br><i>definition_editor</i>: Thomas Moloshok<br><i>editor_note</i>: DS: not an instrument,<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Product_configuration, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: optical filter<br><i>preferred_term</i>: instrument_configuration<br><i>definition</i>: A specification of the assemblage of instrument components.<br></div></html><hr>
<a name='item118'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400002<br></b><b><i>device</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A device is an artefact_object that is designed with the intent to provide an advantage as a participant in a process.</div></i><font color="#000080">artefact_object</font><br><i>definition_editor</i>: OBI InstrumentBranch<br><i>definition_citation</i>: OBI<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: <br><i>preferred_term</i>: device<br><i>definition</i>: A device is an artefact_object that is designed with the intent to provide an advantage as a participant in a process.<br></div></html><hr>
<a name='item119'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400004<br></b><b><i>labware</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The class labware is here as a place holder class to accout for entities such as test tubes, beakers, centrifuge tubes etc. It is expected as we identify subclasses we will be in a better position to identify the hierarchy, the definition and possibly the label for this class.</div></i><font color="#000080">device</font><br><i>definition_editor</i>: OBI InstrumentBranch<br><i>definition_citation</i>: OBI<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: <br><i>preferred_term</i>: TODO:<br><i>definition</i>: Labware is equipment that is inert and are independent_continuants.<br></div></html><hr>
<a name='item120'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000005<br></b><b><i>paper_filter</i></b><i><br></i><font color="#000080">labware</font><br></div></html><hr>
<a name='item121'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000007<br></b><b><i>filtration_instrument</i></b><i><br></i><font color="#000080">labware</font><br></div></html><hr>
<a name='item122'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000004<br></b><b><i>polystyrene_tubes</i></b><i><br></i><font color="#000080">labware</font><br><i>alternative_term</i>: scintillation_tubes<br></div></html><hr>
<a name='item123'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000008<br></b><b><i>supernatant_collection_system_harvesting_frames</i></b><i><br></i><font color="#000080">labware</font><br></div></html><hr>
<a name='item124'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000001<br></b><b><i>microtitre_plate</i></b><i><br></i><font color="#000080">labware</font><br></div></html><hr>
<a name='item125'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400003<br></b><b><i>instrument</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument is a device which provides a mechanical or electronic advantage as a participant in a process</div></i><font color="#000080">device</font><br><i>definition_editor</i>: OBI InstrumentBranch<br><i>editor_note</i>: rem1: At the meeting in Philly I wrote that BS said that it should be a count term, such as a set of instruments. By allowing connected devices we avoid the problem of things like LCMS being considered separate items, since we have a number of discrete functions like separation, making things fly, monitoring the charge and mass of what flies etc that are all part of a single machine.<br><i>definition_citation</i>: OBI<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: <br><i>preferred_term</i>: instrument<br><i>definition</i>: An instrument is a device which provides a mechanical or electronic advantage as a participant in a process<br></div></html><hr>
<a name='item126'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400107<br></b><b><i>computer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A computer is an electronic instrument that stores, retrieves, and processes data, that can be programmed with instructions."</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: raw import<br><i>example_of_usage</i>: "Apple PowerBook, Dell OptiPlex"<br><i>preferred_term</i>: computer<br><i>definition</i>: "A computer is an electronic instrument that stores, retrieves, and processes data, that can be programmed with instructions."<br></div></html><hr>
<a name='item127'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400108<br></b><b><i>heating_block</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An instrument which is used to maintain the temperature of a sample at a defined, constant temperature by heating."</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: raw import<br><i>example_of_usage</i>: An instrument used to heat and/or maintain material at a set temperature.<br><i>preferred_term</i>: heating_block<br><i>definition</i>: "An instrument which is used to maintain the temperature of a sample at a defined, constant temperature by heating."<br></div></html><hr>
<a name='item128'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400114<br></b><b><i>sonicator</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An instrument that converts a variable electrical current to mechanical vibration of a metalic probe. The instrument is used for the lysis of cells, the mixing of compounds or solutions, or to create emulsions."</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Sonicator 3000<br><i>preferred_term</i>: sonicator<br><i>definition</i>: "An instrument that converts a variable electrical current to mechanical vibration of a metalic probe. The instrument is used for the lysis of cells, the mixing of compounds or solutions, or to create emulsions."<br></div></html><hr>
<a name='item129'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400117<br></b><b><i>vacuum_dryer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An instrument which removes liquid by the application of negative pressure,  i.e. vacuum."</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Model 777 Microarray Oven<br><i>preferred_term</i>: vacuum_dryer<br><i>definition</i>: "An instrument which removes liquid by the application of negative pressure,  i.e. vacuum."<br></div></html><hr>
<a name='item130'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400120<br></b><b><i>waterbath</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument that has a water-filled container that can be set and maintained at a defined temperature.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: VWR Signature Deep-Chamber Heated Water Bath<br><i>preferred_term</i>: waterbath<br><i>definition</i>: An instrument that has a water-filled container that can be set and maintained at a defined temperature.<br></div></html><hr>
<a name='item131'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400080<br></b><b><i>optical_subsystem</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An optical subsystem is an instrument that deals with the behavior and properties of light and the interaction of light with matter. In a flow cytometer the optical system consists of excitation optics and collection optics. The excitation optics consist of the laser and lenses that are used to shape and focus the laser beam. The collections optics consist of a collection lens to collect light emitted from the particle_laser beam interaction and a system of optical mirrors and filters to route specified wavelengths of the collected light to designated optical detectors. The optical subsystem in a flow cytometer consists of the equipment used to irradiate particles, and collect the light either emmitted or scattered by those particles."</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: http://en.wikipedia.org/wiki/Optics Intro to Flow by BD www.petermac.org/pdf/Intro%20to%20Flow%20Cytometry%20.pdf<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: optical subsystem of a cytometer<br><i>preferred_term</i>: optical_subsystem<br><i>definition</i>: "An optical subsystem is an instrument that deals with the behavior and properties of light and the interaction of light with matter. In a flow cytometer the optical system consists of excitation optics and collection optics. The excitation optics consist of the laser and lenses that are used to shape and focus the laser beam. The collections optics consist of a collection lens to collect light emitted from the particle_laser beam interaction and a system of optical mirrors and filters to route specified wavelengths of the collected light to designated optical detectors. The optical subsystem in a flow cytometer consists of the equipment used to irradiate particles, and collect the light either emmitted or scattered by those particles."<br></div></html><hr>
<a name='item132'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400065<br></b><b><i>light_source</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Devices that provide light for use in a distant area using a delivery system (e.g., fiberoptics). Light sources may include one of a variety of lamps (e.g., xenon, halogen, mercury). Most light sources are operated from line power, but some may be powered from batteries. They are mostly used in endoscopic, microscopic, and other examination and/or in surgical procedures. The light source is part of the optical subsystem. In a flow cytometer the light source directs high intensity light at particles at the interogation point. The light source in a flow cytometer is usually a laser."</div></i><font color="#000080">optical_subsystem</font><br><i>definition_editor</i>: "John Quinn, Elizabeth M. Goralczyk; Olga Tchuvatkina"<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: air-cooled argon laser<br><i>preferred_term</i>: light_source<br><i>definition</i>: "Devices that provide light for use in a distant area using a delivery system (e.g., fiberoptics). Light sources may include one of a variety of lamps (e.g., xenon, halogen, mercury). Most light sources are operated from line power, but some may be powered from batteries. They are mostly used in endoscopic, microscopic, and other examination and/or in surgical procedures. The light source is part of the optical subsystem. In a flow cytometer the light source directs high intensity light at particles at the interogation point. The light source in a flow cytometer is usually a laser."<br></div></html><hr>
<a name='item133'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400064<br></b><b><i>laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A laser (acronym for light amplification by the stimulated emission of radiation) is an optical source that emits photons in a coherent beam. A laser is the most common way to irradiate a cell in a flow cytometer.</div></i><font color="#000080">light_source</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Laser, 2007-04-30"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: HeCd laser<br><i>preferred_term</i>: laser<br><i>definition</i>: A laser (acronym for light amplification by the stimulated emission of radiation) is an optical source that emits photons in a coherent beam. A laser is the most common way to irradiate a cell in a flow cytometer.<br></div></html><hr>
<a name='item134'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400047<br></b><b><i>gas_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A gas laser is a laser in which an electric current is discharged through a gas to produce light. A gas laser can by used to irradiate cells in a flow cytometer.</div></i><font color="#000080">laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Gas_laser, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: helium-neon gas laser<br><i>preferred_term</i>: gas_laser<br><i>definition</i>: A gas laser is a laser in which an electric current is discharged through a gas to produce light. A gas laser can by used to irradiate cells in a flow cytometer.<br></div></html><hr>
<a name='item135'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400073<br></b><b><i>metal_vapor_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A metal vapor laser is a laser in which the lasing medium is metal vapor. A metal vapor laser can by used to irradiate cells in a flow cytometer.</div></i><font color="#000080">gas_laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Gold vapor laser, Helium-cadmium metal-vapor laser"<br><i>preferred_term</i>: metal_vapor_laser<br><i>definition</i>: A metal vapor laser is a laser in which the lasing medium is metal vapor. A metal vapor laser can by used to irradiate cells in a flow cytometer.<br></div></html><hr>
<a name='item136'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400052<br></b><b><i>helium_cadmium_ion_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The helium-cadmium laser emits wavelengths of 442, 325 and 354 nms. This laser is a metal vapor laser. A helium-cadmium laser can by used to irradiate cells in a flow cytometer."</div></i><font color="#000080">metal_vapor_laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Laser#Gas_lasers, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: KIMMON HeCd 325nm laser<br><i>preferred_term</i>: helium_cadmium_ion_laser<br><i>definition</i>: "The helium-cadmium laser emits wavelengths of 442, 325 and 354 nms. This laser is a metal vapor laser. A helium-cadmium laser can by used to irradiate cells in a flow cytometer."<br></div></html><hr>
<a name='item137'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400060<br></b><b><i>ion_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An ion laser is a gas laser which uses an ionized gas as its lasing medium. An ion laser is a subclass of gas lasers.</div></i><font color="#000080">gas_laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Ion_laser, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: 2 Watt Lexel 88 Argon Ion laser<br><i>preferred_term</i>: ion_laser<br><i>definition</i>: An ion laser is a gas laser which uses an ionized gas as its lasing medium. An ion laser is a subclass of gas lasers.<br></div></html><hr>
<a name='item138'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400010<br></b><b><i>argon_ion_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An argon-ion laser is an ion laser that uses argon as the gas. These lasers are used primarily to emit light at wave lengths of 458 nm, 488 nm or 514.5 nm, though it is possible to use them to emit several wavelengths of blue and green light. Argon-ion lasers can emit light at many different wave lenghts, and excite a number of different flourochromes."</div></i><font color="#000080">ion_laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Laser#Gas_lasers, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: argon ion laser in a cytometer<br><i>preferred_term</i>: argon_ion_laser<br><i>definition</i>: "An argon-ion laser is an ion laser that uses argon as the gas. These lasers are used primarily to emit light at wave lengths of 458 nm, 488 nm or 514.5 nm, though it is possible to use them to emit several wavelengths of blue and green light. Argon-ion lasers can emit light at many different wave lenghts, and excite a number of different flourochromes."<br></div></html><hr>
<a name='item139'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400074<br></b><b><i>mixed_argon-krypton_gas_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A laser in which the lasing medium is a mixture of argon and krypton. A mixed argon-krypton laser can by used to irradiate cells in a flow cytometer.</div></i><font color="#000080">ion_laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.eio.com/repairfaq/sam/laserarg.htm, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: argon-krypton laser in a cytometer<br><i>preferred_term</i>: mixed_argon-krypton_gas_laser<br><i>definition</i>: A laser in which the lasing medium is a mixture of argon and krypton. A mixed argon-krypton laser can by used to irradiate cells in a flow cytometer.<br></div></html><hr>
<a name='item140'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400062<br></b><b><i>krypton_ion_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A krypton-ion laser is an ion laser that uses krypton as the gas. These lasers can emit at 468, 476, 482, 520, 531, 568, 647 (the most powerfull line), and 676 nm all at once. They have much lower gain than argon lasers however. Krypton-ion laser can by used to irradiate cells in a flow cytometer."</div></i><font color="#000080">ion_laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Lexel 95L krypton laser<br><i>preferred_term</i>: krypton_ion_laser<br><i>definition</i>: "A krypton-ion laser is an ion laser that uses krypton as the gas. These lasers can emit at 468, 476, 482, 520, 531, 568, 647 (the most powerfull line), and 676 nm all at once. They have much lower gain than argon lasers however. Krypton-ion laser can by used to irradiate cells in a flow cytometer."<br></div></html><hr>
<a name='item141'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400053<br></b><b><i>helium_neon_ion_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The helium-neon laser (HeNe) emits 543 nm and 633 nm most commonly and can also be used at 543, 594, and 611 nm. This laser is less expensive than most others used in flow cytometry. A helium neon laser can by used to irradiate cells in a flow cytometer."</div></i><font color="#000080">ion_laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Laser#Gas_lasers, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: helium neon ion laser in a cytometer<br><i>preferred_term</i>: helium_neon_ion_laser<br><i>definition</i>: "The helium-neon laser (HeNe) emits 543 nm and 633 nm most commonly and can also be used at 543, 594, and 611 nm. This laser is less expensive than most others used in flow cytometry. A helium neon laser can by used to irradiate cells in a flow cytometer."<br></div></html><hr>
<a name='item142'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400096<br></b><b><i>solid_state_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A solid-state laser is a laser that uses a gain medium that is a solid, rather than a liquid such as dye lasers or a gas such as gas lasers. Semiconductor-based lasers are also in the solid state, but are generally considered separately from solid-state lasers A solid state laser can by used to irradiate cells in a flow cytometer."</div></i><font color="#000080">laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Solid-state_laser, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: Solid State Heat Capacity Laser developed at DOE's Lawrence Livermore National Laboratory for the USA Army's Space and Missile Defense Command<br><i>preferred_term</i>: solid_state_laser<br><i>definition</i>: "A solid-state laser is a laser that uses a gain medium that is a solid, rather than a liquid such as dye lasers or a gas such as gas lasers. Semiconductor-based lasers are also in the solid state, but are generally considered separately from solid-state lasers A solid state laser can by used to irradiate cells in a flow cytometer."<br></div></html><hr>
<a name='item143'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400077<br></b><b><i>neodymium-YAG_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A Neodymium-YAG (yttrium aluminum garnet) laser is a solid state laser in which the lasing medium is a solid rod of crystaline material pumped by a flashlamp or a diode laser. Typical output wavelengths arre 355, 532, and 1064 nm. A neodymium-YAG laser can by used to irradiate cells in a flow cytometer."</div></i><font color="#000080">solid_state_laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: Neodymium-YAG Laser in DURIP99 System<br><i>preferred_term</i>: neodymium-YAG_laser<br><i>definition</i>: "A Neodymium-YAG (yttrium aluminum garnet) laser is a solid state laser in which the lasing medium is a solid rod of crystaline material pumped by a flashlamp or a diode laser. Typical output wavelengths arre 355, 532, and 1064 nm. A neodymium-YAG laser can by used to irradiate cells in a flow cytometer."<br></div></html><hr>
<a name='item144'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400033<br></b><b><i>diode_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A diode laser is a laser in which the active medium is a semiconductor similar to that found in a light-emitting diode. Laser diodes emit at wavelengths from 375 nm to 1800 nm, and wavelengths of over 3 __m have been demonstrated. A diode laser can by used to irradiate cells in a flow cytometer."</div></i><font color="#000080">laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Semiconductor_laser, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "FAX-RS3-H0 diode laser manufactured by Diode Laser Concepts, Inc."<br><i>preferred_term</i>: diode_laser<br><i>definition</i>: "A diode laser is a laser in which the active medium is a semiconductor similar to that found in a light-emitting diode. Laser diodes emit at wavelengths from 375 nm to 1800 nm, and wavelengths of over 3 __m have been demonstrated. A diode laser can by used to irradiate cells in a flow cytometer."<br></div></html><hr>
<a name='item145'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400034<br></b><b><i>dye_laser</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A dye laser is a laser in which the lasing medium is a flourescent de, usually disolved in an organic solvent such as ethanol or ethylene glycol. The particular dye used determines the wavelengths the laser can emit. A dye laser can by used to irradiate cells in a flow cytometer."</div></i><font color="#000080">laser</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Practical Flow Cytometry 4th Edition, Howard Shapiro, ISBN-10: 0471411256, ISBN-13: 978-0471411253"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: Rhodamine 101 dye laser<br><i>preferred_term</i>: dye_laser<br><i>definition</i>: "A dye laser is a laser in which the lasing medium is a flourescent de, usually disolved in an organic solvent such as ethanol or ethylene glycol. The particular dye used determines the wavelengths the laser can emit. A dye laser can by used to irradiate cells in a flow cytometer."<br></div></html><hr>
<a name='item146'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400009<br></b><b><i>arc_lamp</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Arc lamp is the general term for a light source that produces light by an electric arc (or voltaic arc). The lamp consists of two electrodes typically made of tungsten which are separated by a gas. The type of lamp is often named by the gas contained in the bulb; including neon, argon, xenon, krypton, sodium, metal halide, and mercury. The electric arc in an arc lamp consists of gas which is initially ionized by a voltage and is therefore electrically conductive. To start an arc lamp, usually a very high voltage is needed to \ignite\ or \strike\ the arc. This requires an electrical circuit sometimes called an \igniter\, which is part of a larger circuit called the \ballast\. The ballast supplies a suitable voltage and current to the lamp as its electrical characteristics change with temperature and time. Older cytometers may use arc lamps to irradiate particles at the interogation point."</div></i><font color="#000080">light_source</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Arc_lamp, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: The Jablochkoff Candle<br><i>preferred_term</i>: arc_lamp<br><i>definition</i>: "Arc lamp is the general term for a light source that produces light by an electric arc (or voltaic arc). The lamp consists of two electrodes typically made of tungsten which are separated by a gas. The type of lamp is often named by the gas contained in the bulb; including neon, argon, xenon, krypton, sodium, metal halide, and mercury. The electric arc in an arc lamp consists of gas which is initially ionized by a voltage and is therefore electrically conductive. To start an arc lamp, usually a very high voltage is needed to \ignite\ or \strike\ the arc. This requires an electrical circuit sometimes called an \igniter\, which is part of a larger circuit called the \ballast\. The ballast supplies a suitable voltage and current to the lamp as its electrical characteristics change with temperature and time. Older cytometers may use arc lamps to irradiate particles at the interogation point."<br></div></html><hr>
<a name='item147'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400079<br></b><b><i>optical_filter</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Part of the optical subsystem that selectively transmits light having certain properties (often, a particular range of wavelengths, that is, range of colours of light), while blocking the remainder. They are commonly used in photography, in many optical instruments, and to colour stage lighting Optical filters can be arranged to segregate and collect light by wave length."</div></i><font color="#000080">optical_subsystem</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Optical_filter, 2007-04-30"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "720 LP filter, 580/30 BP filter"<br><i>preferred_term</i>: optical_filter<br><i>definition</i>: "Part of the optical subsystem that selectively transmits light having certain properties (often, a particular range of wavelengths, that is, range of colours of light), while blocking the remainder. They are commonly used in photography, in many optical instruments, and to colour stage lighting Optical filters can be arranged to segregate and collect light by wave length."<br></div></html><hr>
<a name='item148'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400067<br></b><b><i>long_pass_filter</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Optical filter that passes high wavelengths of light but attenuates (or reduces) wavelengths lower than the cutoff frequency. A long pass filter with a cutoff of 500 nm would pass all wavelengths greater than 500 nm.</div></i><font color="#000080">optical_filter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/high-pass_filter, 2007-04-30"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: 750 LP filter<br><i>preferred_term</i>: long_pass_filter<br><i>definition</i>: Optical filter that passes high wavelengths of light but attenuates (or reduces) wavelengths lower than the cutoff frequency. A long pass filter with a cutoff of 500 nm would pass all wavelengths greater than 500 nm.<br></div></html><hr>
<a name='item149'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400031<br></b><b><i>dichroic_mirror</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A dichroic mirror is a very-accurate optical filter used to selectively reflect light of a small range of colors while passing other colors. A dichroic filter passes the specified range of light whereas a dichroic mirror reflects the specified range of light.</div></i><font color="#000080">optical_filter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Dichroic_mirror, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: ViewLux Alexa 594 dichroic mirror<br><i>preferred_term</i>: dichroic_mirror<br><i>definition</i>: A dichroic mirror is a very-accurate optical filter used to selectively reflect light of a small range of colors while passing other colors. A dichroic filter passes the specified range of light whereas a dichroic mirror reflects the specified range of light.<br></div></html><hr>
<a name='item150'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400013<br></b><b><i>band_pass_filter</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Optical filter that passes wavelengths of light within a certain range and rejects (attenuates) frequencies outside that range. The passed wavelengths are indicated in the specifications of the filter and its name. A 480/20 bandpass filter pass light with at wavelengths of 460 to 500 nm and attenuates all others.</div></i><font color="#000080">optical_filter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Band_pass_filter, 2007-04-30"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "530/30 BP filter, 585/42 BP filter"<br><i>preferred_term</i>: band_pass_filter<br><i>definition</i>: Optical filter that passes wavelengths of light within a certain range and rejects (attenuates) frequencies outside that range. The passed wavelengths are indicated in the specifications of the filter and its name. A 480/20 bandpass filter pass light with at wavelengths of 460 to 500 nm and attenuates all others.<br></div></html><hr>
<a name='item151'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400030<br></b><b><i>dichroic_filter</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A dichroic filter is a very-accurate optical filter used to selectively pass light of a small range of colors while reflecting other colors. A dichroic filter passes the specified range of light whereas a dichroic mirror reflects the specified range of light.</div></i><font color="#000080">optical_filter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Dichroic_filter, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: Cy3 Dichroic Filter<br><i>preferred_term</i>: dichroic_filter<br><i>definition</i>: A dichroic filter is a very-accurate optical filter used to selectively pass light of a small range of colors while reflecting other colors. A dichroic filter passes the specified range of light whereas a dichroic mirror reflects the specified range of light.<br></div></html><hr>
<a name='item152'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400095<br></b><b><i>short_pass_filter</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Optical filter that passes low wavelengths of light but attenuates (or reduces) wavelengths higher than the cutoff frequency. A short pass filter with a cutoff of 500 nm would pass all wavelengths less than 500 nm.</div></i><font color="#000080">optical_filter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Low-pass_filter, 2007-04-30"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: 620 SP filter<br><i>preferred_term</i>: short_pass_filter<br><i>definition</i>: Optical filter that passes low wavelengths of light but attenuates (or reduces) wavelengths higher than the cutoff frequency. A short pass filter with a cutoff of 500 nm would pass all wavelengths less than 500 nm.<br></div></html><hr>
<a name='item153'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400078<br></b><b><i>obscuration_bar</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An obscuration bar is a strip of metal or other material that serves to block out direct light from the illuminating beam. The obscuration bar prevents the bright light scattered in the forward directions from burning out the collection device.</div></i><font color="#000080">optical_subsystem</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "Flow Cytometry: First Principles, by Alice Longobardi Givan, ISBN-10: 0471382248, ISBN-13: 978-0471382249"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: obscuration bar in a flow cytometer<br><i>preferred_term</i>: obscuration_bar<br><i>definition</i>: An obscuration bar is a strip of metal or other material that serves to block out direct light from the illuminating beam. The obscuration bar prevents the bright light scattered in the forward directions from burning out the collection device.<br></div></html><hr>
<a name='item154'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400119<br></b><b><i>microarray_wash_station</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument used to wash Affymetrix type arrays.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: ArrayIt  Microarray Wash Station<br><i>preferred_term</i>: microarray_wash_station<br><i>definition</i>: An instrument used to wash Affymetrix type arrays.<br></div></html><hr>
<a name='item155'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400113<br></b><b><i>oligonucleotide_synthesizer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument used to chemically synthesize oligonucleotides.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Automated Multiplex Oligonucleotide Synthesizer<br><i>preferred_term</i>: oligonucleotide_synthesizer<br><i>definition</i>: An instrument used to chemically synthesize oligonucleotides.<br></div></html><hr>
<a name='item156'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400109<br></b><b><i>homogenizer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A mechanical instrument which is used to create a uniform dispersion of an insoluble phase within a liquid phase.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: raw import<br><i>example_of_usage</i>: An instrument that can be used to disperse or emulsify material.<br><i>preferred_term</i>: homogenizer<br><i>definition</i>: A mechanical instrument which is used to create a uniform dispersion of an insoluble phase within a liquid phase.<br></div></html><hr>
<a name='item157'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400104<br></b><b><i>array_scanner</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument which acquires visual images of microarray chips by measuring fluorescence (induced with lasers) from labeled molecules on the surface of the microarray chip and converts the information into digital data.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: raw import<br><i>example_of_usage</i>: "GenePix 4200A, GenePix4000B"<br><i>preferred_term</i>: array_scanner<br><i>definition</i>: An instrument which acquires visual images of microarray chips by measuring fluorescence (induced with lasers) from labeled molecules on the surface of the microarray chip and converts the information into digital data.<br></div></html><hr>
<a name='item158'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000002<br></b><b><i>gamma_counter</i></b><i><br></i><font color="#000080">instrument</font><br></div></html><hr>
<a name='item159'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000003<br></b><b><i>incubator</i></b><i><br></i><font color="#000080">instrument</font><br></div></html><hr>
<a name='item160'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400044<br></b><b><i>?flow_cytometer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An instrument for counting, examining and sorting microscopic particles suspended in a stream of fluid. It allows simultaneous multiparametric analysis of the physical and/or chemical characteristics of single cells flowing through an optical and/or electronic detection apparatus. A flow cytometer is an instrument that can be used to quantitatively measure the properties of individual cells in a flowing medium."</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: John Quinn<br><i>editor_note</i>: DS: Is this class justified? (has currently only one subclass), ask Ryan and Function Branch for differrentiae...<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Flow_cytometer, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: FACS Calibur<br><i>preferred_term</i>: flow_cytometer<br><i>definition</i>: "An instrument for counting, examining and sorting microscopic particles suspended in a stream of fluid. It allows simultaneous multiparametric analysis of the physical and/or chemical characteristics of single cells flowing through an optical and/or electronic detection apparatus. A flow cytometer is an instrument that can be used to quantitatively measure the properties of individual cells in a flowing medium."<br></div></html><hr>
<a name='item161'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400008<br></b><b><i>flow_cytometer_analyzer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An analyser is a flow cytometer that is used to measure properties of particles (whole cells, nuclei, chromosomes, diatoms, plankton, bacteria, viruses) by moving these particles through a detection chamber. An analyser is used to collect data for anaylsis."</div></i><font color="#000080">?flow_cytometer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.flocyte.com/FRTP/Resources/flow_cytometry_glossary.htm, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "FACS Calibur, Luminex 100"<br><i>preferred_term</i>: flow_cytometer_analyser<br><i>definition</i>: "An analyser is a flow cytometer that is used to measure properties of particles (whole cells, nuclei, chromosomes, diatoms, plankton, bacteria, viruses) by moving these particles through a detection chamber. An analyser is used to collect data for anaylsis."<br></div></html><hr>
<a name='item162'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400071<br></b><b><i>MACS_Quant</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The MACS Quant is an analyser manufactured by Miltenyi. It uses digital electronics, and has three lasers of wavelengths 405 nm, 488 nm, and 635 nm. It has filters and dectors to collect 7 flourescent parameters and 2 scatter parameters. The MACS Quant is an analyser usually used in research applications."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.miltenyi.com, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "MACS Quant analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: MACS_Quant<br><i>definition</i>: "The MACS Quant is an analyser manufactured by Miltenyi. It uses digital electronics, and has three lasers of wavelengths 405 nm, 488 nm, and 635 nm. It has filters and dectors to collect 7 flourescent parameters and 2 scatter parameters. The MACS Quant is an analyser usually used in research applications."<br></div></html><hr>
<a name='item163'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400049<br></b><b><i>Guava_EasyCyte_Plus</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Guava EasyCyte Plus System is an analyser that includes a single 488 nm laser, and filters and PMTs to collect up to 4 flourescent parameters. It includes a photodiode forward scatter collector and an optional photodiode for side scatter collection.Guava cytometers use aspiration instead of fluid systems to transport cells within the machine. The EasyCyte plus accepts 96 well plates as well as tubes. The Guava EasyCyte Plus cytometer is a small portable cytometer particularly useful for field measurement."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.guavatechnologies.com/main/products/easycyte-new.cfm, 2007-05-10"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Guava EasyCyte Plus at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Guava_EasyCyte_Plus<br><i>definition</i>: "The Guava EasyCyte Plus System is an analyser that includes a single 488 nm laser, and filters and PMTs to collect up to 4 flourescent parameters. It includes a photodiode forward scatter collector and an optional photodiode for side scatter collection.Guava cytometers use aspiration instead of fluid systems to transport cells within the machine. The EasyCyte plus accepts 96 well plates as well as tubes. The Guava EasyCyte Plus cytometer is a small portable cytometer particularly useful for field measurement."<br></div></html><hr>
<a name='item164'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400012<br></b><b><i>Bactiflow</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Bactiflow is an analyser manufactured by Chemunex SA. It is a cell counter for bacteria and other micro organisms. Bacteria are stained with a fluorochrome and the number of particles that flouresce are counted. The system uses digital electronics, a single laser and a single detector. An ALS version is available as well - Automatic Labeling System. The Bactiflow is specialized cytometer used exclusively for counting microbes."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.chemunex.com/products/chemframe.htm, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Bactiflow analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Bactiflow<br><i>definition</i>: "The Bactiflow is an analyser manufactured by Chemunex SA. It is a cell counter for bacteria and other micro organisms. Bacteria are stained with a fluorochrome and the number of particles that flouresce are counted. The system uses digital electronics, a single laser and a single detector. An ALS version is available as well - Automatic Labeling System. The Bactiflow is specialized cytometer used exclusively for counting microbes."<br></div></html><hr>
<a name='item165'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400005<br></b><b><i>A10-Analyzer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The A10 is an analyser manufactured by Apogee. It uses an arc lamp as a light source, with choices of 75W Xe, 75W Xe/Hg or 100W Hg arc lamps. It has filters and colelctors for up to three flourescent parameters and two scatter parameters. It uses analog electronics. The A10 can be used for measuring the properties of inividal cells."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.apogeeflow.com/flow_cytometry_products.htm, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "A10 analyzer at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: A10-Analyzer<br><i>definition</i>: "The A10 is an analyser manufactured by Apogee. It uses an arc lamp as a light source, with choices of 75W Xe, 75W Xe/Hg or 100W Hg arc lamps. It has filters and colelctors for up to three flourescent parameters and two scatter parameters. It uses analog electronics. The A10 can be used for measuring the properties of inividal cells."<br></div></html><hr>
<a name='item166'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400063<br></b><b><i>LactoScope</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The LactoScope is a spectrophotometer with which the composition of milk and milk products is analysed via infrared technology. The LactoScope determines the amount of the constituents fat, protein, lactose and the total solids content with extreme accuracy."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: Josef Spidlen<br><i>definition_citation</i>: "http://www.aicompanies.com/DeltaCD/lacto_ftir_auto.htm, 2007-05-10"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: LactoScope C4 Automatic Economical<br><i>preferred_term</i>: LactoScope<br><i>definition</i>: "The LactoScope is a spectrophotometer with which the composition of milk and milk products is analysed via infrared technology. The LactoScope determines the amount of the constituents fat, protein, lactose and the total solids content with extreme accuracy."<br></div></html><hr>
<a name='item167'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400039<br></b><b><i>FACS_Scan</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The FACS Scan is an analyser manufactured by Becton Dickinson. IT features analog electronics, one 488 nm solid state laser, and the filters and PMTs to collect up to three flourescent parameters The FACS Scan is an analyser usually used for research applications."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.brc.ubc.ca/brc/facs.html, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "FACS Scan analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: FACS_Scan<br><i>definition</i>: "The FACS Scan is an analyser manufactured by Becton Dickinson. IT features analog electronics, one 488 nm solid state laser, and the filters and PMTs to collect up to three flourescent parameters The FACS Scan is an analyser usually used for research applications."<br></div></html><hr>
<a name='item168'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400050<br></b><b><i>Guava_PCA</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Guava Personal Cell Analysis (PCA) System is an analyser that includes a single 532 nm laser, and filters and PMTs to collect up to 2 flourescent parameters. It includes a photodiode forward scatter collector.Guava cytometers use aspiration instead of fluid systems to transport cells within the machine. The PCA96 uses only tubes to introduce specimen. The Guava PCA cytometer is a small portable cytometer particularly useful for field measurement."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.guavatechnologies.com/main/products/PCA-new.cfm, 2007-05-10"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Guava PCA at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Guava_PCA<br><i>definition</i>: "The Guava Personal Cell Analysis (PCA) System is an analyser that includes a single 532 nm laser, and filters and PMTs to collect up to 2 flourescent parameters. It includes a photodiode forward scatter collector.Guava cytometers use aspiration instead of fluid systems to transport cells within the machine. The PCA96 uses only tubes to introduce specimen. The Guava PCA cytometer is a small portable cytometer particularly useful for field measurement."<br></div></html><hr>
<a name='item169'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400025<br></b><b><i>CyFlow_SL3</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The CyFlow Sl 3 is an analyser manufactured by Partec. Its specs are: _ ultracompact and fully equipped mobile/portable instrument _dimensions [cm]: L 43 x H 16 x D 37 _highest stability/robustness and highest precision _5 optical parameters: FSC, SSC, FL-1, FL-2, FL-3 _ 20mW@488nm blue solid state laser _other laser light sources optional (UV, violet, green, red) _WindowsTM XP FloMax_software for realtime data acquisition, data display, and data evaluation _parallel 16 bit digital pulse processing The CyFlow SL3 is a compact and dedicated portable Flow Cytometer for accurate and affordable cell analysis and true volumetric absolute counting in HIV Monitoring and AIDS patient follow-up by precise and direct CD4 and CD4% measurement."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.partec.de/products/cyflowsl3.html, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "CyFlow Sl 3 analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: CyFlow_SL3<br><i>definition</i>: "The CyFlow Sl 3 is an analyser manufactured by Partec. Its specs are: _ ultracompact and fully equipped mobile/portable instrument _dimensions [cm]: L 43 x H 16 x D 37 _highest stability/robustness and highest precision _5 optical parameters: FSC, SSC, FL-1, FL-2, FL-3 _ 20mW@488nm blue solid state laser _other laser light sources optional (UV, violet, green, red) _WindowsTM XP FloMax_software for realtime data acquisition, data display, and data evaluation _parallel 16 bit digital pulse processing The CyFlow SL3 is a compact and dedicated portable Flow Cytometer for accurate and affordable cell analysis and true volumetric absolute counting in HIV Monitoring and AIDS patient follow-up by precise and direct CD4 and CD4% measurement."<br></div></html><hr>
<a name='item170'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400023<br></b><b><i>CYFlow_ML</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The CyFlow ML is an analyser manufactured by Partec. It is a digital machine. It uses 4 light sources: triple laser configurations including new powerful 200mW@488nm blue solid state laser + 100W UV lamp for highest resolution DNA analysis, and can collect 16 optical parameters: FSC1, FSC2, SSC, FL1-FL13. Ultracompact high end desktop multilaser Flow Cytometer for all applications in cell analysis and absolute counting."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.partec.de/products/cyflow-ml.html, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "CyFlow ML analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: CYFlow_ML<br><i>definition</i>: "The CyFlow ML is an analyser manufactured by Partec. It is a digital machine. It uses 4 light sources: triple laser configurations including new powerful 200mW@488nm blue solid state laser + 100W UV lamp for highest resolution DNA analysis, and can collect 16 optical parameters: FSC1, FSC2, SSC, FL1-FL13. Ultracompact high end desktop multilaser Flow Cytometer for all applications in cell analysis and absolute counting."<br></div></html><hr>
<a name='item171'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400036<br></b><b><i>FACS_Caliber</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The FACS Calibur is one of the most popular cytometers in use for research.</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.bdbiosciences.com/immunocytometry_systems/products/display_product.php?keyID=45, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "FACS Calibur at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: FACS_Caliber<br><i>definition</i>: The FACS Calibur is one of the most popular cytometers in use for research.<br></div></html><hr>
<a name='item172'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400024<br></b><b><i>CyFlow_SL</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The CyFlow Sl is an analyser manufactured by Partec. Its specs are: _ ultracompact and fully equipped mobile/portable instrument _dimensions [cm]: L 43 x H 16 x D 37 _highest stability/robustness and highest precision _5 optical parameters: FSC, SSC, FL-1, FL-2, FL-3 _ 20mW@488nm blue solid state laser _other laser light sources optional (UV, violet, green, red) _WindowsTM XP FloMax_software for realtime data acquisition, data display, and data evaluation _parallel 16 bit digital pulse processing Compact mobile Flow Cytometer for any kind of cell analysis and absolute volumetric counting. The CyFlow_ SL allows to analyze forward and side scatter signals in combination with up to 3 fluorescence channels."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.partec.de/products/cyflow.html, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "CyFlow Sl analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: CyFlow_SL<br><i>definition</i>: "The CyFlow Sl is an analyser manufactured by Partec. Its specs are: _ ultracompact and fully equipped mobile/portable instrument _dimensions [cm]: L 43 x H 16 x D 37 _highest stability/robustness and highest precision _5 optical parameters: FSC, SSC, FL-1, FL-2, FL-3 _ 20mW@488nm blue solid state laser _other laser light sources optional (UV, violet, green, red) _WindowsTM XP FloMax_software for realtime data acquisition, data display, and data evaluation _parallel 16 bit digital pulse processing Compact mobile Flow Cytometer for any kind of cell analysis and absolute volumetric counting. The CyFlow_ SL allows to analyze forward and side scatter signals in combination with up to 3 fluorescence channels."<br></div></html><hr>
<a name='item173'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400054<br></b><b><i>ImageStream</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The ImageStream is an analyzer manufactured by Amnis. Its has digital electronics, a single standard 488 nm solid state laser. In addition an optional 658 nm and your choice of either a 405 nm or 375 nm solid state laser can be added. Information is collected using cameras. The ImageStream system_s CCD camera produces six images of each cell, including darkfield, brightfield, and up to four fluorescent colors. Each image is used to calculate over 40 features, so a six-image assay results in ~250 morphometric and photometric features per cell. The ImageStream is a flow cytometer that takes pictures of the cells in flow."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.amnis.com/, 2007-05-10"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "ImageStream at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: ImageStream<br><i>definition</i>: "The ImageStream is an analyzer manufactured by Amnis. Its has digital electronics, a single standard 488 nm solid state laser. In addition an optional 658 nm and your choice of either a 405 nm or 375 nm solid state laser can be added. Information is collected using cameras. The ImageStream system_s CCD camera produces six images of each cell, including darkfield, brightfield, and up to four fluorescent colors. Each image is used to calculate over 40 features, so a six-image assay results in ~250 morphometric and photometric features per cell. The ImageStream is a flow cytometer that takes pictures of the cells in flow."<br></div></html><hr>
<a name='item174'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400018<br></b><b><i>Cell_Lab_Quanta_SC</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Cell Lab Quanta SC is an alalyser manufactured by Becman Coulter. It features a mercury arc lamp optimized at 366, 405, and 435 nm, and a 488 nm laser diode. It has filters and PMTs to collect up to 3 flourescent parameters, and a photodiode for side scatter detection. This cytometer uses Couter-Volume for cell size measurements. The Cell Lab Quanta SC can be used for measuring the properties of inividal cells."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.beckmancoulter.com/cell-lab, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "Cell Lab Quanta SC analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Cell_Lab_Quanta_SC<br><i>definition</i>: "The Cell Lab Quanta SC is an alalyser manufactured by Becman Coulter. It features a mercury arc lamp optimized at 366, 405, and 435 nm, and a 488 nm laser diode. It has filters and PMTs to collect up to 3 flourescent parameters, and a photodiode for side scatter detection. This cytometer uses Couter-Volume for cell size measurements. The Cell Lab Quanta SC can be used for measuring the properties of inividal cells."<br></div></html><hr>
<a name='item175'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400070<br></b><b><i>Luminex_200</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Luminex 200 is an analyser manufactured by Luminex. The optical specifications are: Reporter laser: 532 nm, nominal output 10 - 15 mW, maximum 500 mW, frequency-doubled diode; mode of operation, continuous wave (CW) _ Classification laser: 635 nm, 9.1 __ 6%, maximum output 25 mW, diode; mode of operation, continuous wave (CW) _ Reporter detector: Photomultiplier tube, detection bandwidth of 565 _ 585 nm _ Classification detector and doublet discriminator: Avalanche photo diodes with temperature compensation"</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.luminexcorp.com/support/faqs.html, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Luminex 200 analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Luminex_200<br><i>definition</i>: "The Luminex 200 is an analyser manufactured by Luminex. The optical specifications are: Reporter laser: 532 nm, nominal output 10 - 15 mW, maximum 500 mW, frequency-doubled diode; mode of operation, continuous wave (CW) _ Classification laser: 635 nm, 9.1 __ 6%, maximum output 25 mW, diode; mode of operation, continuous wave (CW) _ Reporter detector: Photomultiplier tube, detection bandwidth of 565 _ 585 nm _ Classification detector and doublet discriminator: Avalanche photo diodes with temperature compensation"<br></div></html><hr>
<a name='item176'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400069<br></b><b><i>Luminex_100</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The Luminex 100 is an analyzer manufactured by Luminex. It is a single laser system (575 nm) with avalanche photodiodes in red and infrared and a single PMT for fluorescnce. The flow chamber is square quartz cuvette</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.luminexcorp.com/products/luminex_100IS.html, 2007-05-10"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Luminex 100 at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Luminex_100<br><i>definition</i>: The Luminex 100 is an analyzer manufactured by Luminex. It is a single laser system (575 nm) with avalanche photodiodes in red and infrared and a single PMT for fluorescnce. The flow chamber is square quartz cuvette<br></div></html><hr>
<a name='item177'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400022<br></b><b><i>Cyan</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A Cyan is an analyser manufactured by Dako Cytomation. It features include digital electronics, three lasers: 488 nm, 635 nm, and 405 nm, and filters and collectors for nine flourescent parameters and two scatter parameters. The Cyan can be used for measuring the properties of inividal cells."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.dakousa.com/prod_productrelatedinformation?url=gprod_cyan_index.htm, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "Cyan analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Cyan<br><i>definition</i>: "A Cyan is an analyser manufactured by Dako Cytomation. It features include digital electronics, three lasers: 488 nm, 635 nm, and 405 nm, and filters and collectors for nine flourescent parameters and two scatter parameters. The Cyan can be used for measuring the properties of inividal cells."<br></div></html><hr>
<a name='item178'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400042<br></b><b><i>FC_500</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The FC 500 is an analyser manufactured by Beckman Coulter. It features digital electronics, 488 nm and 635 nm lasers, filters and PMTs for 5 flourescent parameters, a diode for collecting side scatter and a solid state detector for forward scatter. The FC 500 is an analyser usually used for either research or clinical applications."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.beckmancoulter.com/products/instrument/flowcytometry/fc500series.asp, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "FC 500 analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: FC_500<br><i>definition</i>: "The FC 500 is an analyser manufactured by Beckman Coulter. It features digital electronics, 488 nm and 635 nm lasers, filters and PMTs for 5 flourescent parameters, a diode for collecting side scatter and a solid state detector for forward scatter. The FC 500 is an analyser usually used for either research or clinical applications."<br></div></html><hr>
<a name='item179'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400038<br></b><b><i>FACS_Canto2</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The FACS Canto 2 is an analyser manufactured by BD. It features digital electronics, two solid state lasers at 488 and 633 nm with the option for a third 405 nm laser, and filters and collectors for measuring up to 8 flourescent paramters with either the 2 or 3 laser option. The FSACS Canto 2 is an analyser usually used in clinical aplications."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.bdbiosciences.com/cgi-bin/literature/view?part_num=23-8786-01, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "FACS Canto 2 analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: FACS_Canto2<br><i>definition</i>: "The FACS Canto 2 is an analyser manufactured by BD. It features digital electronics, two solid state lasers at 488 and 633 nm with the option for a third 405 nm laser, and filters and collectors for measuring up to 8 flourescent paramters with either the 2 or 3 laser option. The FSACS Canto 2 is an analyser usually used in clinical aplications."<br></div></html><hr>
<a name='item180'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400006<br></b><b><i>A40-MiniFCM</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The A40-MiniFCM is an analyzer that allows for the choise of one of four lasers (375nm, 405nm,488nm, 532nm, 635nm), and PMTs and filters for collecting up to four parameters. It uses digital electronics. A military version of this cytometer is available as well. The A40-MiniFCM is geared towards the most demanding applications such as archaea, bacteria and large virus."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.apogeeflow.com/flow_cytometry_products.htm, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "A40-MiniFCM analyzer at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: A40-MiniFCM<br><i>definition</i>: "The A40-MiniFCM is an analyzer that allows for the choise of one of four lasers (375nm, 405nm,488nm, 532nm, 635nm), and PMTs and filters for collecting up to four parameters. It uses digital electronics. A military version of this cytometer is available as well. The A40-MiniFCM is geared towards the most demanding applications such as archaea, bacteria and large virus."<br></div></html><hr>
<a name='item181'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400097<br></b><b><i>Somacount</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Somacount is a cytometer manufactured by Bently Instruments. It is a specialized tool for counting somatic cells in milk by specifically staining them with Ethidium Bromide and counting the cells that flouresce. It has one laser, and the filters and a PMT for the single parameter. There are three sizes availalbe, the 150, 300, and 500 with the number indicating the maximum number of cells tthat can be analysed per hour. The Somacount is an example of a very specific use cytometer; it exclusively counts somatic cells in milk."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.bentleyinstruments.com/somacount.html#Anchor-Bentley-49575, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Somacount 300 at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Somacount<br><i>definition</i>: "The Somacount is a cytometer manufactured by Bently Instruments. It is a specialized tool for counting somatic cells in milk by specifically staining them with Ethidium Bromide and counting the cells that flouresce. It has one laser, and the filters and a PMT for the single parameter. There are three sizes availalbe, the 150, 300, and 500 with the number indicating the maximum number of cells tthat can be analysed per hour. The Somacount is an example of a very specific use cytometer; it exclusively counts somatic cells in milk."<br></div></html><hr>
<a name='item182'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400055<br></b><b><i>inFlux_Analyzer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The inFlux Analyzer is an analyzer manufactured by Cytopeia. It is a digital machine, with these laser options: Coherent 70, 90, 300 series water cooled lasers, solid state UV 355nm, Violet 408nm, Violet-Blue 479nm, Blue-488nm, Green-531nm, Red-635nm, Red-647nm. FIlters and PMTs are used for all parameters (including forward light scatter), and up to 12 PMTs can be included. The Influx analyser can be used to measure the properties of individual cells."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.cytopeia.com/analyzer.htm, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Influx Analyzer at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: inFlux_Analyzer<br><i>definition</i>: "The inFlux Analyzer is an analyzer manufactured by Cytopeia. It is a digital machine, with these laser options: Coherent 70, 90, 300 series water cooled lasers, solid state UV 355nm, Violet 408nm, Violet-Blue 479nm, Blue-488nm, Green-531nm, Red-635nm, Red-647nm. FIlters and PMTs are used for all parameters (including forward light scatter), and up to 12 PMTs can be included. The Influx analyser can be used to measure the properties of individual cells."<br></div></html><hr>
<a name='item183'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400029<br></b><b><i>CytoSub</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The CytoSub is an analyser manufactured by Cyto Buoy Inc. They are a single laser, multi parameter instrument. Various types of micro-laser are offered; available measuring parameters are forward light scatter, side scatter, and fluorescence (max. 9 colour bands). The current series have 5 parameters. The CytoSub_ is the submersible version equipped with a high pressure sample inlet loop and a high pressure housing to allow underwater operation down to 200 m, lowered on a cable or mounted on a flooded underwater vehicle;"</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.cytobuoy.com/, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "CytoSub analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: CytoSub<br><i>definition</i>: "The CytoSub is an analyser manufactured by Cyto Buoy Inc. They are a single laser, multi parameter instrument. Various types of micro-laser are offered; available measuring parameters are forward light scatter, side scatter, and fluorescence (max. 9 colour bands). The current series have 5 parameters. The CytoSub_ is the submersible version equipped with a high pressure sample inlet loop and a high pressure housing to allow underwater operation down to 200 m, lowered on a cable or mounted on a flooded underwater vehicle;"<br></div></html><hr>
<a name='item184'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400027<br></b><b><i>CytoBuoy</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The CytoBuoy is an analyser manufactured by Cyto Buoy Inc. They are a single laser, multi parameter instrument. Various types of micro-laser are offered; available measuring parameters are forward light scatter, side scatter, and fluorescence (max. 9 colour bands). The current series have 5 parameters. The CytoBuoy_ is the buoy based version suited for moored operation with wireless data transfer"</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.cytobuoy.com/, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "CytoBuoy analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: CytoBuoy<br><i>definition</i>: "The CytoBuoy is an analyser manufactured by Cyto Buoy Inc. They are a single laser, multi parameter instrument. Various types of micro-laser are offered; available measuring parameters are forward light scatter, side scatter, and fluorescence (max. 9 colour bands). The current series have 5 parameters. The CytoBuoy_ is the buoy based version suited for moored operation with wireless data transfer"<br></div></html><hr>
<a name='item185'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400098<br></b><b><i>SomaScope</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The SomaScope is an instrument to quantify somatic cells in milk.</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: Josef Spidlen<br><i>definition_citation</i>: "http://www.aicompanies.com/DeltaCD/soma_auto_adv.htm, 2007-05-10"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: SomaScope Mark II Automatic Economical<br><i>preferred_term</i>: SomaScope<br><i>definition</i>: The SomaScope is an instrument to quantify somatic cells in milk.<br></div></html><hr>
<a name='item186'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400028<br></b><b><i>CytoSence</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The CytoSense is an analyser manufactured by Cyto Buoy Inc. They are a single laser, multi parameter instrument. Various types of micro-laser are offered; available measuring parameters are forward light scatter, side scatter, and fluorescence (max. 9 colour bands). The current series have 5 parameters. The CytoSense_ is the basic instrument, which can be used for normal laboratory applications, as well as for autonomous monitoring with internal data logging or direct data transmission. The special instrument design and its splashproof housing allow operation on moving platforms and outdoor sites;"</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.cytobuoy.com/, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "CytoSense analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: CytoSence<br><i>definition</i>: "The CytoSense is an analyser manufactured by Cyto Buoy Inc. They are a single laser, multi parameter instrument. Various types of micro-laser are offered; available measuring parameters are forward light scatter, side scatter, and fluorescence (max. 9 colour bands). The current series have 5 parameters. The CytoSense_ is the basic instrument, which can be used for normal laboratory applications, as well as for autonomous monitoring with internal data logging or direct data transmission. The special instrument design and its splashproof housing allow operation on moving platforms and outdoor sites;"<br></div></html><hr>
<a name='item187'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400051<br></b><b><i>Guava_PCA-96</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Guava Personal Cell Analysis-96 (PCA-96) Systems is an analyser that includes a single 532 nm laser, and filters and PMTs to collect up to 2 flourescent parameters. It includes a photodiode forward scatter collector.Guava cytometers use aspiration instead of fluid systems to transport cells within the machine. The PCA96 uses either tubes or 96 well plates to introduce specimen. The Guava PCA -96 cytometer is a small portable cytometer particularly useful for field measurement."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.guavatechnologies.com/main/products/PCA-96new.cfm, 2007-05-10"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Guava PCA-96 at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Guava_PCA-96<br><i>definition</i>: "The Guava Personal Cell Analysis-96 (PCA-96) Systems is an analyser that includes a single 532 nm laser, and filters and PMTs to collect up to 2 flourescent parameters. It includes a photodiode forward scatter collector.Guava cytometers use aspiration instead of fluid systems to transport cells within the machine. The PCA96 uses either tubes or 96 well plates to introduce specimen. The Guava PCA -96 cytometer is a small portable cytometer particularly useful for field measurement."<br></div></html><hr>
<a name='item188'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400048<br></b><b><i>Guava_EasyCyte_Mini</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Guava EasyCyte Mini is an analyser that includes a single 488 nm laser, and filters and PMTs to collect up to 3 flourescent parameters. It includes a photodiode forward scatter collector and an optional photodiode for side scatter collection.Guava cytometers use aspiration instead of fluid systems to transport cells within the machine. The mini accepts tubes only for inputing cells or beads. The Guava EasyCyte Mini cytometer is a small portable cytometer particularly useful for field measurement."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.guavatechnologies.com/main/products/easyCyteMini.cfm, 2007-05-10"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "Guava EasyCyte Mini at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: Guava_EasyCyte_Mini<br><i>definition</i>: "The Guava EasyCyte Mini is an analyser that includes a single 488 nm laser, and filters and PMTs to collect up to 3 flourescent parameters. It includes a photodiode forward scatter collector and an optional photodiode for side scatter collection.Guava cytometers use aspiration instead of fluid systems to transport cells within the machine. The mini accepts tubes only for inputing cells or beads. The Guava EasyCyte Mini cytometer is a small portable cytometer particularly useful for field measurement."<br></div></html><hr>
<a name='item189'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400037<br></b><b><i>FACS_Canto</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The FACS Canto is an alalyser manufactured by Becton Dickinson. It features digital electronics, a solid state 488 nm, and a HeNe 633 nm lasers, and filters and PMTs for collecting up to 6 flourescent parameters. The FACS Canto is an analyser usually used in clinical applications."</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.bdbiosciences.com/pdfs/brochures/23-8742-00.pdf, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: "FACS Canto analyser at TFL, BCCRC, Vancouver"<br><i>preferred_term</i>: FACS_Canto<br><i>definition</i>: "The FACS Canto is an alalyser manufactured by Becton Dickinson. It features digital electronics, a solid state 488 nm, and a HeNe 633 nm lasers, and filters and PMTs for collecting up to 6 flourescent parameters. The FACS Canto is an analyser usually used in clinical applications."<br></div></html><hr>
<a name='item190'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400099<br></b><b><i>flow_cytometer_sorter</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Flow cytometer that analyzes and separates or sorts particles passing through (based on properties measured during anlysis). A sorter is a flow cytometer that can collect cells of interest.</div></i><font color="#000080">flow_cytometer_analyzer</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.flocyte.com/FRTP/Resources/flow_cytometry_glossary.htm, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: "BioSorter2000, LSR2"<br><i>preferred_term</i>: flow_cytometer_sorter<br><i>definition</i>: Flow cytometer that analyzes and separates or sorts particles passing through (based on properties measured during anlysis). A sorter is a flow cytometer that can collect cells of interest.<br></div></html><hr>
<a name='item191'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400015<br></b><b><i>BioSorter2000</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The BioSorter2000 is a sorter manufactured by Union Biometrica. It includes analog electronics, 488/514 nm multi-line Argon-ion laser for fluorescence and a light stabilized 670 nm forward scatter laser for extinction and time-of-flight. It has filters and PMTs for 3 flourescent parameters and photodiodes for scatter, time of flight, and extinction. The flow cell is quartz cuvette. The BioSorterTM 2000 is an instrument for analyzing and sorting objects from 500 microns to 1.5 millimeters in diameter."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.unionbio.com/products/BioSorter.html, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: BioSorter 2000 at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: BioSorter2000<br><i>definition</i>: "The BioSorter2000 is a sorter manufactured by Union Biometrica. It includes analog electronics, 488/514 nm multi-line Argon-ion laser for fluorescence and a light stabilized 670 nm forward scatter laser for extinction and time-of-flight. It has filters and PMTs for 3 flourescent parameters and photodiodes for scatter, time of flight, and extinction. The flow cell is quartz cuvette. The BioSorterTM 2000 is an instrument for analyzing and sorting objects from 500 microns to 1.5 millimeters in diameter."<br></div></html><hr>
<a name='item192'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400026<br></b><b><i>CyFlow_Space</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The CyFlow Space is a sorte manufactured by Partec. Its specs are: 8optical parameters _ 6 colours:FSC, SSC, FL1-FL6 _ 3laser light sources: 200mW@488nm blue solid state laser 25mW@635nm red diode laser 50mW@405nm violet or 8mW@375nm ultraviolet diode laser _ultracompact desktop high end instrument. Parallel 16bit digital pulse processing. The CyFlow Space is a 6-Colour FCM System and Cell Sorter for Clinical Routine and Research."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.partec.de/products/cyflowspace.html, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: CyFlow Space at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: CyFlow_Space<br><i>definition</i>: "The CyFlow Space is a sorte manufactured by Partec. Its specs are: 8optical parameters _ 6 colours:FSC, SSC, FL1-FL6 _ 3laser light sources: 200mW@488nm blue solid state laser 25mW@635nm red diode laser 50mW@405nm violet or 8mW@375nm ultraviolet diode laser _ultracompact desktop high end instrument. Parallel 16bit digital pulse processing. The CyFlow Space is a 6-Colour FCM System and Cell Sorter for Clinical Routine and Research."<br></div></html><hr>
<a name='item193'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400075<br></b><b><i>MoFlo</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The MoFlo is a sorter manufactured by Dako Cytomation. It features digital electronics, the option to include several lasers (solid state 488 nm, 635 nm, and a 351 nm diode laser), and has the filtering and colelction capacity for up to 10 flouresecent parameters. The MoFlo is an instrument that can be used to both analyze quantitatively and collect cells in a flowing medium."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.dakousa.com/index/prod_search/prod_groups.htm?productareaid=16, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: MoFlo at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: MoFlo<br><i>definition</i>: "The MoFlo is a sorter manufactured by Dako Cytomation. It features digital electronics, the option to include several lasers (solid state 488 nm, 635 nm, and a 351 nm diode laser), and has the filtering and colelction capacity for up to 10 flouresecent parameters. The MoFlo is an instrument that can be used to both analyze quantitatively and collect cells in a flowing medium."<br></div></html><hr>
<a name='item194'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400041<br></b><b><i>FACSvantage</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The FACSvantage is sorter manufactured by Becton Dickinson. It has analog electronics, three lasers (several options are available), and the filters and PMTs to collect 6 flourescent parametersand side scatter, and a photodiode to collect forward scatter. The FACSvantage can be used to analyse, sort and collect cells."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.bdbiosciences.com/features/products/display_product.php?keyID=42, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: FACSvantage at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: FACSvantage<br><i>definition</i>: "The FACSvantage is sorter manufactured by Becton Dickinson. It has analog electronics, three lasers (several options are available), and the filters and PMTs to collect 6 flourescent parametersand side scatter, and a photodiode to collect forward scatter. The FACSvantage can be used to analyse, sort and collect cells."<br></div></html><hr>
<a name='item195'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400089<br></b><b><i>Reflection</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Reflection is a sorter manufactured by iCyte. It uses digital signal processing, and can be configured with lasers of the users choice from among these excitation wavelengths: 355, 405,488, 532, 635nm. It accommodates up to 48 traditional PMTs. Options include an acousto-optical tunable filter (AOTF) and 16 channel spectrometer (370 to 730 nm). The various detection components can be uniquely configured across multiple Highly Automated Parallel Sorting (HAPS) modules. The Reflection can be used to analyse, sort, and collect cells."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.i-cyt.com/reflection.htm, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Reflection at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: Reflection<br><i>definition</i>: "The Reflection is a sorter manufactured by iCyte. It uses digital signal processing, and can be configured with lasers of the users choice from among these excitation wavelengths: 355, 405,488, 532, 635nm. It accommodates up to 48 traditional PMTs. Options include an acousto-optical tunable filter (AOTF) and 16 channel spectrometer (370 to 730 nm). The various detection components can be uniquely configured across multiple Highly Automated Parallel Sorting (HAPS) modules. The Reflection can be used to analyse, sort, and collect cells."<br></div></html><hr>
<a name='item196'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400017<br></b><b><i>BioSorter500</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The BioSorter500 is a sorter manufactured by Union Biometrica. It includes analog electronics, 488/514 nm multi-line Argon-ion laser for fluorescence and a light stabilized 670 nm forward scatter laser for extinction and time-of-flight. It has filters and PMTs for 3 flourescent parameters and photodiodes for scatter, time of flight, and extinction. The flow cell is quartz cuvette. The BioSorterTM 500 is an instrument for analyzing and sorting objects from 100-250 microns in diameter and less than 2mm in length."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.unionbio.com/products/BioSorter.html, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: BioSorter 500 at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: BioSorter500<br><i>definition</i>: "The BioSorter500 is a sorter manufactured by Union Biometrica. It includes analog electronics, 488/514 nm multi-line Argon-ion laser for fluorescence and a light stabilized 670 nm forward scatter laser for extinction and time-of-flight. It has filters and PMTs for 3 flourescent parameters and photodiodes for scatter, time of flight, and extinction. The flow cell is quartz cuvette. The BioSorterTM 500 is an instrument for analyzing and sorting objects from 100-250 microns in diameter and less than 2mm in length."<br></div></html><hr>
<a name='item197'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400014<br></b><b><i>BioSorter1000</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The BioSorter1000 is a sorter manufactured by Union Biometrica. It includes analog electronics, 488/514 nm multi-line Argon-ion laser for fluorescence and a light stabilized 670 nm forward scatter laser for extinction and time-of-flight. It has filters and PMTs for 3 flourescent parameters and photodiodes for scatter, time of flight, and extinction. The flow cell is quartz cuvette. The BioSorterTM 1000 is an instrument for analyzing and sorting objects from 200-600 microns in diameter."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.unionbio.com/products/BioSorter.html, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: BioSorter 1000 at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: BioSorter1000<br><i>definition</i>: "The BioSorter1000 is a sorter manufactured by Union Biometrica. It includes analog electronics, 488/514 nm multi-line Argon-ion laser for fluorescence and a light stabilized 670 nm forward scatter laser for extinction and time-of-flight. It has filters and PMTs for 3 flourescent parameters and photodiodes for scatter, time of flight, and extinction. The flow cell is quartz cuvette. The BioSorterTM 1000 is an instrument for analyzing and sorting objects from 200-600 microns in diameter."<br></div></html><hr>
<a name='item198'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400016<br></b><b><i>BioSorter250</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The BioSorter250 is a sorter manufactured by Union Biometrica. It includes analog electronics, 488/514 nm multi-line Argon-ion laser for fluorescence and a light stabilized 670 nm forward scatter laser for extinction and time-of-flight. It has filters and PMTs for 3 flourescent parameters and photodiodes for scatter, time of flight, and extinction. The flow cell is quartz cuvette. The BioSorterTM 250 is an instrument for analyzing and sorting objects from 40-200 microns in diameter."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.unionbio.com/products/BioSorter.html, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: BioSorter 250 at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: BioSorter250<br><i>definition</i>: "The BioSorter250 is a sorter manufactured by Union Biometrica. It includes analog electronics, 488/514 nm multi-line Argon-ion laser for fluorescence and a light stabilized 670 nm forward scatter laser for extinction and time-of-flight. It has filters and PMTs for 3 flourescent parameters and photodiodes for scatter, time of flight, and extinction. The flow cell is quartz cuvette. The BioSorterTM 250 is an instrument for analyzing and sorting objects from 40-200 microns in diameter."<br></div></html><hr>
<a name='item199'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400040<br></b><b><i>FACSAria</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The FASCSAria is a sorter manufactured by Becton Dickinson. It features digital electronics, a solid state 488 nm blue laser, a solid state 407 nm violet laser, and a HeNe (633 nm) ion laser. The Aria has the filters and PMTs to collect side scatter and 9 flourescent parameters. The Aria has a photodiode detector for forward scatter collector. The flow cell is Quartz cuvette. The FACSAria is a sorter used to collect and analyse cells using up to 11 parameters."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.bdbiosciences.com/external_files/is/doc/mkt_lit/brochures/SJ-0003-00Aria.pdf, 2007-05-11"<br><i>curation_status</i>: graph_position_temporary<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: FACSAria at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: FACSAria<br><i>definition</i>: "The FASCSAria is a sorter manufactured by Becton Dickinson. It features digital electronics, a solid state 488 nm blue laser, a solid state 407 nm violet laser, and a HeNe (633 nm) ion laser. The Aria has the filters and PMTs to collect side scatter and 9 flourescent parameters. The Aria has a photodiode detector for forward scatter collector. The flow cell is Quartz cuvette. The FACSAria is a sorter used to collect and analyse cells using up to 11 parameters."<br></div></html><hr>
<a name='item200'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400056<br></b><b><i>Influx_Cell_Sorter</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The Influx Cell Sorter is a sorter manufactured by Cytopeia. It is a digital machine, with these laser options: Coherent 70, 90, 300 series water cooled lasers, solid state UV 355nm, Violet 408nm, Violet-Blue 479nm, Blue-488nm, Green-531nm, Red-635nm, Red-647nm. The sorting is multi-way, index, and proportional sorting. FIlters and PMTs are used for all parameters (including forward light scatter), and up to 12 PMTs can be included. The Influx cell sorter can be used to measure, sort, and collect ndividual cells."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.cytopeia.com/sorter.htm, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: Influx Cell Sorter at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: Influx_Cell_Sorter<br><i>definition</i>: "The Influx Cell Sorter is a sorter manufactured by Cytopeia. It is a digital machine, with these laser options: Coherent 70, 90, 300 series water cooled lasers, solid state UV 355nm, Violet 408nm, Violet-Blue 479nm, Blue-488nm, Green-531nm, Red-635nm, Red-647nm. The sorting is multi-way, index, and proportional sorting. FIlters and PMTs are used for all parameters (including forward light scatter), and up to 12 PMTs can be included. The Influx cell sorter can be used to measure, sort, and collect ndividual cells."<br></div></html><hr>
<a name='item201'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400068<br></b><b><i>LSR2</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"The LSR2 is a sorter manufactured by Becton Dickinson. It features digital electronics, a solid state 488 nm blue laser and optionally can also have any combination of solid state UV (355 nm) and violet (405 nm) lasers and the a HeNe (633 nm) ion laser. The LSR2 has the filters and PMTs to work with 13 flourescent parameters. The LSR II is one of the most common sorters in use."</div></i><font color="#000080">flow_cytometer_sorter</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://www.bdbiosciences.com/external_files/is/doc/mkt_lit/brochures/live/web_enabled/SJ-0142-00LSR2.pdf, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: LSR2 at TSRI Flow Cytometry Core Facility<br><i>preferred_term</i>: LSR2<br><i>definition</i>: "The LSR2 is a sorter manufactured by Becton Dickinson. It features digital electronics, a solid state 488 nm blue laser and optionally can also have any combination of solid state UV (355 nm) and violet (405 nm) lasers and the a HeNe (633 nm) ion laser. The LSR2 has the filters and PMTs to work with 13 flourescent parameters. The LSR II is one of the most common sorters in use."<br></div></html><hr>
<a name='item202'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400072<br></b><b><i>manufacturer_of_flow_cytometer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The manufacturer of flow cytometer term refers to the maker of the instrument. There are numerous manufacturers of flow cytometers.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: John Quinn<br><i>definition_citation</i>: "http://en.wikipedia.org/wiki/Manufacturer, 2007-05-11"<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: graph_position_temporary<br><i>example_of_usage</i>: Partec GmbH<br><i>preferred_term</i>: manufacturer_of_flow_cytometer<br><i>definition</i>: The manufacturer of flow cytometer term refers to the maker of the instrument. There are numerous manufacturers of flow cytometers.<br></div></html><hr>
<a name='item203'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_49<br></b><b><i>mass_spectrometry_instrument</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Frank says: this should be a mass_spectrometer and not have the suffix instrument</div></i><font color="#000080">instrument</font><br><i>definition</i>: A mass spectrometry instrument is an instrument used to separate or analyse molecules of a sample. definition_source: OBI.<br></div></html><hr>
<a name='item204'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400116<br></b><b><i>thermal_cycler</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument that is capable of repeatedly altering and maintaining specific temperatures for defined periods of time. Exact Synonym: thermocycler</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Piko" 96-well Thermal Cycler<br><i>preferred_term</i>: thermal_cycler<br><i>definition</i>: An instrument that is capable of repeatedly altering and maintaining specific temperatures for defined periods of time. Exact Synonym: thermocycler<br></div></html><hr>
<a name='item205'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400115<br></b><b><i>spectrophotometer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument that measures the transmittance of electromagnetic radiation of a defined wavelength by a substance.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Helios Gamma Spectrophotometer<br><i>preferred_term</i>: spectrophotometer<br><i>definition</i>: An instrument that measures the transmittance of electromagnetic radiation of a defined wavelength by a substance.<br></div></html><hr>
<a name='item206'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400118<br></b><b><i>vortexer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A vortexer is an instrument that mixes the contents in a liquid by creating a rotation of the liquid around its own center.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: VWR  Genie 2<br><i>preferred_term</i>: vortexer<br><i>definition</i>: A vortexer is an instrument that mixes the contents in a liquid by creating a rotation of the liquid around its own center.<br></div></html><hr>
<a name='item207'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400103<br></b><b><i>DNA_sequencer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument for the automated determination of the order of deoxynucleotides in deoxyribonucleic acid sequences.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: raw import<br><i>example_of_usage</i>: "ABI 377 DNA Sequencer, ABI 310 DNA Sequencer"<br><i>preferred_term</i>: DNA_sequencer<br><i>definition</i>: An instrument for the automated determination of the order of deoxynucleotides in deoxyribonucleic acid sequences.<br></div></html><hr>
<a name='item208'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400110<br></b><b><i>hybridization_chamber</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A container which is used to maintain constant contact of a liquid on an array.  This can be either a glass vial or slide.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: raw import<br><i>example_of_usage</i>: Glass Array Hybridization Cassette<br><i>preferred_term</i>: hybridization_chamber<br><i>definition</i>: A container which is used to maintain constant contact of a liquid on an array.  This can be either a glass vial or slide.<br></div></html><hr>
<a name='item209'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_48<br></b><b><i>?liquid_chromatography_instrument</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A liquid chromatography instrument is an instrument that collects data from the separation of colecules in liquid. definition_source: OBI.</div></i><font color="#000080">instrument</font><br><i>editor_note</i>: DS: Put under liquid_handler if justified.<br><i>definition</i>: A liquid chromatography instrument is an instrument that collects data from the separation of colecules in liquid. definition_source: OBI.<br></div></html><hr>
<a name='item210'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400121<br></b><b><i>crosslinking_instrument</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument that is able to chemically join two or more molecules.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: Stratalinker<br><i>preferred_term</i>: crosslinking_instrument<br><i>definition</i>: An instrument that is able to chemically join two or more molecules.<br></div></html><hr>
<a name='item211'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400112<br></b><b><i>?liquid_handler</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instrument used for automated liquid transfer and handling. Exact synonym:liquid_handling_instrument</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: DS: Is this class justified? Its a unnamed class. If so, put a fluidic_system and the fluidic_subsystem as subclasses.
TW: This is required by MO.
FG & DS: Capture as function.
All: Needs to be reviewed, according to query use case. If we keep it its kept as unnamed owl class.
The liquid handling class remains but as an undefined class with are unlikely to have children. It is expected that the reasoner would classifiy appropriate classes under this class that meet the have the liquid_handling function relation.<br><i>editor_note</i>: DS: Is this class justified? Its a unnamed class. If so, put a fluidic_system and the fluidic_subsystem as subclasses.
TW: This is required by MO.
FG: Or as function.<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw import<br><i>curation_status</i>: definition_incomplete<br><i>alternative_term</i>: liquid_handling_instrument<br><i>example_of_usage</i>: Beckman BioMek 2000<br><i>preferred_term</i>: ?liquid_handler<br><i>definition</i>: An instrument used for automated liquid transfer and handling. Exact synonym:liquid_handling_instrument<br></div></html><hr>
<a name='item212'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400111<br></b><b><i>hybridization_station</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An instrument which is used to maintain the temperature of one or more hybridization_chamber(s) at a defined, constant temperature."</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: raw import<br><i>example_of_usage</i>: Labnet Problot12<br><i>preferred_term</i>: hybridization_station<br><i>definition</i>: "An instrument which is used to maintain the temperature of one or more hybridization_chamber(s) at a defined, constant temperature."<br></div></html><hr>
<a name='item213'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400105<br></b><b><i>arrayer</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A programmable instrument which deposits biological material onto a substrate in a defined pattern.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: raw import<br><i>example_of_usage</i>: "BioRobotics Microgrid II TAS, Affymetrix GMS 417"<br><i>preferred_term</i>: arrayer<br><i>definition</i>: A programmable instrument which deposits biological material onto a substrate in a defined pattern.<br></div></html><hr>
<a name='item214'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_400106<br></b><b><i>centrifuge</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An instument which uses centrifugal force to separate mixtures of suspended material of different densities by rotating them in a container at high speed.</div></i><font color="#000080">instrument</font><br><i>definition_editor</i>: Trish Whetzel<br><i>definition_citation</i>: MO<br><i>curation_status</i>: definition_incomplete<br><i>curation_status</i>: raw import<br><i>example_of_usage</i>: Eppendorf Multipurpose Centrifuge<br><i>preferred_term</i>: centrifuge<br><i>definition</i>: An instument which uses centrifugal force to separate mixtures of suspended material of different densities by rotating them in a container at high speed.<br></div></html><hr>
<a name='item215'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_44<br></b><b><i>physical_document</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A physical document is an object serving as a record of information by means of symbolic marks. definition_source: OBI.</div></i><font color="#000080">artefact_object</font><br><i>editor_note</i>: BP: moved and renamed for BFO<br><i>editor_note</i>: information_content_entity "is_encoded_in" some physical_document in obi before split (040907).<br><i>definition</i>: A physical document is an object serving as a record of information by means of symbolic marks. definition_source: OBI.<br></div></html><hr>
<a name='item216'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_263<br></b><b><i>biomaterial_entity</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A biomaterial entity is an object of biological origin. definition_source: OBI.</div></i><font color="#000080">object</font><br><i>editor_note</i>: Domain of an object property "has_role_which_can_only_be_played_by_biomaterial_entity"  in obi before split, Range: ?role_which_can_only_be_played_by_biomaterial_entity (LF 040907).<br><i>branch</i>: Biomaterial<br><i>definition</i>: A biomaterial entity is an object of biological origin. definition_source: OBI.<br></div></html><hr>
<a name='item217'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000005<br></b><b><i>substance</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Any form of matter (material) that is homogeneous throughout, i.e. solid, liquid, gas</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Susanna<br><i>editor_note</i>: added as hierarchy term<br><i>definition_citation</i>: none<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: none<br><i>preferred_term</i>: substance<br><i>definition</i>: Any form of matter (material) that is homogeneous throughout, i.e. solid, liquid, gas<br></div></html><hr>
<a name='item218'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#toBeDiscussed<br></b><i></i><font color="#000080">biomaterial_entity</font><br></div></html><hr>
<a name='item219'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000035<br></b><b><i>individual</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Identifier or name of the individual organism from which the biomaterial was derived.</div></i><font color="#000080">toBeDiscussed</font><br><i>definition_editor</i>: Philippe Rocca-Serra<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. Jeremy Bentham<br><i>preferred_term</i>: individual<br><i>definition</i>: Identifier or name of the individual organism from which the biomaterial was derived.<br></div></html><hr>
<a name='item220'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000002<br></b><b><i>immortal cell</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">a cell derived from a multicellular organism that has the potential to replicate indefinately</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Bjoern Peters (added term)<br><i>editor_note</i>: added as hierarchy term<br><i>definition_citation</i>: none<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: a single Hela cell<br><i>preferred_term</i>: immortal cell<br><i>definition</i>: a cell derived from a multicellular organism that has the potential to replicate indefinately<br></div></html><hr>
<a name='item221'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000040<br></b><b><i>organism_status</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Organism_status is the stage premortem or postmortem at which the sample was processed for extraction of biomaterials.</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Tina Boussard<br><i>editor_note</i>: huh? Status does not sound like a material<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. premortem or postmortem<br><i>preferred_term</i>: organism_status<br><i>definition</i>: Organism_status is the stage premortem or postmortem at which the sample was processed for extraction of biomaterials.<br></div></html><hr>
<a name='item222'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000019<br></b><b><i>biosource_provider</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The resource or individual (e.g. company, hospital, commerical supplier) used to obtain or purchase the biomaterial. </div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g The Jackson Laboratory is a biosource_provider<br><i>preferred_term</i>: biosource_provider<br><i>definition</i>: The resource or individual (e.g. company, hospital, commerical supplier) used to obtain or purchase the biomaterial. <br></div></html><hr>
<a name='item223'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000037<br></b><b><i>molecular_mixture</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A molecular mixture is an unspecified collection of heterogeneous molecules, e.g. lipids, carbohydrate , nucleic acids and proteins</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Tina Boussard<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. the material obtained upon homogenizing a biomaterial<br><i>preferred_term</i>: molecular_mixture<br><i>definition</i>: A molecular mixture is an unspecified collection of heterogeneous molecules, e.g. lipids, carbohydrate , nucleic acids and proteins<br></div></html><hr>
<a name='item224'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000004<br></b><b><i>liquid chemical solution</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">a liquid solvent containing one or more solutes</div></i><font color="#000080">molecular_mixture</font><br><i>definition_editor</i>: Bjoern Peters (added term)<br><i>editor_note</i>: added as hierarchy term<br><i>definition_citation</i>: none<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: PBS, RPMI, etc.<br><i>preferred_term</i>: liquid chemical solution<br><i>definition</i>: a liquid solvent containing one or more solutes<br></div></html><hr>
<a name='item225'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000041<br></b><b><i>PBS</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">PBS is a commonly used buffer</div></i><font color="#000080">liquid&middot;chemical&middot;solution</font><br><i>definition_editor</i>: Tina Boussard<br><i>definition_citation</i>: MIACA<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: none<br><i>preferred_term</i>: PBS<br><i>definition</i>: PBS is a commonly used buffer<br></div></html><hr>
<a name='item226'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000047<br></b><b><i>suspension</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Substance floating in a liquid medium</div></i><font color="#000080">molecular_mixture</font><br><i>definition_editor</i>: Susanna<br><i>definition_citation</i>: PSI<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: none<br><i>preferred_term</i>: suspension<br><i>definition</i>: Substance floating in a liquid medium<br></div></html><hr>
<a name='item227'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000031<br></b><b><i>emulsion</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A mixture of immiscible (unblendable) substances, that are finely dispersed and not separate through the action of gravity alone</div></i><font color="#000080">molecular_mixture</font><br><i>definition_editor</i>: Susanna<br><i>definition_citation</i>: PSI<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: lipid emulsion, oil in water emulsion, perfluoro-chemical emulsion, photographic emulsion<br><i>preferred_term</i>: emulsion<br><i>definition</i>: A mixture of immiscible (unblendable) substances, that are finely dispersed and not separate through the action of gravity alone<br></div></html><hr>
<a name='item228'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000045<br></b><b><i>solution</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A homogeneous mixture of two or more substances</div></i><font color="#000080">molecular_mixture</font><br><i>definition_editor</i>: Susanna<br><i>definition_citation</i>: PSI<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: none<br><i>preferred_term</i>: solution<br><i>definition</i>: A homogeneous mixture of two or more substances<br></div></html><hr>
<a name='item229'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#NaturalBiomaterialEntity<br></b><i></i><font color="#000080">biomaterial_entity</font><br></div></html><hr>
<a name='item230'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_251<br></b><b><i>organism</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An organism is biomaterial entity that is an individual living system, such as animal, plant or micro-organism, that is capable of reproduction, growth and maintenance. definition_source: OBI.</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>editor_note</i>: organism "has_role_which_can_only_be_played_by_biomaterial_entity" some specimen_role, organism "has_role" some investigation_object_role in obi before split (LF 040907).<br><i>editor_note</i>: examples: Archaea, Bacteria,Eukaryota,Viroids, Viruses rem1: OBI should link to an external resource (e.g. NCBI Taxonomy) Altdef: an organism is a biomaterial entity which is self-contained<br><i>definition</i>: An organism is biomaterial entity that is an individual living system, such as animal, plant or micro-organism, that is capable of reproduction, growth and maintenance. definition_source: OBI.<br></div></html><hr>
<a name='item231'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_123<br></b><b><i>eukaryote</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A eukaryote is an organism that contains specialized organelles in the cytoplasm, a membrane-bound nucleus enclosing genetic material organized into chromosomes, and an elaborate system of division by mitosis or meiosis. definition_source: OBI.</div></i><font color="#000080">organism</font><br><i>definition</i>: A eukaryote is an organism that contains specialized organelles in the cytoplasm, a membrane-bound nucleus enclosing genetic material organized into chromosomes, and an elaborate system of division by mitosis or meiosis. definition_source: OBI.<br></div></html><hr>
<a name='item232'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000011<br></b><b><i>virus</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Organism with a nucleic acid genome that must replicate in a living cell, as they cannot carry out metabolic processes on their own.</div></i><font color="#000080">organism</font><br><i>definition_editor</i>: Maura Gasparetto<br><i>editor_note</i>: merged Maura Gasparetto _ Helen Parkinson<br><i>definition_citation</i>: Immunobiology by Janeway & Travers 1997<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: HPV<br><i>preferred_term</i>: virus<br><i>definition</i>: Organism with a nucleic acid genome that must replicate in a living cell, as they cannot carry out metabolic processes on their own.<br></div></html><hr>
<a name='item233'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_124<br></b><b><i>viroid</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A viroid is an organism that consists of nucleic acid only and is not associated with any protein. The RNA is not translated, but replicated in the host cell. definition_source: OBI.</div></i><font color="#000080">organism</font><br><i>definition</i>: A viroid is an organism that consists of nucleic acid only and is not associated with any protein. The RNA is not translated, but replicated in the host cell. definition_source: OBI.<br></div></html><hr>
<a name='item234'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_122<br></b><b><i>bacterium</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A bacterium is an organism that is a single-celled prokaryote. definition_source: OBI.</div></i><font color="#000080">organism</font><br><i>definition</i>: A bacterium is an organism that is a single-celled prokaryote. definition_source: OBI.<br></div></html><hr>
<a name='item235'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_125<br></b><b><i>virus</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A virus is an organism that infects cells in biological organisms. Viruses are obligate intracellular parasites; they can reproduce only by invading and taking over other cells as they lack the cellular machinery for self reproduction. definition_source:en.wikipedia.org/wiki/Virus</div></i><font color="#000080">organism</font><br><i>definition</i>: A virus is an organism that infects cells in biological organisms. Viruses are obligate intracellular parasites; they can reproduce only by invading and taking over other cells as they lack the cellular machinery for self reproduction. definition_source:en.wikipedia.org/wiki/Virus<br></div></html><hr>
<a name='item236'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_121<br></b><b><i>archae_bacterium</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An archae bacterium is an organism that is a single-celled prokaryote genetically distinct from bacteria. definition_source: OBI.</div></i><font color="#000080">organism</font><br><i>definition</i>: An archae bacterium is an organism that is a single-celled prokaryote genetically distinct from bacteria. definition_source: OBI.<br></div></html><hr>
<a name='item237'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000043<br></b><b><i>population</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A population is the total number of inhabitants constituting a particular race, class, or group in a specified area.</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>definition_editor</i>: Tina Boussard<br><i>definition_citation</i>: ArrayExpress<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. Japanese in Tokyo Japan, or a cell population<br><i>preferred_term</i>: population<br><i>definition</i>: A population is the total number of inhabitants constituting a particular race, class, or group in a specified area.<br></div></html><hr>
<a name='item238'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000025<br></b><b><i>cohort</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cohort is a group of individuals, identified by a common characteristic, who are studied over a period of time as part of an epidemiological investigation.</div></i><font color="#000080">population</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: Tina Boussard/PharmGBK<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. A group of individuals with the same disease<br><i>preferred_term</i>: cohort<br><i>definition</i>: A cohort is a group of individuals, identified by a common characteristic, who are studied over a period of time as part of an epidemiological investigation.<br></div></html><hr>
<a name='item239'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000044<br></b><b><i>serotype</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A serotype is a subdivision of a species of microorganism, eg, a bacteria, based upon its particular antigens.</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>definition_editor</i>: Tina Boussard<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. Streptococcus pyogenes M3 serotype<br><i>preferred_term</i>: serotype<br><i>definition</i>: A serotype is a subdivision of a species of microorganism, eg, a bacteria, based upon its particular antigens.<br></div></html><hr>
<a name='item240'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000046<br></b><b><i>strain</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A strain is the collective descendants of a common ancestor; synonyms: race, stock, line, or breed.</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>definition_editor</i>: Tina Boussard<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. Canton S is a strain of Drosophila<br><i>preferred_term</i>: strain<br><i>definition</i>: A strain is the collective descendants of a common ancestor; synonyms: race, stock, line, or breed.<br></div></html><hr>
<a name='item241'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000007<br></b><b><i>unicellular organism</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Organism made up of a single cell</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>definition_editor</i>: Bjoern Peters (added term)<br><i>editor_note</i>: added as hierarchy term<br><i>definition_citation</i>: none<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E Coli<br><i>preferred_term</i>: unicellular organism<br><i>definition</i>: Organism made up of a single cell<br></div></html><hr>
<a name='item242'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000030<br></b><b><i>ecotype</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An ecotype of a species is subgroup of members of that species characterized by the ecological surroundings it inhabits.  e.g. the A. thaliana Ecotype Ler. Source wikipedia</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. the A. thaliana Ecotype Ler<br><i>preferred_term</i>: ecotype<br><i>definition</i>: An ecotype of a species is subgroup of members of that species characterized by the ecological surroundings it inhabits.  e.g. the A. thaliana Ecotype Ler. Source wikipedia<br></div></html><hr>
<a name='item243'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000022<br></b><b><i>cell</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The smallest structural unit of living organisms that is able to grow and reproduce independently.</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>definition_editor</i>: Compiled by Helen Parkinson for Transcriptomics thanks to Adam Witney<br><i>definition_citation</i>: North Carolina Biotechnology Center<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: Human lymphocyte<br><i>preferred_term</i>: cell<br><i>definition</i>: The smallest structural unit of living organisms that is able to grow and reproduce independently.<br></div></html><hr>
<a name='item244'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000000<br></b><b><i>Epstein_Barr_virus_transformed_b_cell</i></b><i><br></i><font color="#000080">cell</font><br></div></html><hr>
<a name='item245'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_9999999<br></b><b><i>cytotoxic_t-cell</i></b><i><br></i><font color="#000080">cell</font><br><i>curation_status</i>: graph position temporary<br></div></html><hr>
<a name='item246'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000038<br></b><b><i>organism</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An organism is a living complex adaptive system of organs that influence each other in such a way that they function in some way as a stable whole. Synonym, whole_organism</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>definition_editor</i>: Tina Boussard<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. a mouse<br><i>preferred_term</i>: organism<br><i>definition</i>: An organism is a living complex adaptive system of organs that influence each other in such a way that they function in some way as a stable whole. Synonym, whole_organism<br></div></html><hr>
<a name='item247'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_126<br></b><b><i>biological_macromolecule</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A biological macromolecule is an anatomical structure which consists of ordered aggregates of any of the following molecules: nucleotides, amino acids, fatty acids and sugars. definition_source: FMA</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>editor_note</i>: example: collagen, DNA, neurotransmitter, troponin. refer to FMA definition (check for ID)<br><i>definition</i>: A biological macromolecule is an anatomical structure which consists of ordered aggregates of any of the following molecules: nucleotides, amino acids, fatty acids and sugars. definition_source: FMA<br></div></html><hr>
<a name='item248'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000039<br></b><b><i>organism_part</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An organism_part is a part of organism's anatomy or substance arising from an organism from which the biomaterial was derived, excludes cells. E.g. tissue, organ, system, sperm, blood or body location (arm).</div></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>definition_editor</i>: Tina Boussard<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. tissue, organ, system, sperm, blood or body location (arm).<br><i>preferred_term</i>: organism_part<br><i>definition</i>: An organism_part is a part of organism's anatomy or substance arising from an organism from which the biomaterial was derived, excludes cells. E.g. tissue, organ, system, sperm, blood or body location (arm).<br></div></html><hr>
<a name='item249'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000018<br></b><b><i>serum</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Fluid component of clotted blood.</div></i><font color="#000080">organism_part</font><br><i>definition_editor</i>: Maura Gasparetto<br><i>editor_note</i>: This is in FMA<br><i>definition_citation</i>: Immunobiology by Janeway & Travers 1997<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: Fetal Bovine Serum<br><i>preferred_term</i>: serum<br><i>definition</i>: Fluid component of clotted blood.<br></div></html><hr>
<a name='item250'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000017<br></b><b><i>plasma</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Fluid componenet of blood containing water, electrolytes, and the plasma proteins. </div></i><font color="#000080">organism_part</font><br><i>definition_editor</i>: Maura Gasparetto<br><i>editor_note</i>: This is in FMA<br><i>definition_citation</i>: Immunobiology by Janeway & Travers 1997<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: Albumin, Calcium, Sodium<br><i>preferred_term</i>: plasma<br><i>definition</i>: Fluid componenet of blood containing water, electrolytes, and the plasma proteins. <br></div></html><hr>
<a name='item251'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_280<br></b><b><i>in_vivo_biomaterial</i></b><i><br></i><font color="#000080">NaturalBiomaterialEntity</font><br><i>editor_note</i>: This is a place holder, indicating that we have trouble dealing with the state / site of biomaterials, i.e. in vivo, ex vivo, in vitro<br></div></html><hr>
<a name='item252'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_133<br></b><b><i>entity_of_organismal_origin</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An entity of organismal origin is a biomaterial entity which is obtained from an organism. definition_source: OBI.</div></i><font color="#000080">biomaterial_entity</font><br><i>definition</i>: An entity of organismal origin is a biomaterial entity which is obtained from an organism. definition_source: OBI.<br></div></html><hr>
<a name='item253'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_128<br></b><b><i>cell_component</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cell component is an entity of organismal origin which has a definable shape, bounded predominantly by bonafide boundaries. definition_source: FMA</div></i><font color="#000080">entity_of_organismal_origin</font><br><i>definition</i>: A cell component is an entity of organismal origin which has a definable shape, bounded predominantly by bonafide boundaries. definition_source: FMA<br></div></html><hr>
<a name='item254'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_261<br></b><b><i>multicellular_entity_of_organismal_origin</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A multicellular entity of organismal origin is an entity of organismal origin that is a heterogeneous biomaterial entity which is either part of an organism or which is obtained from an organism by *material_transformation process* e.g. 'extraction'. definition_source: OBI.</div></i><font color="#000080">entity_of_organismal_origin</font><br><i>editor_note</i>: multicellular_entity_of_organismal_origin "has_role" some ex-vivo_state in obi before split (LF 040907).<br><i>editor_note</i>: rem1: OBI should link to an external resource (e.g. Common Anatomy Reference Ontology CARO)<br><i>definition</i>: A multicellular entity of organismal origin is an entity of organismal origin that is a heterogeneous biomaterial entity which is either part of an organism or which is obtained from an organism by *material_transformation process* e.g. 'extraction'. definition_source: OBI.<br></div></html><hr>
<a name='item255'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_127<br></b><b><i>cell_of_organismal_origin</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cell of organismal origin is an entity of organismal origin that contains nuclear and cytoplasmic material enclosed by a semipermeable membrane. definition_source: OBI.</div></i><font color="#000080">entity_of_organismal_origin</font><br><i>editor_note</i>: rem1:link to Cell Type Ontology (CO)<br><i>definition</i>: A cell of organismal origin is an entity of organismal origin that contains nuclear and cytoplasmic material enclosed by a semipermeable membrane. definition_source: OBI.<br></div></html><hr>
<a name='item256'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_129<br></b><b><i>cell_line</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cell line is an entity of organismal origin that represents generations of a primary culture. definition_source: OBI.</div></i><font color="#000080">entity_of_organismal_origin</font><br><i>definition</i>: A cell line is an entity of organismal origin that represents generations of a primary culture. definition_source: OBI.<br></div></html><hr>
<a name='item257'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_131<br></b><b><i>immortalized_cell_line</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An immortalized cell line is a cell line that is able to replicate indefinitely. definition_source: OBI.</div></i><font color="#000080">cell_line</font><br><i>definition</i>: An immortalized cell line is a cell line that is able to replicate indefinitely. definition_source: OBI.<br></div></html><hr>
<a name='item258'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000014<br></b><b><i>fragment derived from protein</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is a biomaterial which is derived from a  protein</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Bjoern Peters<br><i>editor_note</i>: original term: protein fragment split into synthetic polypeptide and fragment derived from protein<br><i>definition_citation</i>: IEDB<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: the peptide with sequence SIINFEKL, which is eluted from a cell expressing hen egg lysozyme<br><i>preferred_term</i>: fragment derived from protein<br><i>definition</i>: is a biomaterial which is derived from a  protein<br></div></html><hr>
<a name='item259'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#xenograft<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A graft of tissue or cells from one species to an unlike species (or genus or family)</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: <br><i>definition</i>: A graft of tissue or cells from one species to an unlike species (or genus or family)<br></div></html><hr>
<a name='item260'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_336<br></b><b><i>ChEBI_objects</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">objects of chemical or biological interest</div></i><font color="#000080">biomaterial_entity</font><br><i>editor_note</i>: Import of ChEBI ontology<br><i>definition</i>: objects of chemical or biological interest<br></div></html><hr>
<a name='item261'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000009<br></b><b><i>chromium_51</i></b><i><br></i><font color="#000080">ChEBI_objects</font><br></div></html><hr>
<a name='item262'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000006<br></b><b><i>Triton X-100</i></b><i><br></i><font color="#000080">ChEBI_objects</font><br></div></html><hr>
<a name='item263'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000012<br></b><b><i>artificially engineered biomaterial</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Laboratory created combination of discrete biomaterials not usually found in nature. Created with the purpose of facilitating laboratory research.</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Bjoern Peters<br><i>editor_note</i>: merged with Construct<br><i>definition_citation</i>: IEDB<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: chimeric mAb, chimeric mouse, GST-Rho protein<br><i>preferred_term</i>: artificially engineered biomaterial<br><i>definition</i>: Laboratory created combination of discrete biomaterials not usually found in nature. Created with the purpose of facilitating laboratory research.<br></div></html><hr>
<a name='item264'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000026<br></b><b><i>cultivar</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cultivar is a plant variety obtained in agriculture or horticulture clearly distinguishable from others by one or more characteristics and which when reproduced retain those characteristics.</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. Solanum tuberosum has a virus resistant cultivar Alcmaria<br><i>preferred_term</i>: cultivar<br><i>definition</i>: A cultivar is a plant variety obtained in agriculture or horticulture clearly distinguishable from others by one or more characteristics and which when reproduced retain those characteristics.<br></div></html><hr>
<a name='item265'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#chimera<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An organism which contains cells or tissues with a different genotype</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition</i>: An organism which contains cells or tissues with a different genotype<br></div></html><hr>
<a name='item266'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000001<br></b><b><i>cell derived from multicellular organism</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cell derived from a multicellular organism</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Bjoern Peters (added term)<br><i>editor_note</i>: added as hierarchy term<br><i>definition_citation</i>: none<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: lymphocyte<br><i>preferred_term</i>: cell derived from multicellular organism<br><i>definition</i>: A cell derived from a multicellular organism<br></div></html><hr>
<a name='item267'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#ExperimentalBiomateralEntity<br></b><i></i><font color="#000080">biomaterial_entity</font><br></div></html><hr>
<a name='item268'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000016<br></b><b><i>organ_section</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An organ_section is a portion of an organ removed from the context of the organ</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Helen Parkinson<br><i>editor_note</i>: section needs to be added as process<br><i>definition_citation</i>: CEBS<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. A liver slice used in a perfusion experiment<br><i>preferred_term</i>: organ_section<br><i>definition</i>: An organ_section is a portion of an organ removed from the context of the organ<br></div></html><hr>
<a name='item269'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000029<br></b><b><i>display_library</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">group of  biomaterials in which a mixture of genes or gene fragments is expressed and can be individually selected and amplified.</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Bjoern Peters<br><i>definition_citation</i>: IEDB<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: Phage display library encoding fragments of human antibodies. m-rna library encoding for 9-mer peptides<br><i>preferred_term</i>: display_library<br><i>definition</i>: group of  biomaterials in which a mixture of genes or gene fragments is expressed and can be individually selected and amplified.<br></div></html><hr>
<a name='item270'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000021<br></b><b><i>cdna_library</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Mixed population of cDNAs (complementaryDNA) made from mRNA from a defined source, usually a specific cell type. This term should be associated only to nucleic acid interactors not to their proteins product. For instance in 2h screening use living cells (MI:0349) as sample process.</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Luisa Montecchi<br><i>definition_citation</i>: PSI<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: collection of cDNA derived from mouse splenocytes. PMID:6110205<br><i>preferred_term</i>: cdna_library<br><i>definition</i>: Mixed population of cDNAs (complementaryDNA) made from mRNA from a defined source, usually a specific cell type. This term should be associated only to nucleic acid interactors not to their proteins product. For instance in 2h screening use living cells (MI:0349) as sample process.<br></div></html><hr>
<a name='item271'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI/Biomaterial.owl#OBI_1000024<br></b><b><i>cell_pellet</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cell pellet is an aggegration of cells produced by the application of centrifugal force</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: CEBS<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. The product of a centriguation process on a cell culture<br><i>preferred_term</i>: cell_pellet<br><i>definition</i>: A cell pellet is an aggegration of cells produced by the application of centrifugal force<br></div></html><hr>
<a name='item272'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000000<br></b><b><i>cell culture</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">population of cells and cell culture medium maintained in vitro. Can be homogeneous (monoclonal),  heterogenous (oligoclonal, polyclonal)</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Bjoern Peters (added term)<br><i>editor_note</i>: added as hierarchy term<br><i>definition_citation</i>: none<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: Jurkat cell line in RPMI w/ 10% FCS. PBMCs were purified from blood sample and put into tissue culture media.<br><i>preferred_term</i>: cell culture<br><i>definition</i>: population of cells and cell culture medium maintained in vitro. Can be homogeneous (monoclonal),  heterogenous (oligoclonal, polyclonal)<br></div></html><hr>
<a name='item273'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000023<br></b><b><i>cell_culture_supernatant</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Supernatant of a cell culture.  Contains media, supplements, and secreted products of the cells.</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Bjoern Peters<br><i>definition_citation</i>: IEDB<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: solution containing RPMI, fetal calf sera, antibiotics, and monoclonal antibody UCHT1.<br><i>preferred_term</i>: cell_culture_supernatant<br><i>definition</i>: Supernatant of a cell culture.  Contains media, supplements, and secreted products of the cells.<br></div></html><hr>
<a name='item274'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000036<br></b><b><i>lysate</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A mixture (collection) of cell components created by rupturing of the cell wall</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Susanna<br><i>definition_citation</i>: PSI<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: cell-free lysate, whole lysate, nuclear lysate<br><i>preferred_term</i>: lysate<br><i>definition</i>: A mixture (collection) of cell components created by rupturing of the cell wall<br></div></html><hr>
<a name='item275'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000003<br></b><b><i>library</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">collection of biomaterials engineered to identify qualities of a subset of its members?</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Bjoern Peters<br><i>editor_note</i>: added as hierarchy term<br><i>definition_citation</i>: none<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: cdna_library, phage display library<br><i>preferred_term</i>: library<br><i>definition</i>: collection of biomaterials engineered to identify qualities of a subset of its members?<br></div></html><hr>
<a name='item276'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000049<br></b><b><i>whole_mount</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A whole_mount is a histological preparation where the whole specimen is mounted or spread on the glass slide</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: OBI<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. a whole mouse embryo labelled with anti-sonic hedgehog<br><i>preferred_term</i>: whole_mount<br><i>definition</i>: A whole_mount is a histological preparation where the whole specimen is mounted or spread on the glass slide<br></div></html><hr>
<a name='item277'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000048<br></b><b><i>transgenic_organism</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is an organism which has been modified to express a gene not normally part of it</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Bjoern Peters<br><i>definition_citation</i>: IEDB<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: HLA-A*0201 transgenic mice, Vaccinia virus expressing the LCMV gp protein<br><i>preferred_term</i>: transgenic_organism<br><i>definition</i>: is an organism which has been modified to express a gene not normally part of it<br></div></html><hr>
<a name='item278'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000020<br></b><b><i>Bronchial_Aveolar_Lavage [BAL]</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Group of biomaterials present in the bronchial aveolar space of an organism which are collected through lavage including the reagents used to for the lavage process, organisms, cells, and cellular secretions present in the bronchial aveolar space.</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Bjoern Peters<br><i>definition_citation</i>: IEDB<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: solution containing lung derived T cells, eosinophils, and TNFa.<br><i>preferred_term</i>: Bronchial_Aveolar_Lavage [BAL]<br><i>definition</i>: Group of biomaterials present in the bronchial aveolar space of an organism which are collected through lavage including the reagents used to for the lavage process, organisms, cells, and cellular secretions present in the bronchial aveolar space.<br></div></html><hr>
<a name='item279'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000015<br></b><b><i>synthetic polypeptide</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">artificially engineered biomaterial consisting of a chain of amino acids</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Bjoern Peters<br><i>editor_note</i>: original term: protein fragment split into synthetic polypeptide and fragment derived from protein<br><i>definition_citation</i>: IEDB<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: the synthesized peptide SIINFEKL which also occurs in hen-egg lysozyme<br><i>preferred_term</i>: synthetic polypeptide<br><i>definition</i>: artificially engineered biomaterial consisting of a chain of amino acids<br></div></html><hr>
<a name='item280'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000042<br></b><b><i>peptide_construct</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is a biomaterial containing a number of peptides conjugated together in a non-linear fashion</div></i><font color="#000080">ExperimentalBiomateralEntity</font><br><i>definition_editor</i>: Bjoern Peters<br><i>definition_citation</i>: IEDB<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: Multiple Antigenic Peptide (MAP) construct containing T and B cell epitopes from p. falciparum congugated to a central glycin<br><i>preferred_term</i>: peptide_construct<br><i>definition</i>: is a biomaterial containing a number of peptides conjugated together in a non-linear fashion<br></div></html><hr>
<a name='item281'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000027<br></b><b><i>differentiation_state</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The differentiation_state is a descriptor of a cell's properties at some defined point in the differentiation process recognisable by some observable characteristic such as morphology.</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: ArrayExpress<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. prostate epithelial transit amplifying cells<br><i>preferred_term</i>: differentiation_state<br><i>definition</i>: The differentiation_state is a descriptor of a cell's properties at some defined point in the differentiation process recognisable by some observable characteristic such as morphology.<br></div></html><hr>
<a name='item282'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#geneticInformationContent<br></b><i></i><font color="#000080">biomaterial_entity</font><br></div></html><hr>
<a name='item283'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000028<br></b><b><i>diplotype</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A diplotype is the association of two alleles of a haplotype (or simple gene allele) in a diploid organism. Synonym haplotype pair</div></i><font color="#000080">geneticInformationContent</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: Tina Boussard/PharmGBK<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g TCCG/CCTG at the ADH1B<br><i>preferred_term</i>: diplotype<br><i>definition</i>: A diplotype is the association of two alleles of a haplotype (or simple gene allele) in a diploid organism. Synonym haplotype pair<br></div></html><hr>
<a name='item284'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000033<br></b><b><i>genotype</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The total sum of the genetic information of an organism that is known and relevant to the experiment being performed, including chromosomal, plasmid, viral or other genetic material which has been introduced into the organism either prior to or during the experiment.</div></i><font color="#000080">geneticInformationContent</font><br><i>definition_editor</i>: Philippe Rocca-Serra<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. p53 codon 72 genotype<br><i>preferred_term</i>: genotype<br><i>definition</i>: The total sum of the genetic information of an organism that is known and relevant to the experiment being performed, including chromosomal, plasmid, viral or other genetic material which has been introduced into the organism either prior to or during the experiment.<br></div></html><hr>
<a name='item285'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000034<br></b><b><i>haplotype</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A haplotype, or ""haploid genotype"", is a set of allelic states found at neighbouring loci in a chromosome, as inherited from a parent. Haplotype can be determined also as the genetic constitution of an individual chromosome. In the case of diploid organisms such as humans, the haplotype will contain one member of the pair of alleles for each site.(adapted from en.wikipedia.org/wiki/Haplotype)</div></i><font color="#000080">geneticInformationContent</font><br><i>definition_editor</i>: Philippe Rocca-Serra<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. he AMH 1.15 haplotype is also referred to as the Atlantic Modal Cluster or AMC<br><i>preferred_term</i>: haplotype<br><i>definition</i>: A haplotype, or ""haploid genotype"", is a set of allelic states found at neighbouring loci in a chromosome, as inherited from a parent. Haplotype can be determined also as the genetic constitution of an individual chromosome. In the case of diploid organisms such as humans, the haplotype will contain one member of the pair of alleles for each site.(adapted from en.wikipedia.org/wiki/Haplotype)<br></div></html><hr>
<a name='item286'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000008<br></b><b><i>allele</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An allele is one of two or more alternative forms of a gene or marker sequence at a given locus and differing from other alleles at one or more mutational sites based on phenotype or sequence. E</div></i><font color="#000080">geneticInformationContent</font><br><i>definition_editor</i>: Helen Parkinson<br><i>editor_note</i>: bin under common 'Genotype' Placeholder. Do we also want recessive_allele etc? as Children or are these qualities. HP will check with PATO<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: .g. XPD 751Gln allele<br><i>preferred_term</i>: allele<br><i>definition</i>: An allele is one of two or more alternative forms of a gene or marker sequence at a given locus and differing from other alleles at one or more mutational sites based on phenotype or sequence. E<br></div></html><hr>
<a name='item287'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000032<br></b><b><i>genetic_modification</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The genetic modification introduced into the organism from which the biomaterial was derived. Examples of genetic modification include specification of a transgene or the gene knocked-out or details of transient transfection.</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Philippe Rocca-Serra<br><i>definition_citation</i>: MO<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: E.g. a deletion<br><i>preferred_term</i>: genetic_modification<br><i>definition</i>: The genetic modification introduced into the organism from which the biomaterial was derived. Examples of genetic modification include specification of a transgene or the gene knocked-out or details of transient transfection.<br></div></html><hr>
<a name='item288'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_1000006<br></b><b><i>supernatant</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Liquid portion of complex biomaterial resulting from a separation from a solid portion</div></i><font color="#000080">biomaterial_entity</font><br><i>definition_editor</i>: Bjoern Peters (added term)<br><i>editor_note</i>: added as hierarchy term<br><i>definition_citation</i>: none<br><i>curation_status</i>: raw_import<br><i>example_of_usage</i>: none<br><i>preferred_term</i>: supernatant<br><i>definition</i>: Liquid portion of complex biomaterial resulting from a separation from a solid portion<br></div></html><hr>
<a name='item289'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#Site<br></b><b><i>site</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Examples: a city, a nasal cavity, a blood vein, an environment, sinuses, canals, ventricles, the lumen of the gastrointestinal tract, the location of the battle of Austerlitz, the interior of the aorta, the room in which one is located, the top of one's desk including the things that are on it and the atmosphere surrounding them.</div></i><font color="#000080">(<i>and</i>&nbsp;independent_continuant<br>&nbsp;(<i>disjoint-with</i>&nbsp;object_aggregate&nbsp;fiat_object_part&nbsp;object&nbsp;object_boundary))</font><br></div></html><hr>
<a name='item290'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_321<br></b><b><i>generically_dependent_continuant</i></b><i><br></i><font color="#000080">continuant</font><br><i>branch</i>: generically_dependent_continuant<br></div></html><hr>
<a name='item291'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_295<br></b><b><i>non-realizable_information_entity</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Example: Manual for Microarray Platform, Code for Microsoft Word, Description of physical appearance of a mouse in an experiment, HLA typing information for donors enrolled in a study</div></i><font color="#000080">generically_dependent_continuant</font><br><i>editor_note</i>: Created as a holder class for specification and instance_level_information. Several classes below have been slightly modified to accomodate their new placement<br><i>definition</i>: an information entity is a dependent_continuant which conveys meaning and can be documented and communicated.<br></div></html><hr>
<a name='item292'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_342<br></b><b><i>information_content_entity</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">information_content_entity</div></i><font color="#000080">non-realizable_information_entity</font><br><i>editor_note</i>: information_content_entity "is_encoded_in" some digital_entity in obi before split (040907). information_content_entity "is_encoded_in" some physical_document in obi before split (040907).<br></div></html><hr>
<a name='item293'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_315<br></b><b><i>data_display_element</i></b><i><br></i><font color="#000080">information_content_entity</font><br></div></html><hr>
<a name='item294'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_298<br></b><b><i>tabular_data_report</i></b><i><br></i><font color="#000080">data_display_element</font><br></div></html><hr>
<a name='item295'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_293<br></b><b><i>figure</i></b><i><br></i><font color="#000080">data_display_element</font><br></div></html><hr>
<a name='item296'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_17<br></b><b><i>cardinal_part_of_value</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cardinal part of value is a dependent continuant which is a number or quality, or group of numbers or qualities, that only exists in relation to what it describes, e.g., qualitative, quantitative. definition_source: OBI.</div></i><font color="#000080">information_content_entity</font><br><i>editor_note</i>: BFO problem<br><i>definition</i>: A cardinal part of value is a dependent continuant which is a number or quality, or group of numbers or qualities, that only exists in relation to what it describes, e.g., qualitative, quantitative. definition_source: OBI.<br></div></html><hr>
<a name='item297'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_77<br></b><b><i>cardinal_part_of_qualitative_value</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cardinal part of qualitative value is a cardinal part of value which is a non-numerical value or group of values that only exists in relation to what it describes, e.g. quality, character. definition_source: OBI.</div></i><font color="#000080">cardinal_part_of_value</font><br><i>editor_note</i>: rem1: Revisit issue of what a qualitative value should include: intensity? rem2: this should go to deleted classes but for now kept here as a reminder until we have identified a mechanism to reference out to external ontologies<br><i>definition</i>: A cardinal part of qualitative value is a cardinal part of value which is a non-numerical value or group of values that only exists in relation to what it describes, e.g. quality, character. definition_source: OBI.<br></div></html><hr>
<a name='item298'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_80<br></b><b><i>intensity</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Intensity is a cardinal part of qualitative value which measures the amount of energy transmitted, e.g. sonic, electromagnetic, chromatic. definition_source: OBI.</div></i><font color="#000080">cardinal_part_of_qualitative_value</font><br><i>editor_note</i>: rem1: May want to reconsider having intensity defined as qualitative, as opposed to quantitative. rem2: not just intensity but degree eg 'severity', 'hue', 'saturation'<br><i>definition</i>: Intensity is a cardinal part of qualitative value which measures the amount of energy transmitted, e.g. sonic, electromagnetic, chromatic. definition_source: OBI.<br></div></html><hr>
<a name='item299'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_78<br></b><b><i>cardinal_part_of_quantitative_value</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A cardinal part of quantitative value is a cardinal part of value which is a numerical value or group of numerical values that only exists in relation to what it describes, e.g. a confidence interval, numerical designation, unit. definition_source: OBI.</div></i><font color="#000080">cardinal_part_of_value</font><br><i>definition</i>: A cardinal part of quantitative value is a cardinal part of value which is a numerical value or group of numerical values that only exists in relation to what it describes, e.g. a confidence interval, numerical designation, unit. definition_source: OBI.<br></div></html><hr>
<a name='item300'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_85<br></b><b><i>unit</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A unit is a cardinal part of quantitative value which is any division of quantity accepted as a standard of measurement or exchange. definition_source: http://wordnet.princeton.edu/perl/webwn?s=unit, Jan 3, 2007.</div></i><font color="#000080">cardinal_part_of_quantitative_value</font><br><i>definition</i>: A unit is a cardinal part of quantitative value which is any division of quantity accepted as a standard of measurement or exchange. definition_source: http://wordnet.princeton.edu/perl/webwn?s=unit, Jan 3, 2007.<br></div></html><hr>
<a name='item301'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_84<br></b><b><i>magnitude</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Magnitude is a cardinal part of quantitative value which is a property of relative size or extent, whether large or small. definition_source: http://wordnet.princeton.edu/perl/webwn?s=magnitude, Jan 3, 2007.</div></i><font color="#000080">cardinal_part_of_quantitative_value</font><br><i>definition</i>: Magnitude is a cardinal part of quantitative value which is a property of relative size or extent, whether large or small. definition_source: http://wordnet.princeton.edu/perl/webwn?s=magnitude, Jan 3, 2007.<br></div></html><hr>
<a name='item302'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_300<br></b><b><i>narrative_object</i></b><i><br></i><font color="#000080">information_content_entity</font><br></div></html><hr>
<a name='item303'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_86<br></b><b><i>conclusion</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A conclusion is a narrative object making a statement about biomedical reality based in part on an interpretation of data generated in an investigation.</div></i><font color="#000080">narrative_object</font><br><i>editor_note</i>: rem0: need to work on data, inference and evidence rem1: it does not feel right that conclusion is a part of design specification<br><i>example_of_usage</i>: Nicotine is an addictive drug. Drug A is more efficacious in the treatment of cardiovascular disease than drug B in 40% of the population. The genome of clonal isolate of LCM virus is ... There was no statistically significant difference in treatment A or B. 
Negative example: We have a stored a sample in Biobank:<br><i>definition</i>: A conclusion is a narrative object making a statement about biomedical reality based in part on an interpretation of data generated in an investigation.<br></div></html><hr>
<a name='item304'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_13<br></b><b><i>objective</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An objective is a cardinal part of design specification which describes the motivation or reason for an investigation. definition_source: OBI.</div></i><font color="#000080">narrative_object</font><br><i>editor_note</i>: example: the objective of our work is to build an ontology, the hypothesis underlying the work is that it will support and benefit interoperability example: The aim of the study is designed to examine the effects of using different software, software parameters, or algorithms (e.g. data processing software).<br><i>definition</i>: An objective is a cardinal part of design specification which describes the motivation or reason for an investigation. definition_source: OBI.<br></div></html><hr>
<a name='item305'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_338<br></b><b><i>diagnosis</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is a narrative_object</div></i><font color="#000080">narrative_object</font><br><i>definition</i>: is a narrative_object<br></div></html><hr>
<a name='item306'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_337<br></b><b><i>patent</i></b><i><br></i><font color="#000080">narrative_object</font><br></div></html><hr>
<a name='item307'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_12<br></b><b><i>hypothesis</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px"> A hypothesis is a a cardinal part of design specification which states testable cause and effect relationships of certain facts or observations. definition_source: OBI.</div></i><font color="#000080">narrative_object</font><br><i>editor_note</i>: altdef: An experimental hypothesis is a statement about cause-effect relations between known and unknown (target) variables of the domain of the experiment ""that shall be in accordance with known facts*"" to be verified at the experiment. Reference: The Oxford English Dictionary. Oxford University Press, 2 Ed., 1989 (taken from EXPO, Larissa Soldatova)<br><i>definition</i>:  A hypothesis is a a cardinal part of design specification which states testable cause and effect relationships of certain facts or observations. definition_source: OBI.<br></div></html><hr>
<a name='item308'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_275<br></b><b><i>journal_article</i></b><i><br></i><font color="#000080">narrative_object</font><br></div></html><hr>
<a name='item309'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_286<br></b><b><i>clinical_finding</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">clinical_finding is a narrative_object authored by a clinician_practitioner</div></i><font color="#000080">narrative_object</font><br></div></html><hr>
<a name='item310'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_322<br></b><b><i>report</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A report is a narrative object that describes how a process occured</div></i><font color="#000080">narrative_object</font><br><i>definition</i>: A report is a narrative object that describes how a process occured<br></div></html><hr>
<a name='item311'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_46<br></b><b><i>investigation_record</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An investition_report is a report on an investigation definition_source: OBI.</div></i><font color="#000080">report</font><br><i>editor_note</i>: rem1:tells you how the experiment really proceeded. Might not be the method the protocol described.<br><i>definition</i>: An investition_report is a report on an investigation definition_source: OBI.<br></div></html><hr>
<a name='item312'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_305<br></b><b><i>data_object</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Example: Description of physical appearance of a mouse in an experiment, HLA typing information for donors enrolled in a study</div></i><font color="#000080">information_content_entity</font><br><i>definition</i>: Instance level information is information describing a specific instance of an entity<br></div></html><hr>
<a name='item313'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_325<br></b><b><i>image</i></b><i><br></i><font color="#000080">data_object</font><br></div></html><hr>
<a name='item314'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_18<br></b><b><i>value</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A value is a dependent continuant which consists of a scale plus a magnitude. definition_source: OBI.</div></i><font color="#000080">data_object</font><br><i>editor_note</i>: BFO problem<br><i>definition</i>: A value is a dependent continuant which consists of a scale plus a magnitude. definition_source: OBI.<br></div></html><hr>
<a name='item315'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_147<br></b><b><i>quantitative_value</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A quantitative value is a value which consists of a unit plus a numerical quantity which is measured, assigned, or computed. definition_source: OBI.</div></i><font color="#000080">value</font><br><i>definition</i>: A quantitative value is a value which consists of a unit plus a numerical quantity which is measured, assigned, or computed. definition_source: OBI.<br></div></html><hr>
<a name='item316'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_83<br></b><b><i>quantitative_confidence_value</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A quantitative confidence value is a quantitative value which is used to indicate the degree of uncertainty about a measurement. definition_source: OBI.</div></i><font color="#000080">quantitative_value</font><br><i>definition</i>: A quantitative confidence value is a quantitative value which is used to indicate the degree of uncertainty about a measurement. definition_source: OBI.<br></div></html><hr>
<a name='item317'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_151<br></b><b><i>standard_error</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Standard error is a quantitative confidence value which is the standard deviations of the sample in a frequency distribution, obtained by dividing the standard deviation by the total number of cases in the frequency distribution. definition_source:  Dictionary.com http://dictionary.reference.com/browse/standard%20error, Jan 3, 2007.</div></i><font color="#000080">quantitative_confidence_value</font><br><i>definition</i>: Standard error is a quantitative confidence value which is the standard deviations of the sample in a frequency distribution, obtained by dividing the standard deviation by the total number of cases in the frequency distribution. definition_source:  Dictionary.com http://dictionary.reference.com/browse/standard%20error, Jan 3, 2007.<br></div></html><hr>
<a name='item318'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_150<br></b><b><i>P_value</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A p-value is a quantitative confidence value which represents the probability of obtaining a result at least as extreme as that actually obtained, assuming that the actual value was the result of chance alone. definition_source: http://en.wikipedia.org/wiki/P-value, Jan 3, 2007.</div></i><font color="#000080">quantitative_confidence_value</font><br><i>definition</i>: A p-value is a quantitative confidence value which represents the probability of obtaining a result at least as extreme as that actually obtained, assuming that the actual value was the result of chance alone. definition_source: http://en.wikipedia.org/wiki/P-value, Jan 3, 2007.<br></div></html><hr>
<a name='item319'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_148<br></b><b><i>qualitative_value</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A qualitative value is a value which is non-numeric, and therefore describes the quality, characteristics or condition of an entity. definition_source</div></i><font color="#000080">value</font><br><i>editor_note</i>: rem0: components of this term are to be described in an external resource, typically PATO<br><i>definition</i>: A qualitative value is a value which is non-numeric, and therefore describes the quality, characteristics or condition of an entity. definition_source<br></div></html><hr>
<a name='item320'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_43<br></b><b><i>data_set</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A data set is an independent continuant which is a collection of numerical information that has something in common and is considered to form a unit. definition_source: OBI.</div></i><font color="#000080">data_object</font><br><i>definition</i>: A data set is an independent continuant which is a collection of numerical information that has something in common and is considered to form a unit. definition_source: OBI.<br></div></html><hr>
<a name='item321'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_344<br></b><b><i>information_entity</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is a set of instructions or rules that specifying how an entity should be realized</div></i><font color="#000080">generically_dependent_continuant</font><br><i>editor_note</i>: Part of range for an object property "is_potentially_realizable_as" in obi before split (LF 040907)<br><i>definition</i>: is a set of instructions or rules that specifying how an entity should be realized<br></div></html><hr>
<a name='item322'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_270<br></b><b><i>algorithm</i></b><i><br></i><font color="#000080">information_entity</font><br></div></html><hr>
<a name='item323'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_187<br></b><b><i>data_format_specification</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A data format specification is a plan which organizes information. Example: The ISO document specifying what encompasses an XML document; The instructions in a XSD file</div></i><font color="#000080">information_entity</font><br><i>definition</i>: A data format specification is a plan which organizes information. Example: The ISO document specifying what encompasses an XML document; The instructions in a XSD file<br></div></html><hr>
<a name='item324'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500000<br></b><b><i>study_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Description of how an experiment is performed. Experiments are designed by applying applications to biomaterials in an ordered fashion. Applications define protocols which are used to moving biomaterial through a series of manipulations to generate one or more desired products. methods describe the particulars of the manipulation, reagents and associated materials used in the manipulation of the biomaterial."</div></i><font color="#000080">information_entity</font><br><i>definition_editor</i>: KPC<br><i>editor_note</i>: From Kevin Clancy not yet discussed<br><i>definition_citation</i>: KPC<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "Cloning Experimental Design to produce an inframe expression clone using Gateway technology for the human, mouse and rat SOD1 mRNAs. IN each case. mRNA is isolated and sequence specific oligos with Gateway recombination sites are used to amplify the cDNA. The clones are combined with the entry vector pENTR201. CLones are sized by PCR of the inserts follwed by gel electrophoresis against a DNA ladder. Clones are sequence verified using Sanger sequencing and an ABI 373."<br><i>preferred_term</i>: study_design<br><i>definition</i>: "Description of how an experiment is performed. Experiments are designed by applying applications to biomaterials in an ordered fashion. Applications define protocols which are used to moving biomaterial through a series of manipulations to generate one or more desired products. methods describe the particulars of the manipulation, reagents and associated materials used in the manipulation of the biomaterial."<br></div></html><hr>
<a name='item325'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500008<br></b><b><i>balanced_incomplete_block_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">definition to be added</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: PMID: 12933549<br><i>curation_status</i>: definition_incomplete<br><i>example_of_usage</i>: PMID: 12933549<br><i>preferred_term</i>: balanced_incomplete_block_design<br><i>definition</i>: definition to be added<br></div></html><hr>
<a name='item326'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500010<br></b><b><i>reference_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A reference experiment design type is where all samples are compared to a common reference.</div></i><font color="#000080">balanced_incomplete_block_design</font><br><i>definition_editor</i>: PRS on behalf of MO<br><i>editor_note</i>: MO<br><i>definition_citation</i>: MO_699<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: PMID: 12933549<br><i>preferred_term</i>: reference_design<br><i>definition</i>: A reference experiment design type is where all samples are compared to a common reference.<br></div></html><hr>
<a name='item327'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500009<br></b><b><i>loop_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A loop experiment design is where labeled extracts are compared in consecutive pairs. synonym: circular design</div></i><font color="#000080">balanced_incomplete_block_design</font><br><i>definition_editor</i>: PRS on behalf of MO<br><i>editor_note</i>: MO<br><i>definition_citation</i>: MO_912<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: PMID: 12933549<br><i>preferred_term</i>: loop_design<br><i>definition</i>: A loop experiment design is where labeled extracts are compared in consecutive pairs. synonym: circular design<br></div></html><hr>
<a name='item328'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500020<br></b><b><i>time_series_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Groups of assays that are related as part of a time series.</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS on behalf of MO<br><i>editor_note</i>: MO<br><i>definition_citation</i>: MO_887<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 14744830-Microarrays are powerful tools for surveying the expression levels of many thousands of genes simultaneously. They belong to the new genomics technologies which have important applications in the biological, agricultural and pharmaceutical sciences. There are myriad sources of uncertainty in microarray experiments, and rigorous experimental design is essential for fully realizing the potential of these valuable resources. Two questions frequently asked by biologists on the brink of conducting cDNA or two-colour, spotted microarray experiments are 'Which mRNA samples should be competitively hybridized together on the same slide?' and 'How many times should each slide be replicated?' Early experience has shown that whilst the field of classical experimental design has much to offer this emerging multi-disciplinary area, new approaches which accommodate features specific to the microarray context are needed. In this paper, we propose optimal designs for factorial and time course experiments, which are special designs arising quite frequently in microarray experimentation. Our criterion for optimality is statistical efficiency based on a new notion of admissible designs; our approach enables efficient designs to be selected subject to the information available on the effects of most interest to biologists, the number of arrays available for the experiment, and other resource or practical constraints, including limitations on the amount of mRNA probe. We show that our designs are superior to both the popular reference designs, which are highly inefficient, and to designs incorporating all possible direct pairwise comparisons. Moreover, our proposed designs represent a substantial practical improvement over classical experimental designs which work in terms of standard interactions and main effects. The latter do not provide a basis for meaningful inference on the effects of most interest to biologists, nor make the most efficient use of valuable and limited resources."<br><i>preferred_term</i>: time_series_design<br><i>definition</i>: Groups of assays that are related as part of a time series.<br></div></html><hr>
<a name='item329'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#intervention_study<br></b><i></i><font color="#000080">(<i>and</i>&nbsp;study_design<br>&nbsp;(is_realized_as&nbsp;<i>some</i><br>&nbsp;&nbsp;(<i>and</i>&nbsp;process<br>&nbsp;&nbsp;&nbsp;(has_part&nbsp;<i>some</i><br>&nbsp;&nbsp;&nbsp;&nbsp;(has_input&nbsp;<i>some</i>&nbsp;(has_role&nbsp;<i>some</i>&nbsp;treated_role))))))</font><br></div></html><hr>
<a name='item330'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500006<br></b><b><i>parallel group_design (syn:Independent measures design)</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An independent measure design is a study design which uses unique experimental unit each experimental group, in other word no two individuals are shared between experimental groups, hence also known as parallel group design"</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: http://www.holah.karoo.net/experimentaldesigns.htm<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17408389-Purpose: Proliferative vitreoretinopathy (PVR) is the most important reason for blindness following retinal detachment. Presently, vitreous tamponades such as gas or silicone oil cannot contact the lower part of the retina. A heavier-than-water tamponade displaces the inflammatory and PVR-stimulating environment from the inferior area of the retina. The Heavy Silicone Oil versus Standard Silicone Oil Study (HSO Study) is designed to answer the question of whether a heavier-than-water tamponade improves the prognosis of eyes with PVR of the lower retina. Methods: The HSO Study is a multicentre, randomized, prospective controlled clinical trial comparing two endotamponades within a two-arm parallel group design. Patients with inferiorly and posteriorly located PVR are randomized to either heavy silicone oil or standard silicone oil as a tamponading agent. Three hundred and fifty consecutive patients are recruited per group. After intraoperative re-attachment, patients are randomized to either standard silicone oil (1000 cSt or 5000 cSt) or Densiron((R)) as a tamponading agent. The main endpoint criteria are complete retinal attachment at 12 months and change of visual acuity (VA) 12 months postoperatively compared with the preoperative VA. Secondary endpoints include complete retinal attachment before endotamponade removal, quality of life analysis and the number of retina affecting re-operation within 1 year of follow-up. Results: The design and early recruitment phase of the study are described. Conclusions: The results of this study will uncover whether or not heavy silicone oil improves the prognosis of eyes with PVR."<br><i>preferred_term</i>: parallel group_design (syn:Independent measures design)<br><i>definition</i>: "An independent measure design is a study design which uses unique experimental unit each experimental group, in other word no two individuals are shared between experimental groups, hence also known as parallel group design"<br></div></html><hr>
<a name='item331'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500019<br></b><b><i>self_vs_self_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A self vs. self experiment design investigates variance and error estimates in the experimental system, and is where the same extract is compared."</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS on behalf of MO<br><i>editor_note</i>: MO<br><i>definition_citation</i>: MO_490<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17411393-Dye-specific bias effects, commonly observed in the two-color microarray platform, are normally corrected using the dye swap design. This design, however, is relatively expensive and labor-intensive. We propose a self-self hybridization design as an alternative to the dye swap design. In this design, the treated and control samples are labeled with Cy5 and Cy3 (or Cy3 and Cy5), respectively, without dye swap, along with a set of self-self hybridizations on the control sample. We compare this design with the dye swap design through investigation of mouse primary hepatocytes treated with three peroxisome proliferator-activated receptor-alpha (PPARalpha) agonists at three dose levels. Using Agilent's Whole Mouse Genome microarray, differentially expressed genes (DEG) were determined for both the self-self hybridization and dye swap designs. The DEG concordance between the two designs was over 80% across each dose treatment and chemical. Furthermore, 90% of DEG-associated biological pathways were in common between the designs, indicating that biological interpretations would be consistent. The reduced labor and expense for the self-self hybridization design make it an efficient substitute for the dye swap design. For example, in larger toxicogenomic studies, only about half the chips are required for the self-self hybridization design compared to that needed in the dye swap design."<br><i>preferred_term</i>: self_vs_self_design<br><i>definition</i>: "A self vs. self experiment design investigates variance and error estimates in the experimental system, and is where the same extract is compared."<br></div></html><hr>
<a name='item332'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500002<br></b><b><i>sequential_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A repeated measure design is a study design which use the same individuals and exposure them to a set of conditions. The effect of order and practice can be confounding factor in such designs</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: http://www.holah.karoo.net/experimentaldesigns.htm<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: sequential_design<br><i>definition</i>: A repeated measure design is a study design which use the same individuals and exposure them to a set of conditions. The effect of order and practice can be confounding factor in such designs<br></div></html><hr>
<a name='item333'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500003<br></b><b><i>cross_over_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"a cross over design is a study design which ensures that experimental units receive,in sequence, the treatment (or the control), and then, after a specified time, switch to the control (or treatment). In this design, patients serveas their own controls, and randomization is used to determine the ordering which a patient receives the treatment and control. Often, wash out periods are devised between treatments"</div></i><font color="#000080">sequential_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>editor_note</i>: Replace patient or subject (keep track of this in the definition<br><i>definition_citation</i>: (source: http://www.sbu.se/Filer/Content0/publikationer/1/literaturesearching_1993/glossary.html)<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17601993-Objective: HIV-infected patients with lipodystrophy (HIV-lipodystrophy) are insulin resistant and have elevated plasma free fatty acid (FFA) concentrations. We aimed to explore the mechanisms underlying FFA-induced insulin resistance in patients with HIV-lipodystrophy. Research Design and Methods: Using a randomized placebo-controlled cross-over design, we studied the effects of an overnight acipimox-induced suppression of FFA on glucose and FFA metabolism by using stable isotope labelled tracer techniques during basal conditions and a two-stage euglycemic, hyperinsulinemic clamp (20 mU insulin/m(2)/min; 50 mU insulin/m(2)/min) in nine patients with nondiabetic HIV-lipodystrophy. All patients received antiretroviral therapy. Biopsies from the vastus lateralis muscle were obtained during each stage of the clamp. Results: Acipimox treatment reduced basal FFA rate of appearance by 68.9% (52.6%-79.5%) and decreased plasma FFA concentration by 51.6 % (42.0%-58.9%), (both, P < 0.0001). Endogenous glucose production was not influenced by acipimox. During the clamp the increase in glucose-uptake was significantly greater after acipimox treatment compared to placebo (acipimox: 26.85 (18.09-39.86) vs placebo: 20.30 (13.67-30.13) mumol/kg/min; P < 0.01). Insulin increased phosphorylation of Akt (Thr(308)) and GSK-3beta (Ser(9)), decreased phosphorylation of glycogen synthase (GS) site 3a+b and increased GS-activity (I-form) in skeletal muscle (P < 0.01). Acipimox decreased phosphorylation of GS (site 3a+b) (P < 0.02) and increased GS-activity (P < 0.01) in muscle. Conclusion: The present study provides direct evidence that suppression of lipolysis in patients with HIV-lipodystrophy improves insulin-stimulated peripheral glucose-uptake. The increased glucose-uptake may in part be explained by increased dephosphorylation of GS (site 3a+b) resulting in increased GS activity."<br><i>preferred_term</i>: cross_over_design<br><i>definition</i>: "a cross over design is a study design which ensures that experimental units receive,in sequence, the treatment (or the control), and then, after a specified time, switch to the control (or treatment). In this design, patients serveas their own controls, and randomization is used to determine the ordering which a patient receives the treatment and control. Often, wash out periods are devised between treatments"<br></div></html><hr>
<a name='item334'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500004<br></b><b><i>n-to-1_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"N-of-1 design is a cross-over design in which the same patient is repeatedly randomised to receive either the experimental treatment or its control (Senn, 1993)."</div></i><font color="#000080">cross_over_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: Adapted from  http://www.childrens-mercy.org/stats/definitions/crossover.htm and source:http://symptomresearch.nih.gov/chapter_6/sec1/csss1pg1.htm)<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: n-to-1_design<br><i>definition</i>: "N-of-1 design is a cross-over design in which the same patient is repeatedly randomised to receive either the experimental treatment or its control (Senn, 1993)."<br></div></html><hr>
<a name='item335'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500014<br></b><b><i>factorial_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"factorial design is_a Study design which is used to evaluate two or more factors simultaneously. The treatments are combinations of levels of the factors. The advantages of factorial designs over one-factor-at-a-time experiments is that they are more efficient and they allow interactions to be detected. In statistics, a factorial design experiment is an experiment whose design consists of two or more factors, each with discrete possible values or ""levels"", and whose experimental units take on all possible combinations of these levels across all such factors. Such an experiment allows studying the effect of each factor on the response variable, as well as the effects of interactions between factors on the response variable."</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: http://www.stats.gla.ac.uk/steps/glossary/anova.html#facdes And from wikipedia (01/03/2007): http://en.wikipedia.org/wiki/Factorial_experiment)<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17582121-Our objective was to examine the effects of dietary cation-anion difference (DCAD) with different concentrations of dietary crude protein (CP) on performance and acid-base status in early lactation cows. Six lactating Holstein cows averaging 44 d in milk were used in a 6 x 6 Latin square design with a 2 x 3 factorial arrangement of treatments: DCAD of -3, 22, or 47 milliequivalents (Na + K - Cl - S)/100 g of dry matter (DM), and 16 or 19% CP on a DM basis. Linear increases with DCAD occurred in DM intake, milk fat percentage, 4% fat-corrected milk production, milk true protein, milk lactose, and milk solids-not-fat. Milk production itself was unaffected by DCAD. Jugular venous blood pH, base excess and HCO3(-) concentration, and urine pH increased, but jugular venous blood Cl- concentration, urine titratable acidity, and net acid excretion decreased linearly with increasing DCAD. An elevated ratio of coccygeal venous plasma essential AA to nonessential AA with increasing DCAD indicated that N metabolism in the rumen was affected, probably resulting in more microbial protein flowing to the small intestine. Cows fed 16% CP had lower urea N in milk than cows fed 19% CP; the same was true for urea N in coccygeal venous plasma and urine. Dry matter intake, milk production, milk composition, and acid-base status did not differ between the 16 and 19% CP treatments. It was concluded that DCAD affected DM intake and performance of dairy cows in early lactation. Feeding 16% dietary CP to cows in early lactation, compared with 19% CP, maintained lactation performance while reducing urea N excretion in milk and urine."<br><i>preferred_term</i>: factorial_design<br><i>definition</i>: "factorial design is_a Study design which is used to evaluate two or more factors simultaneously. The treatments are combinations of levels of the factors. The advantages of factorial designs over one-factor-at-a-time experiments is that they are more efficient and they allow interactions to be detected. In statistics, a factorial design experiment is an experiment whose design consists of two or more factors, each with discrete possible values or ""levels"", and whose experimental units take on all possible combinations of these levels across all such factors. Such an experiment allows studying the effect of each factor on the response variable, as well as the effects of interactions between factors on the response variable."<br></div></html><hr>
<a name='item336'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500015<br></b><b><i>2x2_factorial_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">2x2 factorial design is factorial design which has 2 experimental factors (aka independent variables) and 2 factor levels per experimental factors</div></i><font color="#000080">factorial_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: PMID: 17561240<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17561240-The present experiment evaluates the effects of intermittent exposure to a social stimulus on ethanol and water drinking in rats. Four groups of rats were arranged in a 2x2 factorial design with 2 levels of Social procedure (Intermittent Social vs Continuous Social) and 2 levels of sipper Liquid (Ethanol vs Water). Intermittent Social groups received 35 trials per session. Each trial consisted of the insertion of the sipper tube for 10 s followed by lifting of the guillotine door for 15 s. The guillotine door separated the experimental rat from the conspecific rat in the wire mesh cage during the 60 s inter-trial interval. The Continuous Social groups received similar procedures except that the guillotine door was raised during the entire duration of the session. For the Ethanol groups, the concentrations of ethanol in the sipper [3, 4, 6, 8, 10, 12, 14, and 16% (vol/vol)] increased across sessions, while the Water groups received 0% ethanol (water) in the sipper throughout the experiment. Both Social procedures induced more intake of ethanol than water. The Intermittent Social procedure induced more ethanol intake at the two highest ethanol concentration blocks (10-12% and 14-16%) than the Continuous Social procedure, but this effect was not observed with water. Effects of social stimulation on ethanol drinking are discussed."<br><i>preferred_term</i>: 2x2_factorial_design<br><i>definition</i>: 2x2 factorial design is factorial design which has 2 experimental factors (aka independent variables) and 2 factor levels per experimental factors<br></div></html><hr>
<a name='item337'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500016<br></b><b><i>fractional_factorial_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A fractional factorial design is_a study design in which only an adequately chosen fraction of the treatment combinations required for the complete factorial experiment is selected to be run</div></i><font color="#000080">factorial_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: http://www.itl.nist.gov/div898/handbook/pri/section3/pri334.htm From ASQC (1983) Glossary & Tables for Statistical Quality Control<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: fractional_factorial_design<br><i>definition</i>: A fractional factorial design is_a study design in which only an adequately chosen fraction of the treatment combinations required for the complete factorial experiment is selected to be run<br></div></html><hr>
<a name='item338'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500017<br></b><b><i>dye_swap_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"An experiment design type where the label orientations are reversed. exact synonym: flip dye, dye flip"</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS on behalf of MO<br><i>editor_note</i>: MO<br><i>definition_citation</i>: MO_858<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17411393-Dye-specific bias effects, commonly observed in the two-color microarray platform, are normally corrected using the dye swap design. This design, however, is relatively expensive and labor-intensive. We propose a self-self hybridization design as an alternative to the dye swap design. In this design, the treated and control samples are labeled with Cy5 and Cy3 (or Cy3 and Cy5), respectively, without dye swap, along with a set of self-self hybridizations on the control sample. We compare this design with the dye swap design through investigation of mouse primary hepatocytes treated with three peroxisome proliferator-activated receptor-alpha (PPARalpha) agonists at three dose levels. Using Agilent's Whole Mouse Genome microarray, differentially expressed genes (DEG) were determined for both the self-self hybridization and dye swap designs. The DEG concordance between the two designs was over 80% across each dose treatment and chemical. Furthermore, 90% of DEG-associated biological pathways were in common between the designs, indicating that biological interpretations would be consistent. The reduced labor and expense for the self-self hybridization design make it an efficient substitute for the dye swap design. For example, in larger toxicogenomic studies, only about half the chips are required for the self-self hybridization design compared to that needed in the dye swap design."<br><i>preferred_term</i>: dye_swap_design<br><i>definition</i>: "An experiment design type where the label orientations are reversed. exact synonym: flip dye, dye flip"<br></div></html><hr>
<a name='item339'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500007<br></b><b><i>randomized_complete_block_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A randomized complete block design is_a study design which assigns randomly treatments to block. The number of units per block equals the number of treatment so each block receives each treatment exactly once (hence the qualifier 'complete'). The design was originally devised from field trials used in agronomy and agriculture. The analysis assumes that there is no interaction between block and treatment. The method was then used in other settings So The randomised complete block design is a design in which the subjects are matched according to a variable which the experimenter wishes to control. The subjects are put into groups (blocks) of the same size as the number of treatments. The members of each block are then randomly assigned to different treatment groups.</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: http://www.tufts.edu/~gdallal/ranblock.htm<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: """http://www.stats.gla.ac.uk/steps/glossary/anova.html"",(""A researcher is carrying out a study of the effectiveness of four different skin creams for the treatment of a certain skin disease. He has eighty subjects and plans to divide them into 4 treatment groups of twenty subjects each. Using a randomised blocks""&"" design, the subjects are assessed and put in blocks of four according to how severe their skin condition is; the four most severe cases are the first block, the next four most severe cases are the second block, and so on to the twentieth block. The four ""&""members of each block are then randomly assigned, one to each of the four treatment groups. http://www.stats.gla.ac.uk/steps/glossary/anova.html#rbd""))"<br><i>preferred_term</i>: randomized_complete_block_design<br><i>definition</i>: A randomized complete block design is_a study design which assigns randomly treatments to block. The number of units per block equals the number of treatment so each block receives each treatment exactly once (hence the qualifier 'complete'). The design was originally devised from field trials used in agronomy and agriculture. The analysis assumes that there is no interaction between block and treatment. The method was then used in other settings So The randomised complete block design is a design in which the subjects are matched according to a variable which the experimenter wishes to control. The subjects are put into groups (blocks) of the same size as the number of treatments. The members of each block are then randomly assigned to different treatment groups.<br></div></html><hr>
<a name='item340'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500012<br></b><b><i>graeco_latin_square_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Greco-Latin square design is a study design which relates to Latin square design</div></i><font color="#000080">randomized_complete_block_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>editor_note</i>: only 2 articles in pubmed ->probably irrelevant<br><i>definition_citation</i>: Adapted from: http://www.itl.nist.gov/div898/handbook/pri/section3/pri3321.htm and<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 6846242-Beaton et al (Am J Clin Nutr 1979;32:2546-59) reported on the partitioning of variance in 1-day dietary data for the intake of energy, protein, total carbohydrate, total fat, classes of fatty acids, cholesterol, and alcohol. Using the same food intake data and the expanded National Heart, Lung and Blood Institute food composition data base, these analyses of sources of variance have been expanded to include classes of carbohydrate, vitamin A, vitamin C, thiamin, riboflavin, niacin, calcium, iron, total ash, caffeine, and crude fiber. The analyses relate to observed intakes (replicated six times) of 30 adult males and 30 adult females obtained under a paired Graeco-Latin square design with sequence of interview, interviewer, and day of the week as determinants. Neither sequence nor interviewer made consistent contribution to variance. In females, day of the week had a significant effect for several nutrients. The major partitioning of variance was between interindividual variation (between subjects) and intraindividual variation (within subjects) which included both true day-to-day variation in intake and methodological variation. For all except caffeine, the intraindividual variability of 1-day data was larger than the interindividual variability. For vitamin A, almost all of the variance was associated with day-to-day variability. One day data provide a very inadequate estimate of usual intake of individuals. In the design of nutrition studies it is critical that the intended use of dietary data be a major consideration in deciding on methodology. There is no ""ideal"" dietary method. There may be preferred methods for particular purposes."<br><i>preferred_term</i>: graeco_latin_square_design<br><i>definition</i>: Greco-Latin square design is a study design which relates to Latin square design<br></div></html><hr>
<a name='item341'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500011<br></b><b><i>latin_square_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Latin square design is_a study design which allows in its simpler form controlling 2 levels of nuisance variables (also known as blocking variables).he 2 nuisance factors are divided into a tabular grid with the property that each row and each column receive each treatment exactly once.</div></i><font color="#000080">randomized_complete_block_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: Adapted from: http://www.itl.nist.gov/div898/handbook/pri/section3/pri3321.htm and<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17582121-Our objective was to examine the effects of dietary cation-anion difference (DCAD) with different concentrations of dietary crude protein (CP) on performance and acid-base status in early lactation cows. Six lactating Holstein cows averaging 44 d in milk were used in a 6 x 6 Latin square design with a 2 x 3 factorial arrangement of treatments: DCAD of -3, 22, or 47 milliequivalents (Na + K - Cl - S)/100 g of dry matter (DM), and 16 or 19% CP on a DM basis. Linear increases with DCAD occurred in DM intake, milk fat percentage, 4% fat-corrected milk production, milk true protein, milk lactose, and milk solids-not-fat. Milk production itself was unaffected by DCAD. Jugular venous blood pH, base excess and HCO3(-) concentration, and urine pH increased, but jugular venous blood Cl- concentration, urine titratable acidity, and net acid excretion decreased linearly with increasing DCAD. An elevated ratio of coccygeal venous plasma essential AA to nonessential AA with increasing DCAD indicated that N metabolism in the rumen was affected, probably resulting in more microbial protein flowing to the small intestine. Cows fed 16% CP had lower urea N in milk than cows fed 19% CP; the same was true for urea N in coccygeal venous plasma and urine. Dry matter intake, milk production, milk composition, and acid-base status did not differ between the 16 and 19% CP treatments. It was concluded that DCAD affected DM intake and performance of dairy cows in early lactation. Feeding 16% dietary CP to cows in early lactation, compared with 19% CP, maintained lactation performance while reducing urea N excretion in milk and urine."<br><i>preferred_term</i>: latin_square_design<br><i>definition</i>: Latin square design is_a study design which allows in its simpler form controlling 2 levels of nuisance variables (also known as blocking variables).he 2 nuisance factors are divided into a tabular grid with the property that each row and each column receive each treatment exactly once.<br></div></html><hr>
<a name='item342'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500013<br></b><b><i>hyper_graeco_latin_square_design</i></b><i><br></i><font color="#000080">randomized_complete_block_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>editor_note</i>: no example found in pubmed<br><i>definition_citation</i>: Adapted from: http://www.itl.nist.gov/div898/handbook/pri/section3/pri3321.htm and<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: hyper_graeco_latin_square_design<br><i>definition</i>: <br></div></html><hr>
<a name='item343'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500001<br></b><b><i>clinical_study_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"Plan for the precise procedure to be followed in a clinical trial, including planned and actual timing of events, choice of control group, method of allocating treatments, blinding methods; assigns a subject to pass through one or more epochs in the course of a trial. Specific design elements, e.g., crossover, parallel; dose-escalation [Modified from Pocock, Clinical Trials: A Practical Approach]"</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: The definition needs to be extended to other things than simply patients<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: Clinical Research Glossary Version 4.0 CDICS glossary group<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: clinical_study_design<br><i>definition</i>: "Plan for the precise procedure to be followed in a clinical trial, including planned and actual timing of events, choice of control group, method of allocating treatments, blinding methods; assigns a subject to pass through one or more epochs in the course of a trial. Specific design elements, e.g., crossover, parallel; dose-escalation [Modified from Pocock, Clinical Trials: A Practical Approach]"<br></div></html><hr>
<a name='item344'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500005<br></b><b><i>matched_pairs_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"A matched pair design is a study design which use groups of individuals associated (hence matched) to each other based on a set of criteria, one member going to one treatment, the other member receiving the other treatment."</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: http://www.holah.karoo.net/experimentaldesigns.htm<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17288613-BSTRACT: BACKGROUND: Physicians in Canadian emergency departments (EDs) annually treat 185,000 alert and stable trauma victims who are at risk for cervical spine (C-spine) injury. However, only 0.9% of these patients have suffered a cervical spine fracture. Current use of radiography is not efficient. The Canadian C-Spine Rule is designed to allow physicians to be more selective and accurate in ordering C-spine radiography, and to rapidly clear the C-spine without the need for radiography in many patients. The goal of this phase III study is to evaluate the effectiveness of an active strategy to implement the Canadian C-Spine Rule into physician practice. Specific objectives are to: 1) determine clinical impact, 2) determine sustainability, 3) evaluate performance, and 4) conduct an economic evaluation. METHODS: We propose a matched-pair cluster design study that compares outcomes during three consecutive 12-months ""before,"" ""after,"" and ""decay"" periods at six pairs of ""intervention"" and ""control"" sites. These 12 hospital ED sites will be stratified as ""teaching"" or ""community"" hospitals, matched according to baseline C-spine radiography ordering rates, and then allocated within each pair to either intervention or control groups. During the ""after"" period at the intervention sites, simple and inexpensive strategies will be employed to actively implement the Canadian C-Spine Rule. The following outcomes will be assessed: 1) measures of clinical impact, 2) performance of the Canadian C-Spine Rule, and 3) economic measures. During the 12-month ""decay"" period, implementation strategies will continue, allowing us to evaluate the sustainability of the effect. We estimate a sample size of 4,800 patients in each period in order to have adequate power to evaluate the main outcomes. DISCUSSION: Phase I successfully derived the Canadian C-Spine Rule and phase II confirmed the accuracy and safety of the rule, hence, the potential for physicians to improve care. What remains unknown is the actual change in clinical behaviors that can be affected by implementation of the Canadian C-Spine Rule, and whether implementation can be achieved with simple and inexpensive measures. We believe that the Canadian C-Spine Rule has the potential to significantly reduce health care costs and improve the efficiency of patient flow in busy Canadian EDs."<br><i>preferred_term</i>: matched_pairs_design<br><i>definition</i>: "A matched pair design is a study design which use groups of individuals associated (hence matched) to each other based on a set of criteria, one member going to one treatment, the other member receiving the other treatment."<br></div></html><hr>
<a name='item345'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500018<br></b><b><i>replicate_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A replicate experimental design type is where a series of replicates are performed to evaluate reproducibility or as a pilot study to determine the appropriate number of replicates for a subsequent experiments.</div></i><font color="#000080">study_design</font><br><i>definition_editor</i>: PRS on behalf of MO<br><i>editor_note</i>: MO<br><i>definition_citation</i>: MO_885<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: replicate_design<br><i>definition</i>: A replicate experimental design type is where a series of replicates are performed to evaluate reproducibility or as a pilot study to determine the appropriate number of replicates for a subsequent experiments.<br></div></html><hr>
<a name='item346'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_335<br></b><b><i>investigation_design</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Investigation_design is a plan for an investigation. definition_source: OBI.</div></i><font color="#000080">information_entity</font><br><i>definition</i>: Investigation_design is a plan for an investigation. definition_source: OBI.<br></div></html><hr>
<a name='item347'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_272<br></b><b><i>protocol</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: Protocol is a plan outlining a set of instructions which can be realized as a protocol application. Example: The SOP describing the execution of a clinical trial. Affymetrix Mass 5 analysis. The dose and route of administration of myosin into a mouse.</div></i><font color="#000080">information_entity</font><br><i>definition</i>: Definition: Protocol is a plan outlining a set of instructions which can be realized as a protocol application. Example: The SOP describing the execution of a clinical trial. Affymetrix Mass 5 analysis. The dose and route of administration of myosin into a mouse.<br></div></html><hr>
<a name='item348'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_349<br></b><b><i>instruction_trigger</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is a plan to carry out a process or an event if the trigger condition is fulfilled</div></i><font color="#000080">information_entity</font><br><i>editor_note</i>: <br><i>definition</i>: is a plan to carry out a process or an event if the trigger condition is fulfilled<br></div></html><hr>
<a name='item349'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500021<br></b><b><i>rule</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">"a rule is an executable which guides, defines, restricts actions"</div></i><font color="#000080">instruction_trigger</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: PRS<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: rule<br><i>definition</i>: "a rule is an executable which guides, defines, restricts actions"<br></div></html><hr>
<a name='item350'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500022<br></b><b><i>stopping_rule</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">a stopping rule (criterion) is_a *rule* which causes a *stopping process* to happen</div></i><font color="#000080">rule</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: PRS<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17591081-BACKGROUND/AIMS: To investigate the viral kinetics of Chinese CHC patients received pegylated interferon plus ribavirin and examine the impact of HCV genotypes and severity of liver disease. METHODOLOGY: 65 treatment-naove CHC patients who finished a 24-week therapy with peginterferon (alpha-2b (1.5 mcg/kg/week) plus ribavirin (1000-1200 mg /day) and 24 weeks of follow-up were enrolled. Hepatic fibrosis was graded by the METAVIR scoring system. Serum quantitative HCV RNA was determined by Versant HCV RNA 3.0 assay (Bayer Inc.). RESULTS: Genotype non-1 patients responded quickly and a higher percentage of them achieved undetectable HCV RNA (< 615 IU/mL) at week 4 compared with genotype 1 patients (93% vs. 69%, p = 0.018). Degree of hepatic fibrosis significantly affected end-of-treatment and sustained response (SVR). For patients who did not achieve early virological response (EVR), the negative predictive value for SVR was 100%. In genotype 1 patients, undetectable HCV RNA by week 4 was a good marker to predict treatment response, with a positive predictive value of 84% and a negative predictive value of 82%. CONCLUSIONS: EVR can be applied to Chinese patients as an early stopping rule. A 24-week duration of pegylated IFN/ribavirin might be adequate for genotype 1 patients who rapidly responded to therapy."<br><i>preferred_term</i>: stopping_rule<br><i>definition</i>: a stopping rule (criterion) is_a *rule* which causes a *stopping process* to happen<br></div></html><hr>
<a name='item351'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500023<br></b><b><i>compliance_rule</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">a compliance rule is a rule which ensures a compliance process occurs</div></i><font color="#000080">rule</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: PRS<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: compliance_rule<br><i>definition</i>: a compliance rule is a rule which ensures a compliance process occurs<br></div></html><hr>
<a name='item352'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500024<br></b><b><i>standard_compliance_rule</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">a  standard compliance rule is a compliance rule which defines conformity to a representation standard</div></i><font color="#000080">compliance_rule</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: PRS<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: standard_compliance_rule<br><i>definition</i>: a  standard compliance rule is a compliance rule which defines conformity to a representation standard<br></div></html><hr>
<a name='item353'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500025<br></b><b><i>ethical_standard_compliance_rule</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">an ethical standard compliance rule is_a *compliance rule* which enable a  *ethical compliance process* to occur</div></i><font color="#000080">standard_compliance_rule</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: PRS<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: ethical_standard_compliance_rule<br><i>definition</i>: an ethical standard compliance rule is_a *compliance rule* which enable a  *ethical compliance process* to occur<br></div></html><hr>
<a name='item354'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500026<br></b><b><i>eligibility_rule</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">an eligibility rule (criterion) is_a *rule* which defines the positive and negative criteria decide an entity to enter a study and that will define the population of the study</div></i><font color="#000080">rule</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: Adapted from Clinical Research Glossary Version 4.0 CDICS glossary group<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17579629 -Eligibility criteria included: untreated ED-SCLC; age >/=70 and performance status 0-2, or age <70 and PS 3. "<br><i>preferred_term</i>: eligibility_rule<br><i>definition</i>: an eligibility rule (criterion) is_a *rule* which defines the positive and negative criteria decide an entity to enter a study and that will define the population of the study<br></div></html><hr>
<a name='item355'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500027<br></b><b><i>inclusion_rule</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">an inclusion rule (criterion) is_ rule*  in a protocol that prospective subjects must meet to be eligible for participation in a study</div></i><font color="#000080">eligibility_rule</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: Adapted from Clinical Research Glossary Version 4.0 CDICS glossary group<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: "PMID: 17599746-RESULTS: Fifteen trials met our inclusion criteria. The summary effect size was 0.35 (95% confidence interval 0.14, 0.56). I(2) was 0.80."<br><i>preferred_term</i>: inclusion_rule<br><i>definition</i>: an inclusion rule (criterion) is_ rule*  in a protocol that prospective subjects must meet to be eligible for participation in a study<br></div></html><hr>
<a name='item356'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_500028<br></b><b><i>exclusion_rule</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">an exclusion rule (criterion) is_a *eligibility rule*  prospective subjects should not meet to be eligible for participation in a study</div></i><font color="#000080">eligibility_rule</font><br><i>definition_editor</i>: PRS<br><i>editor_note</i>: MSI<br><i>definition_citation</i>: Adapted from Clinical Research Glossary Version 4.0 CDICS glossary group<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: PMID: 17600285-Exclusion criteria included the use of any topical ophthalmic or topical oral medication and/or history of ocular or oral surgery within the past six months.<br><i>preferred_term</i>: exclusion_rule<br><i>definition</i>: an exclusion rule (criterion) is_a *eligibility rule*  prospective subjects should not meet to be eligible for participation in a study<br></div></html><hr>
<a name='item357'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_331<br></b><b><i>time_trigger</i></b><i><br></i><font color="#000080">instruction_trigger</font><br></div></html><hr>
<a name='item358'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_310<br></b><b><i>adverse_event_trigger</i></b><i><br></i><font color="#000080">instruction_trigger</font><br></div></html><hr>
<a name='item359'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_306<br></b><b><i>digital_entity</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Def: digital entity is an information entity which is a collection of bits that can be interpreted by a computer. Two digital entities are the same if they are bitwise identical.</div></i><font color="#000080">generically_dependent_continuant</font><br><i>editor_note</i>: information_content_entity "is_encoded_in" some digital_entity in obi before split (040907).<br><i>example_of_usage</i>: pdf file, jpeg file, XML file<br><i>definition</i>: Def: digital entity is an information entity which is a collection of bits that can be interpreted by a computer. Two digital entities are the same if they are bitwise identical.<br></div></html><hr>
<a name='item360'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_351<br></b><b><i>binary_executable</i></b><i><br></i><font color="#000080">digital_entity</font><br></div></html><hr>
<a name='item361'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_352<br></b><b><i>digital_document</i></b><i><br></i><font color="#000080">digital_entity</font><br></div></html><hr>
<a name='item362'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_353<br></b><b><i>source_code_module</i></b><i><br></i><font color="#000080">digital_entity</font><br></div></html><hr>
<a name='item363'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#SpatialRegion<br></b><b><i>spatial_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: An independent continuant that is neither bearer of qualities nor inheres in any other entities.</div></i><font color="#000080">(<i>and</i>&nbsp;continuant<br>&nbsp;(<i>or</i>&nbsp;one_dimensional_region&nbsp;three_dimensional_region&nbsp;two_dimensional_region<br>&nbsp;&nbsp;zero_dimensional_region)<br>&nbsp;entity<br>&nbsp;(<i>disjoint-with</i>&nbsp;specifically_dependent_continuant&nbsp;independent_continuant&nbsp;occurrent<br>&nbsp;&nbsp;quality&nbsp;realizable_entity))</font><br></div></html><hr>
<a name='item364'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#TwoDimensionalRegion<br></b><b><i>two_dimensional_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A spatial region with two dimensions.</div></i><font color="#000080">(<i>and</i>&nbsp;spatial_region<br>&nbsp;(<i>disjoint-with</i>&nbsp;three_dimensional_region&nbsp;one_dimensional_region<br>&nbsp;&nbsp;zero_dimensional_region))</font><br></div></html><hr>
<a name='item365'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#ThreeDimensionalRegion<br></b><b><i>three_dimensional_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A spatial region with three dimensions.</div></i><font color="#000080">(<i>and</i>&nbsp;spatial_region<br>&nbsp;(<i>disjoint-with</i>&nbsp;two_dimensional_region&nbsp;one_dimensional_region<br>&nbsp;&nbsp;zero_dimensional_region))</font><br></div></html><hr>
<a name='item366'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#ZeroDimensionalRegion<br></b><b><i>zero_dimensional_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A spatial region with no dimensions.</div></i><font color="#000080">(<i>and</i>&nbsp;spatial_region<br>&nbsp;(<i>disjoint-with</i>&nbsp;two_dimensional_region&nbsp;three_dimensional_region<br>&nbsp;&nbsp;one_dimensional_region))</font><br></div></html><hr>
<a name='item367'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/snap#OneDimensionalRegion<br></b><b><i>one_dimensional_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A spatial region with one dimension.</div></i><font color="#000080">(<i>and</i>&nbsp;spatial_region<br>&nbsp;(<i>disjoint-with</i>&nbsp;two_dimensional_region&nbsp;three_dimensional_region<br>&nbsp;&nbsp;zero_dimensional_region))</font><br></div></html><hr>
<a name='item368'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#Occurrent<br></b><b><i>occurrent</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: An entity that has temporal parts and that happens, unfolds or develops through time. Sometimes also called perdurants.</div></i><font color="#000080">(<i>and</i>&nbsp;(<i>not</i>&nbsp;continuant)&nbsp;entity<br>&nbsp;(<i>or</i>&nbsp;processual_entity&nbsp;spatiotemporal_region&nbsp;temporal_region))</font><br></div></html><hr>
<a name='item369'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#SpatiotemporalRegion<br></b><b><i>spatiotemporal_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: An occurrent at or in which processual entities can be located.</div></i><font color="#000080">(<i>and</i>&nbsp;(<i>not</i>&nbsp;temporal_region)&nbsp;occurrent<br>&nbsp;(<i>or</i>&nbsp;connected_spatiotemporal_region&nbsp;scattered_spatiotemporal_region))</font><br></div></html><hr>
<a name='item370'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#ConnectedSpatiotemporalRegion<br></b><b><i>connected_spatiotemporal_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A space time region that has temporal and spatial dimensions such that all points within the spatiotemporal region are mediately or immediately connected to all other points within the same space time region.</div></i><font color="#000080">(<i>and</i>&nbsp;(<i>or</i>&nbsp;spatiotemporal_instant&nbsp;spatiotemporal_interval)<br>&nbsp;spatiotemporal_region)</font><br></div></html><hr>
<a name='item371'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#SpatiotemporalInterval<br></b><b><i>spatiotemporal_interval</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A connected space time region that endures for more than a single moment of time.</div></i><font color="#000080">(<i>and</i>&nbsp;(<i>not</i>&nbsp;spatiotemporal_instant)&nbsp;connected_spatiotemporal_region)</font><br></div></html><hr>
<a name='item372'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#SpatiotemporalInstant<br></b><b><i>spatiotemporal_instant</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A connected space time region at a specific moment.</div></i><font color="#000080">(<i>and</i>&nbsp;(<i>not</i>&nbsp;spatiotemporal_interval)&nbsp;connected_spatiotemporal_region)</font><br></div></html><hr>
<a name='item373'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#ScatteredSpatiotemporalRegion<br></b><b><i>scattered_spatiotemporal_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A space time region that has spatial and temporal dimensions and every spatial and temporal point of which is not connected with every other spatial and temporal point of which.</div></i><font color="#000080">(<i>and</i>&nbsp;(<i>not</i>&nbsp;connected_spatiotemporal_region)&nbsp;spatiotemporal_region)</font><br></div></html><hr>
<a name='item374'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#ProcessualEntity<br></b><b><i>processual_entity</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: An occurrent that exists in time by occurring or happening, has temporal parts and always involves and depends on some entity.</div></i><font color="#000080">(<i>and</i>&nbsp;occurrent<br>&nbsp;(<i>or</i>&nbsp;fiat_process_part&nbsp;process&nbsp;process_aggregate&nbsp;process_boundary<br>&nbsp;&nbsp;processual_context)<br>&nbsp;(<i>disjoint-with</i>&nbsp;temporal_region&nbsp;spatiotemporal_region))</font><br></div></html><hr>
<a name='item375'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#ProcessAggregate<br></b><b><i>process_aggregate</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A processual entity that is a meriological sum of processes and possesses non-connected boundaries.</div></i><font color="#000080">(<i>and</i>&nbsp;processual_entity<br>&nbsp;(<i>disjoint-with</i>&nbsp;processual_context&nbsp;process_boundary&nbsp;process&nbsp;fiat_process_part))</font><br></div></html><hr>
<a name='item376'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#ProcessualContext<br></b><b><i>processual_context</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: An occurrent consisting of a characteristic spatial shape inhering in some arrangement of other occurrents. Processual contexts are characteristically entities at or in which other occurrents can be located or occur.</div></i><font color="#000080">(<i>and</i>&nbsp;processual_entity<br>&nbsp;(<i>disjoint-with</i>&nbsp;fiat_process_part&nbsp;process_boundary&nbsp;process&nbsp;process_aggregate))</font><br></div></html><hr>
<a name='item377'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#Process<br></b><b><i>process</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A processual entity that is a maximally connected spatio-temporal whole and has bona fide beginnings and endings corresponding to real discontinuities.</div></i><font color="#000080">(<i>and</i>&nbsp;processual_entity<br>&nbsp;(<i>disjoint-with</i>&nbsp;processual_context&nbsp;process_boundary&nbsp;fiat_process_part<br>&nbsp;&nbsp;process_aggregate))</font><br></div></html><hr>
<a name='item378'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_69<br></b><b><i>protocol_application</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A protocol application is a process carried out to bring about the effect defined in the protocol applied. Every protocol_application has one or more inputs and outputs definition_source: OBI.</div></i><font color="#000080">process</font><br><i>editor_note</i>: rem1: This is an actual part of the experiment. It includes the protocol that the experimenter intends to carry out, deviations from this and unintended variables, such as the earthquake that occurred while the experiment was taking place.<br><i>editor_note</i>: protocol_application "part_of_process" some sample_preparation_for_assay in obi before split (LF 040907)<br><i>branch</i>: protocol_application<br><i>definition_citation</i>: OBI<br><i>definition</i>: A protocol application is a process carried out to bring about the effect defined in the protocol applied. Every protocol_application has one or more inputs and outputs definition_source: OBI.<br></div></html><hr>
<a name='item379'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_94<br></b><b><i>material_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">primary input and output is material</div></i><font color="#000080">protocol_application</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>editor_note</i>: we may not need this one in the long term, and just keep the children? (TC 14 June)<br><i>branch</i>: protocol_application<br><i>definition</i>: primary input and output is material<br></div></html><hr>
<a name='item380'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600006<br></b><b><i>Killing</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">difficult to place this properly - JG</div></i><font color="#000080">material_transformation</font><br><i>Source</i>: Biomaterial_branch<br><i>definition_editor</i>: Jennifer Fostel<br><i>branch</i>: protocol_application<br><i>definition_citation</i>: CEBS<br><i>alternative_term</i>: death_status_type<br><i>alternative_term</i>: killing<br><i>example_of_usage</i>: Death process (e.g. unscheduled death) is out of scope but should be added somewhere. All killings have death process as a part, but not all death processes are part of a killing.<br><i>definition</i>: A protocol application in which an animal is sacrificed.<br></div></html><hr>
<a name='item381'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600014<br></b><b><i>Material Separation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">input one material, output, two or more materials with separation in space</div></i><font color="#000080">(<i>and</i>&nbsp;material_transformation&nbsp;(has_input&nbsp;<i>exactly</i>&nbsp;1)<br>&nbsp;(has_output&nbsp;<i>min</i>&nbsp;2))</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: FACS sorting of CD8+ and CD8- T cells<br><i>definition</i>: input one material, output, two or more materials with separation in space<br></div></html><hr>
<a name='item382'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600017<br></b><b><i>Separating Based On Characteristic</i></b><i><br></i><font color="#000080">Material&middot;Separation</font><br></div></html><hr>
<a name='item383'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600019<br></b><b><i>Separating Based On Quality</i></b><i><br></i><font color="#000080">Separating&middot;Based&middot;On&middot;Characteristic</font><br></div></html><hr>
<a name='item384'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000014<br></b><b><i>separating_based_on_size</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">add the quality definition later
size is a proxy for being a cell in the chromium release assay</div></i><font color="#000080">Separating&middot;Based&middot;On&middot;Quality</font><br></div></html><hr>
<a name='item385'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600022<br></b><b><i>Resection</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The removal of all or part of an intestine via a surgical procedure</div></i><font color="#000080">Separating&middot;Based&middot;On&middot;Characteristic</font><br><i>definition_editor</i>: Alan Ruttenberg<br><i>definition_citation</i>: www.crohnsresource.com/glossary.jsp (via google define:resection)<br><i>definition</i>: The removal of all or part of an intestine via a surgical procedure<br></div></html><hr>
<a name='item386'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600018<br></b><b><i>Material Portioning</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">In which a substance is partitioned into a number of portions that are similar in composition. e.g. aliquoting.</div></i><font color="#000080">Material&middot;Separation</font><br></div></html><hr>
<a name='item387'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600005<br></b><b><i>Taking Sample From Organism</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is a protocol application in which biomaterial from a anatomical source that is located_in the organism is removed</div></i><font color="#000080">Material&middot;Separation</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>editor_note</i>: located in or part of<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: taking a sputum sample from a cancer patient, taking the spleen from a killed mouse<br><i>definition</i>: is a protocol application in which biomaterial from a anatomical source that is located_in the organism is removed<br></div></html><hr>
<a name='item388'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600009<br></b><b><i>Material Exposure</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">one input material is exposed to its environment</div></i><font color="#000080">material_transformation</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: Cells frozen at -80C.  Cy5-labeled DNA is irradiated with a laser to excite the fluorophore.<br><i>definition</i>: one input material is exposed to its environment<br></div></html><hr>
<a name='item389'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600011<br></b><b><i>Acclimatization</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A protocol application in which and object is left in an environment for some period of time, until some qualities of interest are in equilibreum with the environment.</div></i><font color="#000080">Material&middot;Exposure</font><br><i>Source</i>: Biomaterial_branch<br><i>definition_editor</i>: Jennifer Fostel<br><i>branch</i>: protocol_application<br><i>definition_citation</i>: CEBS<br><i>definition</i>: A protocol application in which and object is left in an environment for some period of time, until some qualities of interest are in equilibreum with the environment.<br></div></html><hr>
<a name='item390'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600001<br></b><b><i>material_combination</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A material transformation in which more than one input material are combined to procude one output material</div></i><font color="#000080">material_transformation</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: Adding 30ul of dNTP to PCR master mix.<br><i>definition</i>: A material transformation in which more than one input material are combined to procude one output material<br></div></html><hr>
<a name='item391'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600007<br></b><b><i>administering_substance_in_vivo</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">a material combination in which a substance is intentionally given to an organism through a specified route of administration</div></i><font color="#000080">material_combination</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>editor_note</i>: needs roles such as perturber and perturbee (children of input role). Perturb is too strong. Host might be the name for one role. Others considered: Doner, Donated, Acceptor.<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: injecting mice with 10 ug morphine intranasally, a patient taking two pills of 1 mg aspirin orally<br><i>definition</i>: a material combination in which a substance is intentionally given to an organism through a specified route of administration<br></div></html><hr>
<a name='item392'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600020<br></b><b><i>adding_reagent_to_substance</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A material combination in which a substance is intentionally given to an organism or part</div></i><font color="#000080">material_combination</font><br><i>example_of_usage</i>: <br><i>definition</i>: A material combination in which a substance is intentionally given to an organism or part<br></div></html><hr>
<a name='item393'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600000<br></b><b><i>administering_substance_to_cell culture</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">material combination where one of the input materials is a cell culture</div></i><font color="#000080">adding_reagent_to_substance</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: adding fetal calf serum to a 96 well plaste containing PBMCs from a patient<br><i>definition</i>: material combination where one of the input materials is a cell culture<br></div></html><hr>
<a name='item394'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000010<br></b><b><i>cell_chromium_labeling</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">test: check that this is returned for query: has_participant some (thing and has_role some reagent_role)</div></i><font color="#000080">(<i>and</i>&nbsp;(has_input&nbsp;<i>some</i>&nbsp;cell)&nbsp;(utilizes_reagent&nbsp;<i>some</i>&nbsp;chromium_51)<br>&nbsp;(has_output&nbsp;<i>some</i>&nbsp;(<i>and</i>&nbsp;cell&nbsp;(has_quality&nbsp;<i>some</i>&nbsp;radioactive)))<br>&nbsp;administering_substance_to_cell&middot;culture)</font><br></div></html><hr>
<a name='item395'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000013<br></b><b><i>cell_co-culture</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">and has_output cell culture and has_duration 4 hour</div></i><font color="#000080">(<i>and</i>&nbsp;(has_input&nbsp;<i>min</i>&nbsp;2)&nbsp;(has_input&nbsp;<i>all</i>&nbsp;cell)<br>&nbsp;(utilizes_instrument&nbsp;<i>some</i>&nbsp;incubator)&nbsp;material_combination)</font><br><i>example_of_usage</i>: Culturing  cytotoxic T-lymphocytes together with target cells in order to study lysis of the target cells. See chromium_release_assay<br><i>definition</i>: A material combination in which cell cultures of two or more different types are are combined and allowed to culture as one.<br></div></html><hr>
<a name='item396'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600016<br></b><b><i>material_pooling</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A material combination in which the elements being combined are mixed but do not otherwise change.</div></i><font color="#000080">material_combination</font><br><i>example_of_usage</i>: Combining spleens of 20 mice<br><i>definition</i>: A material combination in which the elements being combined are mixed but do not otherwise change.<br></div></html><hr>
<a name='item397'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#PA_to_be_reviewed<br></b><i></i><font color="#000080">protocol_application</font><br><i>branch</i>: protocol_application<br></div></html><hr>
<a name='item398'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_257<br></b><b><i>DNA_extraction</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A DNA extraction is a material transformation to purify DNA from cells. definition_source: OBI.</div></i><font color="#000080">PA_to_be_reviewed</font><br><i>branch</i>: protocol_application<br><i>definition</i>: A DNA extraction is a material transformation to purify DNA from cells. definition_source: OBI.<br></div></html><hr>
<a name='item399'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_185<br></b><b><i>imaging_assay</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An imaging assay is an assay to produce a picture of an entity. definition_source: OBI.</div></i><font color="#000080">PA_to_be_reviewed</font><br><i>branch</i>: protocol_application<br><i>definition</i>: An imaging assay is an assay to produce a picture of an entity. definition_source: OBI.<br></div></html><hr>
<a name='item400'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_157<br></b><b><i>clinical_chemistry_assay</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A clinical chemistry assay is an assay that determineds the level of an analyte in a biomaterial entity. definition_source: OBI.</div></i><font color="#000080">PA_to_be_reviewed</font><br><i>branch</i>: protocol_application<br><i>definition</i>: A clinical chemistry assay is an assay that determineds the level of an analyte in a biomaterial entity. definition_source: OBI.<br></div></html><hr>
<a name='item401'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_183<br></b><b><i>mass_spectrometry_assay</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A mass spectrometry assay is an assay to measure and analyze the molecules of a sample based on it's mass and charge. definition_source: OBI.</div></i><font color="#000080">PA_to_be_reviewed</font><br><i>branch</i>: protocol_application<br><i>definition</i>: A mass spectrometry assay is an assay to measure and analyze the molecules of a sample based on it's mass and charge. definition_source: OBI.<br></div></html><hr>
<a name='item402'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_175<br></b><b><i>?sample-population_formation_process</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">BP: place under new main class: material_enrollment or something. A protocol application where input are criteria and output is material</div></i><font color="#000080">PA_to_be_reviewed</font><br><i>editor_note</i>: rem1: for describing recruitment process, cohort formation,but also statistical models<br><i>branch</i>: protocol_application<br><i>definition</i>: A sample population formation process is a protocol application to gather entities to form a group, e.g. cohort. definition_source: OBI.<br></div></html><hr>
<a name='item403'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_253<br></b><b><i>nucleic_acid_hybridization_assay</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A nucleic acid hybridization assay is an assay to join two complementary strands of DNA or one of DNA and of RNA to form a double-stranded molecule. definition_source:www.ornl.gov/sci/techresources/Human_Genome/publicat/97pr/09gloss.html</div></i><font color="#000080">PA_to_be_reviewed</font><br><i>branch</i>: protocol_application<br><i>definition</i>: A nucleic acid hybridization assay is an assay to join two complementary strands of DNA or one of DNA and of RNA to form a double-stranded molecule. definition_source:www.ornl.gov/sci/techresources/Human_Genome/publicat/97pr/09gloss.html<br></div></html><hr>
<a name='item404'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_184<br></b><b><i>DNA_sequencing_assay</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A DNA sequencing assay is an assay to determine the order of nucleic acids in DNA. definition_source: OBI.</div></i><font color="#000080">PA_to_be_reviewed</font><br><i>branch</i>: protocol_application<br><i>definition</i>: A DNA sequencing assay is an assay to determine the order of nucleic acids in DNA. definition_source: OBI.<br></div></html><hr>
<a name='item405'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_294<br></b><b><i>observation_assay</i></b><i><br></i><font color="#000080">PA_to_be_reviewed</font><br><i>branch</i>: protocol_application<br></div></html><hr>
<a name='item406'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_182<br></b><b><i>nuclear_magnetic_resonance_assay</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A nuclear magnetic resonance assay is an assay used to identify the chemical structure of a compound or biological macromolecule. definition_source: www.nature.com/nrd/journal/v2/n5/glossary/nrd1086_glossary.html</div></i><font color="#000080">PA_to_be_reviewed</font><br><i>branch</i>: protocol_application<br><i>definition</i>: A nuclear magnetic resonance assay is an assay used to identify the chemical structure of a compound or biological macromolecule. definition_source: www.nature.com/nrd/journal/v2/n5/glossary/nrd1086_glossary.html<br></div></html><hr>
<a name='item407'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_73<br></b><b><i>sample_preparation_for_assay</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A sample_preparation_for_assay is a protocol_application including material_enrollments and biomaterial_transformations. definition_source: OBI.</div></i><font color="#000080">PA_to_be_reviewed</font><br><i>editor_note</i>: <br><i>editor_note</i>: rem1: the 'relates to a set of biological materials' covers environmental samples and data<br><i>branch</i>: protocol_application<br><i>alternative_term_citation</i>: Nutrigenomics<br><i>alternative_term</i>: study<br><i>definition</i>: A sample_preparation_for_assay is a protocol_application including material_enrollments and biomaterial_transformations. definition_source: OBI.<br></div></html><hr>
<a name='item408'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_95<br></b><b><i>data_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A data transformation is a protocol application whose output data results from transforming the input data. definition_source: OBI.</div></i><font color="#000080">protocol_application</font><br><i>branch</i>: data_transformation<br><i>branch</i>: protocol_application<br><i>definition</i>: A data transformation is a protocol application whose output data results from transforming the input data. definition_source: OBI.<br></div></html><hr>
<a name='item409'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200015<br></b><b><i>mathematical_function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A function is a procedure of associating a unique output for each input of a specified type. A function expresses the intuitive idea of deterministic dependence between two quantities, one of which is viewed as primary (the independent variable, argument of the function, or its ""input"") and the other as secondary (the dependent variable, the value of the function, or the ""output"").</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: mathematical_function<br><i>definition</i>: A function is a procedure of associating a unique output for each input of a specified type. A function expresses the intuitive idea of deterministic dependence between two quantities, one of which is viewed as primary (the independent variable, argument of the function, or its ""input"") and the other as secondary (the dependent variable, the value of the function, or the ""output"").<br></div></html><hr>
<a name='item410'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200018<br></b><b><i>logarithm</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A logarithm to base b of a number x, written log_b(x), is the exponent y that satisfies x = b^y.</div></i><font color="#000080">mathematical_function</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: logarithm<br><i>definition</i>: A logarithm to base b of a number x, written log_b(x), is the exponent y that satisfies x = b^y.<br></div></html><hr>
<a name='item411'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200016<br></b><b><i>sine</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Sine is a trigonometric function; having an arbitrary right triangle the sine of an angle can be defined as the ratio of the length of the opposite side to the length of the hypotenuse, i.e., sin(A) = opposite/hypotenuse, where opposite side is the side opposite to the angle A, and hypotenuse is the side opposite the right angle of the triangle. For angles greater than 2pi or less than -2pi; sin(A) = sin(A+2pi*k) for any integer k.</div></i><font color="#000080">mathematical_function</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: sine<br><i>definition</i>: Sine is a trigonometric function; having an arbitrary right triangle the sine of an angle can be defined as the ratio of the length of the opposite side to the length of the hypotenuse, i.e., sin(A) = opposite/hypotenuse, where opposite side is the side opposite to the angle A, and hypotenuse is the side opposite the right angle of the triangle. For angles greater than 2pi or less than -2pi; sin(A) = sin(A+2pi*k) for any integer k.<br></div></html><hr>
<a name='item412'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200019<br></b><b><i>exponential_function</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The exponential function exp(x) is defined as e^x, where e is the base of the natural logarithm (i.e., approximately 2.718281828). More general, an exponential function with base a is defined as a^x, where a is any constant greater than 0. Exponential functions multiply at a constant rate.</div></i><font color="#000080">mathematical_function</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: exponential_function<br><i>definition</i>: The exponential function exp(x) is defined as e^x, where e is the base of the natural logarithm (i.e., approximately 2.718281828). More general, an exponential function with base a is defined as a^x, where a is any constant greater than 0. Exponential functions multiply at a constant rate.<br></div></html><hr>
<a name='item413'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200017<br></b><b><i>cosine</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Cosine is a trigonometric function; having an arbitrary right triangle the cosine of an angle can be defined as the ratio of the length of the adjacent side to the length of the hypotenuse, i.e., cos(A) = adjacent/hypotenuse, where adjacent side is the side that is in contact with the angle A and the right angle, and hypotenuse is the side opposite the right angle of the triangle. For angles greater than 2pi or less than -2pi; cos(A) = cos(A+2pi*k) for any integer k.</div></i><font color="#000080">mathematical_function</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: cosine<br><i>definition</i>: Cosine is a trigonometric function; having an arbitrary right triangle the cosine of an angle can be defined as the ratio of the length of the adjacent side to the length of the hypotenuse, i.e., cos(A) = adjacent/hypotenuse, where adjacent side is the side that is in contact with the angle A and the right angle, and hypotenuse is the side opposite the right angle of the triangle. For angles greater than 2pi or less than -2pi; cos(A) = cos(A+2pi*k) for any integer k.<br></div></html><hr>
<a name='item414'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200067<br></b><b><i>unclassified</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">All classes yet to be suitably placed in a hierarchy</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: unclassified<br><i>definition</i>: All classes yet to be suitably placed in a hierarchy<br></div></html><hr>
<a name='item415'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200069<br></b><b><i>newman_global_modularity_calculation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Newman Global Modularity Calculation is the summation of network subgraph Newman modularities where For each separate subgraph in the collection of subgraphs that is the network, the Newman modularity is calculated.  Then the Global modularity is obtained by summing all of the separate Newman modularities for the subgraphs.  The Newman modularity for a subgraph is the fraction of network edges that fall within the subgraph minus the expected fraction of the same edges connecting randomly.</div></i><font color="#000080">unclassified</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>editor_note</i>: was classified under algorithm class which is not acceptable super-class<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: newman_global_modularity_calculation<br><i>definition</i>: Newman Global Modularity Calculation is the summation of network subgraph Newman modularities where For each separate subgraph in the collection of subgraphs that is the network, the Newman modularity is calculated.  Then the Global modularity is obtained by summing all of the separate Newman modularities for the subgraphs.  The Newman modularity for a subgraph is the fraction of network edges that fall within the subgraph minus the expected fraction of the same edges connecting randomly.<br></div></html><hr>
<a name='item416'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200068<br></b><b><i>intra-subgraph_connectivity_calculation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The intra-subgraph connectivity calculation is where averaging the vertex connectivity across all vertices in a subgraph.  For all rows (vertices) of the adjacency matrix, the number of direct connections (numbers > 0 as opposed to = 0) is summed and divided by the number of rows (vertices)</div></i><font color="#000080">unclassified</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: intra-subgraph_connectivity_calculation<br><i>definition</i>: The intra-subgraph connectivity calculation is where averaging the vertex connectivity across all vertices in a subgraph.  For all rows (vertices) of the adjacency matrix, the number of direct connections (numbers > 0 as opposed to = 0) is summed and divided by the number of rows (vertices)<br></div></html><hr>
<a name='item417'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200070<br></b><b><i>edge_weighting</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Edge weighting is the substitution or transformation of edge length using numerical data.  Data input include a symmetric adjacency matrix for a network and a second data set, for example a list of interactor pairs and a confidence score associated with the experimental detection of each pair's interaction.  Each element in the adjacency matrix is transformed or replaced with the corresponding number in the second data set.  Output data are a modified adjacency matrix reflecting the transformed state of the network.  </div></i><font color="#000080">unclassified</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>editor_note</i>: was classified under algorithm class which is not acceptable super-class<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: edge_weighting<br><i>definition</i>: Edge weighting is the substitution or transformation of edge length using numerical data.  Data input include a symmetric adjacency matrix for a network and a second data set, for example a list of interactor pairs and a confidence score associated with the experimental detection of each pair's interaction.  Each element in the adjacency matrix is transformed or replaced with the corresponding number in the second data set.  Output data are a modified adjacency matrix reflecting the transformed state of the network.  <br></div></html><hr>
<a name='item418'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200046<br></b><b><i>filtering</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Filtering is a process of data transformation where any observations that do not fulfill a preformulated condition are excluded from the data.</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: filtering<br><i>definition</i>: Filtering is a process of data transformation where any observations that do not fulfill a preformulated condition are excluded from the data.<br></div></html><hr>
<a name='item419'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200048<br></b><b><i>structural_filtering</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Structural filtering is the removal or combination of network vertices or edges from the data set.  Operations such as removing edges or vertices from the network or combining connected vertices into a single vertex to represent a collection of vertices.</div></i><font color="#000080">filtering</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: structural_filtering<br><i>definition</i>: Structural filtering is the removal or combination of network vertices or edges from the data set.  Operations such as removing edges or vertices from the network or combining connected vertices into a single vertex to represent a collection of vertices.<br></div></html><hr>
<a name='item420'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200049<br></b><b><i>low_intensity_filter</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A transformation method that involves removal of low intensity values in a data set. For microarrays and gels the threshold for exclusion is usually based on some estimation of local or global background intensity</div></i><font color="#000080">filtering</font><br><i>definition_editor</i>: Philippe Rocca-Serra<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: low_intensity_filter<br><i>definition</i>: A transformation method that involves removal of low intensity values in a data set. For microarrays and gels the threshold for exclusion is usually based on some estimation of local or global background intensity<br></div></html><hr>
<a name='item421'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200047<br></b><b><i>edge_filtering</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Removal of edges from the network, generally based on confidence criteria.  To remove an edge from the network data, a non-zero number in the adjacency matrix would simply be changed to zero.  Selection of edges for such an operation could be done arbitrarily but is more typically useful when edges are removed based on a cutoff parameter if the adjacency matrix contains widely varied numbers across a range of values.</div></i><font color="#000080">filtering</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: EPRI; PVM<br><i>preferred_term</i>: edge_filtering<br><i>definition</i>: Removal of edges from the network, generally based on confidence criteria.  To remove an edge from the network data, a non-zero number in the adjacency matrix would simply be changed to zero.  Selection of edges for such an operation could be done arbitrarily but is more typically useful when edges are removed based on a cutoff parameter if the adjacency matrix contains widely varied numbers across a range of values.<br></div></html><hr>
<a name='item422'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200034<br></b><b><i>statistical_error</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A calculation or estimation of difference between the input observed data value(s) and the true value(s)</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: Type I Error<br><i>preferred_term</i>: statistical_error<br><i>definition</i>: A calculation or estimation of difference between the input observed data value(s) and the true value(s)<br></div></html><hr>
<a name='item423'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200036<br></b><b><i>type_II_error</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">This outputs a measure of the error that a false null hypothesis failed to be rejected</div></i><font color="#000080">statistical_error</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: type_II_error<br><i>definition</i>: This outputs a measure of the error that a false null hypothesis failed to be rejected<br></div></html><hr>
<a name='item424'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200035<br></b><b><i>type_I_error</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">This outputs a measure of the error that a true null hypothesis is incorrectly rejected</div></i><font color="#000080">statistical_error</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: type_I_error<br><i>definition</i>: This outputs a measure of the error that a true null hypothesis is incorrectly rejected<br></div></html><hr>
<a name='item425'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200052<br></b><b><i>probabilistic_algorithm</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A probabilistic algorithm is one which involves an element of probability or randomness in the transformation of the data.</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: probabilistic_algorithm<br><i>definition</i>: A probabilistic algorithm is one which involves an element of probability or randomness in the transformation of the data.<br></div></html><hr>
<a name='item426'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200053<br></b><b><i>expectation_maximization</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">EM is a probabilistic algorithm used to estimate the maximum likelihood of parameters from existing data where the model involves unobserved latent variables. The input to this method is the data model for which the estimation is performed over and the output is an approximateed probability function.</div></i><font color="#000080">probabilistic_algorithm</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: expectation_maximization<br><i>definition</i>: EM is a probabilistic algorithm used to estimate the maximum likelihood of parameters from existing data where the model involves unobserved latent variables. The input to this method is the data model for which the estimation is performed over and the output is an approximateed probability function.<br></div></html><hr>
<a name='item427'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200037<br></b><b><i>partitioning_method</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A partitioning method is one which divides a data set into several subsets based on specific condition(s). This method takes as input an unpartitioned (superset) data model and produces as output a partitioned data set (two or more subsets).</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: partitioning_method<br><i>definition</i>: A partitioning method is one which divides a data set into several subsets based on specific condition(s). This method takes as input an unpartitioned (superset) data model and produces as output a partitioned data set (two or more subsets).<br></div></html><hr>
<a name='item428'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200038<br></b><b><i>classification_method</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Clustering is the partitioning of a data set into related subsets with a known assigned label, the input is a single superset of  (unclassified) data and the output is partitioned data with instances assigned a class label</div></i><font color="#000080">partitioning_method</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: classification_method<br><i>definition</i>: Clustering is the partitioning of a data set into related subsets with a known assigned label, the input is a single superset of  (unclassified) data and the output is partitioned data with instances assigned a class label<br></div></html><hr>
<a name='item429'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200059<br></b><b><i>mhc_binding_prediction</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An MHC binding prediction takes an input of a biomaterial sequence and through an analysis of this sequence, produces as output a prediction of the likelihood that the biomaterial will bind to an MHC molecule.</div></i><font color="#000080">classification_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: mhc_binding_prediction<br><i>definition</i>: An MHC binding prediction takes an input of a biomaterial sequence and through an analysis of this sequence, produces as output a prediction of the likelihood that the biomaterial will bind to an MHC molecule.<br></div></html><hr>
<a name='item430'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200058<br></b><b><i>b_cell_epitope_prediction</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A B cell epitope prediction takes as input an antigen sequence, and through an analysis of this sequence, produces as output a prediction of the likelihood the biomaterial is a B Cell Epitope.</div></i><font color="#000080">classification_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: b_cell_epitope_prediction<br><i>definition</i>: A B cell epitope prediction takes as input an antigen sequence, and through an analysis of this sequence, produces as output a prediction of the likelihood the biomaterial is a B Cell Epitope.<br></div></html><hr>
<a name='item431'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200060<br></b><b><i>t_cell_epitope_prediction</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A T cell epitope prediction takes as input an antigen sequence, and through an analysis of this sequence, produces as output a prediction of the likelihood the biomaterial is a T cell epitope.</div></i><font color="#000080">classification_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: t_cell_epitope_prediction<br><i>definition</i>: A T cell epitope prediction takes as input an antigen sequence, and through an analysis of this sequence, produces as output a prediction of the likelihood the biomaterial is a T cell epitope.<br></div></html><hr>
<a name='item432'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200039<br></b><b><i>clustering_method</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Clustering is the partitioning of a data set into related subsets (clusters), the input is a single cluster of data and the output is partitioned data or information on cluster labels</div></i><font color="#000080">partitioning_method</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: clustering_method<br><i>definition</i>: Clustering is the partitioning of a data set into related subsets (clusters), the input is a single cluster of data and the output is partitioned data or information on cluster labels<br></div></html><hr>
<a name='item433'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200042<br></b><b><i>hierarchical_clustering</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Hierarchical clustering is a method which can be used to cluster an input data set by either gradually merging data points (agglomerative of bottom-up) or by dividing superclusters (divisive or top-down).The output of this method is a hierarchical mapping of data points to clusters.</div></i><font color="#000080">clustering_method</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: hierarchical_clustering<br><i>definition</i>: Hierarchical clustering is a method which can be used to cluster an input data set by either gradually merging data points (agglomerative of bottom-up) or by dividing superclusters (divisive or top-down).The output of this method is a hierarchical mapping of data points to clusters.<br></div></html><hr>
<a name='item434'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200043<br></b><b><i>average_linkage</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Average Linkage is a hierarchical clustering method in which the distance between clusters is calculated as the average distance from the members of one cluster to the members of another cluster. The output of this method is a hierarchical mapping of data points to clusters with an associated distance metric.</div></i><font color="#000080">hierarchical_clustering</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: average_linkage<br><i>definition</i>: Average Linkage is a hierarchical clustering method in which the distance between clusters is calculated as the average distance from the members of one cluster to the members of another cluster. The output of this method is a hierarchical mapping of data points to clusters with an associated distance metric.<br></div></html><hr>
<a name='item435'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200045<br></b><b><i>simple_linkage</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Simple linkage is a hierarchical clustering method in which the distance is calculated as the shortest distance from any member of one cluster to any member of the other cluster. The output of this method is a hierarchical mapping of data points to clusters with an associated distance metric.</div></i><font color="#000080">hierarchical_clustering</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: simple_linkage<br><i>definition</i>: Simple linkage is a hierarchical clustering method in which the distance is calculated as the shortest distance from any member of one cluster to any member of the other cluster. The output of this method is a hierarchical mapping of data points to clusters with an associated distance metric.<br></div></html><hr>
<a name='item436'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200044<br></b><b><i>complete_linkage</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Complete linkage is a hierarchical clustering method in which the distance is calculated as the longest distance from any member of one cluster to any member of the other cluster. The output of this method is a hierarchical mapping of data points to clusters with an associated distance metric.</div></i><font color="#000080">hierarchical_clustering</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: complete_linkage<br><i>definition</i>: Complete linkage is a hierarchical clustering method in which the distance is calculated as the longest distance from any member of one cluster to any member of the other cluster. The output of this method is a hierarchical mapping of data points to clusters with an associated distance metric.<br></div></html><hr>
<a name='item437'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200041<br></b><b><i>k-means_clustering</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">K-Means clustering is a method for partitioning data into K specified clusters.  The input is unclustered data with a value for k and the output is a cluster label assigned to each data point, often with a metric value, such as distance to cluster centroid.</div></i><font color="#000080">clustering_method</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: k-means_clustering<br><i>definition</i>: K-Means clustering is a method for partitioning data into K specified clusters.  The input is unclustered data with a value for k and the output is a cluster label assigned to each data point, often with a metric value, such as distance to cluster centroid.<br></div></html><hr>
<a name='item438'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200040<br></b><b><i>modular_decomposition</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Molecular decompostion is the partition of a network into distinct subgraphs for the purpose of identifying functional clusters.  The network data is run through any of several existing algorithms designed to partition a network into distinct subgraphs for the purpose of isolating groups of functionally linked biological elements such as proteins.</div></i><font color="#000080">partitioning_method</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: modular_decomposition<br><i>definition</i>: Molecular decompostion is the partition of a network into distinct subgraphs for the purpose of identifying functional clusters.  The network data is run through any of several existing algorithms designed to partition a network into distinct subgraphs for the purpose of isolating groups of functionally linked biological elements such as proteins.<br></div></html><hr>
<a name='item439'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_260<br></b><b><i>data_normalization</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A data normalization is data transformation to adjust for differences among data from varying sources in order to create a common basis for comparison. definition_source:www.hud.gov/offices/pih/programs/ph/phecc/definitions.cfm</div></i><font color="#000080">data_transformation</font><br><i>branch</i>: protocol_application<br><i>definition</i>: A data normalization is data transformation to adjust for differences among data from varying sources in order to create a common basis for comparison. definition_source:www.hud.gov/offices/pih/programs/ph/phecc/definitions.cfm<br></div></html><hr>
<a name='item440'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200054<br></b><b><i>averaging_method</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">To be defined</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: none<br><i>definition_citation</i>: none<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: averaging_method<br><i>definition</i>: To be defined<br></div></html><hr>
<a name='item441'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200055<br></b><b><i>dye_swap_merge</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A dye_swap_merge is a data transformation which takes as input microarray data from multiple hybridizations performed using two channel technology where the merging is on a feature (spot) by feature basis. The input is typically raw data obtained from the scanning software from each hybridization and the output is a merged file where spots that fail consistency checks are flagged.</div></i><font color="#000080">averaging_method</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: dye_swap_merge<br><i>definition</i>: A dye_swap_merge is a data transformation which takes as input microarray data from multiple hybridizations performed using two channel technology where the merging is on a feature (spot) by feature basis. The input is typically raw data obtained from the scanning software from each hybridization and the output is a merged file where spots that fail consistency checks are flagged.<br></div></html><hr>
<a name='item442'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200056<br></b><b><i>moving_average</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A type of data transformation in which an average, usually the mean, is calculated across values within a sliding window. e.g. The moving average of signal intensity, or array CGH, data may be used in normalization.</div></i><font color="#000080">averaging_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: moving_average<br><i>definition</i>: A type of data transformation in which an average, usually the mean, is calculated across values within a sliding window. e.g. The moving average of signal intensity, or array CGH, data may be used in normalization.<br></div></html><hr>
<a name='item443'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200057<br></b><b><i>replicate_analysis</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A transformation method in which replicate intensity or ratio data sets are combined. Analysis of the mean and standard deviation values for replicate data points can be used to identify and potentially exclude low quality data.</div></i><font color="#000080">averaging_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: replicate_analysis<br><i>definition</i>: A transformation method in which replicate intensity or ratio data sets are combined. Analysis of the mean and standard deviation values for replicate data points can be used to identify and potentially exclude low quality data.<br></div></html><hr>
<a name='item444'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200031<br></b><b><i>statistical_significance_method</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A statisical significance method is one which takes one or more inputs and produces a measure of the probabilty of a hypothesis being tested as output</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: p-value<br><i>preferred_term</i>: statistical_significance_method<br><i>definition</i>: A statisical significance method is one which takes one or more inputs and produces a measure of the probabilty of a hypothesis being tested as output<br></div></html><hr>
<a name='item445'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200032<br></b><b><i>differentially_expressed</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">This method takes the expressions of a gene under two or more conditions as input and produces output which represents whether or not they are statistically significantly different</div></i><font color="#000080">statistical_significance_method</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: differentially_expressed<br><i>definition</i>: This method takes the expressions of a gene under two or more conditions as input and produces output which represents whether or not they are statistically significantly different<br></div></html><hr>
<a name='item446'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200033<br></b><b><i>p-value</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A method which outputs a probability that the input, some observed result, is at least as extreme as that obtained, assuming the null hypothesis is true, as a result of chance alone</div></i><font color="#000080">statistical_significance_method</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: p-value<br><i>definition</i>: A method which outputs a probability that the input, some observed result, is at least as extreme as that obtained, assuming the null hypothesis is true, as a result of chance alone<br></div></html><hr>
<a name='item447'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200065<br></b><b><i>network-independent_data_superimposition</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Combination of network data and additional data to aid in network analysis</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: network-independent_data_superimposition<br><i>definition</i>: Combination of network data and additional data to aid in network analysis<br></div></html><hr>
<a name='item448'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200050<br></b><b><i>data_reduction_method</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A data reduction method is one which receives a data model as input and transforms or manipulates this in some way as to produce a reduced or simplified data model as output.</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: data_reduction_method<br><i>definition</i>: A data reduction method is one which receives a data model as input and transforms or manipulates this in some way as to produce a reduced or simplified data model as output.<br></div></html><hr>
<a name='item449'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200051<br></b><b><i>principal_component_analysis</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Principal component analysis is a data transformation process for reducing multidimensional datasets (input)  to lower dimensions (output of the method).</div></i><font color="#000080">data_reduction_method</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: principal_component_analysis<br><i>definition</i>: Principal component analysis is a data transformation process for reducing multidimensional datasets (input)  to lower dimensions (output of the method).<br></div></html><hr>
<a name='item450'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200001<br></b><b><i>mathematical_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A mathematical transformation is a procedure relying on mathematical function to convert a value into another</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: mathematical_transformation<br><i>definition</i>: A mathematical transformation is a procedure relying on mathematical function to convert a value into another<br></div></html><hr>
<a name='item451'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200014<br></b><b><i>split-scale_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Scaling using a split-scale function. The split-scale function consists of a logarithmic transformation function applied to high values and a linear transformation function applied to low values. The split scale transformation is defined as if(x ? t):split-scale(x, a, b, c, r, d) = a*x + b, otherwise:split-scale(x, a, b, c, r, d) = log10 (c*x) * r/d, where x is the source value that is being transformed and a, b, c, r, and d are real constants. The transition point and the constants may be chosen so that the slope (first derivative) of the resulting split-scale transformation function is continuous as shown in Transformation-ML specification. Alternativelly, the transition point may be determined by minimizing the absolute deviation of the variance of log ratios.</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: split-scale_transformation<br><i>definition</i>: Scaling using a split-scale function. The split-scale function consists of a logarithmic transformation function applied to high values and a linear transformation function applied to low values. The split scale transformation is defined as if(x ? t):split-scale(x, a, b, c, r, d) = a*x + b, otherwise:split-scale(x, a, b, c, r, d) = log10 (c*x) * r/d, where x is the source value that is being transformed and a, b, c, r, and d are real constants. The transition point and the constants may be chosen so that the slope (first derivative) of the resulting split-scale transformation function is continuous as shown in Transformation-ML specification. Alternativelly, the transition point may be determined by minimizing the absolute deviation of the variance of log ratios.<br></div></html><hr>
<a name='item452'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200002<br></b><b><i>logistic-log_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Mathematical function (curve-fitting method) used to model serially diluted standard reference serum to form characteristic curves; typically used in the enzyme-linked immunosorbent assay (ELISA). The transformation is computed as OD = d + {(a - d)I1 + (dilution/c)b}, where a and d represent the upper and lower asymptotes, respectively, of the curve and correspond to the theoretical OD of the assay at infinite and zero concentrations, respectively; c is the dilution associated with the point of symmetry of the sigmoid and is located at the midpoint of the assay found at the inflection point of the curve; b is a curvature parameter and is related to the slope of the curve.</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: logistic-log_transformation<br><i>definition</i>: Mathematical function (curve-fitting method) used to model serially diluted standard reference serum to form characteristic curves; typically used in the enzyme-linked immunosorbent assay (ELISA). The transformation is computed as OD = d + {(a - d)I1 + (dilution/c)b}, where a and d represent the upper and lower asymptotes, respectively, of the curve and correspond to the theoretical OD of the assay at infinite and zero concentrations, respectively; c is the dilution associated with the point of symmetry of the sigmoid and is located at the midpoint of the assay found at the inflection point of the curve; b is a curvature parameter and is related to the slope of the curve.<br></div></html><hr>
<a name='item453'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200009<br></b><b><i>linear_combination</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A mathematical combination using a first degree polynomial to linearly combine values. The first degree polynomial is defined as f(x_1, x_2, ..., x_n, a1, a2, ..., a_n, b) = a_1*x_1+a_2*x_2+ ... +a_n*x_n + b, where a_1 ... a_n are real constants, b is a real constant, x_1 ... x_n are source values that are being combined.</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: linear_combination<br><i>definition</i>: A mathematical combination using a first degree polynomial to linearly combine values. The first degree polynomial is defined as f(x_1, x_2, ..., x_n, a1, a2, ..., a_n, b) = a_1*x_1+a_2*x_2+ ... +a_n*x_n + b, where a_1 ... a_n are real constants, b is a real constant, x_1 ... x_n are source values that are being combined.<br></div></html><hr>
<a name='item454'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200004<br></b><b><i>log-log_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Mathematical function (curve-fitting method) used to model serially diluted standard reference serum to form characteristic curves; typically used in the enzyme-linked immunosorbent assay (ELISA). It transform the response variable y to log(y) and the explanatory variable x to log(x).</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: log-log_transformation<br><i>definition</i>: Mathematical function (curve-fitting method) used to model serially diluted standard reference serum to form characteristic curves; typically used in the enzyme-linked immunosorbent assay (ELISA). It transform the response variable y to log(y) and the explanatory variable x to log(x).<br></div></html><hr>
<a name='item455'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200007<br></b><b><i>box-cox_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Transformation of data according to the methods of Box and Cox as described in the article Box, G. E. P. and Cox, D.R. (1964) An analysis of transformations. Journal of Royal Statistical Society, Series B, vol. 26, pp. 211-246.</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: box-cox_transformation<br><i>definition</i>: Transformation of data according to the methods of Box and Cox as described in the article Box, G. E. P. and Cox, D.R. (1964) An analysis of transformations. Journal of Royal Statistical Society, Series B, vol. 26, pp. 211-246.<br></div></html><hr>
<a name='item456'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200013<br></b><b><i>quadratic_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Scaling using a quadratic function. The quadratic function is defined as quadratic(x, a, b, c) = a*x*x + b*x + c, where a, b, c are real constants and x is the source value that is being transformed.</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: quadratic_transformation<br><i>definition</i>: Scaling using a quadratic function. The quadratic function is defined as quadratic(x, a, b, c) = a*x*x + b*x + c, where a, b, c are real constants and x is the source value that is being transformed.<br></div></html><hr>
<a name='item457'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200008<br></b><b><i>hyperlog_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Parameter scaling using a hyperlog function. The hyperlog (HL) function is defined as follows:HL(parameter, b, d, r) = root(EH(y, b, d, r) ? parameter), where root() is a standard root finding algorithm (e.g., Newton's method) that finds y such that EH(y, b, d, r) = parameter. The EH function is defined as follows:if(y ? 0):EH(y, b, d, r) = 10^(y * d / r) + b * (d / r) * y - 1; otherwise:EH(y, b, d, r) = - 10^(-y * d / r) + b * (d / r) * y + 1, where r, d, and b are positive real constants and parameter is the source parameter. The hyperlog function is defined in Bagwell C.B. (2006). Hyperlog - a flexible log-like transform for negative, zero, and positive valued data. Cytometry A 64, 34-42.</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: hyperlog_transformation<br><i>definition</i>: Parameter scaling using a hyperlog function. The hyperlog (HL) function is defined as follows:HL(parameter, b, d, r) = root(EH(y, b, d, r) ? parameter), where root() is a standard root finding algorithm (e.g., Newton's method) that finds y such that EH(y, b, d, r) = parameter. The EH function is defined as follows:if(y ? 0):EH(y, b, d, r) = 10^(y * d / r) + b * (d / r) * y - 1; otherwise:EH(y, b, d, r) = - 10^(-y * d / r) + b * (d / r) * y + 1, where r, d, and b are positive real constants and parameter is the source parameter. The hyperlog function is defined in Bagwell C.B. (2006). Hyperlog - a flexible log-like transform for negative, zero, and positive valued data. Cytometry A 64, 34-42.<br></div></html><hr>
<a name='item458'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200010<br></b><b><i>linear_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Scaling using a linear function defined as linear(x, a, b) = a * x + b, where a, b are real constants and x is the source value that is being transformed.</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: linear_transformation<br><i>definition</i>: Scaling using a linear function defined as linear(x, a, b) = a * x + b, where a, b are real constants and x is the source value that is being transformed.<br></div></html><hr>
<a name='item459'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200003<br></b><b><i>logit-log_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Mathematical function (curve-fitting method) used to model serially diluted standard reference serum to form characteristic curves; typically used in the enzyme-linked immunosorbent assay (ELISA).</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: logit-log_transformation<br><i>definition</i>: Mathematical function (curve-fitting method) used to model serially diluted standard reference serum to form characteristic curves; typically used in the enzyme-linked immunosorbent assay (ELISA).<br></div></html><hr>
<a name='item460'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200006<br></b><b><i>biexponential_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Scaling using a bi-exponential function. The bi-exponential function is defined as BiEx(x, a, b, c, d, f) = root(B(y, a, b, c, d, f) - x), where root() is a standard root finding algorithm (e.g., Newton's method) that finds y such that B(y, a, b, c, d, f) = parameter. The B function is defined as B(y, a, b, c, d, f) = a * e^(b * y) - c * e^(-d * y) + f, where e is the base of natural logarithm, a, b, c, d are positive real constants, f is a real constant and x is the source value that is being transformed.</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: biexponential_transformation<br><i>definition</i>: Scaling using a bi-exponential function. The bi-exponential function is defined as BiEx(x, a, b, c, d, f) = root(B(y, a, b, c, d, f) - x), where root() is a standard root finding algorithm (e.g., Newton's method) that finds y such that B(y, a, b, c, d, f) = parameter. The B function is defined as B(y, a, b, c, d, f) = a * e^(b * y) - c * e^(-d * y) + f, where e is the base of natural logarithm, a, b, c, d are positive real constants, f is a real constant and x is the source value that is being transformed.<br></div></html><hr>
<a name='item461'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200011<br></b><b><i>logicle_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Scaling using a logicle function. The logicle function is defined as logicle(x, T, w, m) = root(S(y, T, w, m) - x), where root() is a standard root finding algorithm (e.g., Newton's method) that finds y such that S(y, T, w, m) = x. The S function is defined as follows:if(y ? w):S(y, T, w, m) = Te^(-(m-w)) * (e^(y-w) - p^2*e^(-(y-w)/p) + p^2 - 1), otherwise:S(y, T, w, m) = - S(w - y, T, w, m), where the operands T, and m are positive real constants, w is a non-negative real constant, e is the base of natural logarithm and x is the value that is being transformed. The logicle function is defined in Parks D.R., Roederer M., Moore W.A. (2006). A new ""Logicle"" display method avoids deceptive effects of logarithmic scaling for low signals and compensated data. Cytometry A 69, 541-551. Logicle functions create a subset of biexponential functions (they are parameterized a different way).</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: logicle_transformation<br><i>definition</i>: Scaling using a logicle function. The logicle function is defined as logicle(x, T, w, m) = root(S(y, T, w, m) - x), where root() is a standard root finding algorithm (e.g., Newton's method) that finds y such that S(y, T, w, m) = x. The S function is defined as follows:if(y ? w):S(y, T, w, m) = Te^(-(m-w)) * (e^(y-w) - p^2*e^(-(y-w)/p) + p^2 - 1), otherwise:S(y, T, w, m) = - S(w - y, T, w, m), where the operands T, and m are positive real constants, w is a non-negative real constant, e is the base of natural logarithm and x is the value that is being transformed. The logicle function is defined in Parks D.R., Roederer M., Moore W.A. (2006). A new ""Logicle"" display method avoids deceptive effects of logarithmic scaling for low signals and compensated data. Cytometry A 69, 541-551. Logicle functions create a subset of biexponential functions (they are parameterized a different way).<br></div></html><hr>
<a name='item462'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200005<br></b><b><i>nonlinear_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Mathematical transformation that cannot be expressed as a linear function, i.e., it CANNOT be expressed in the form of Y = AX + B, where Y is the result vector, A is the coefficient matrix, X is the source vector, and B is a vector of additive coefficients.</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: nonlinear_transformation<br><i>definition</i>: Mathematical transformation that cannot be expressed as a linear function, i.e., it CANNOT be expressed in the form of Y = AX + B, where Y is the result vector, A is the coefficient matrix, X is the source vector, and B is a vector of additive coefficients.<br></div></html><hr>
<a name='item463'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200012<br></b><b><i>scaling</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Scaling is a mathematical transformation that transforms a single value into another single value; typically, such a transformation may be expressed as a function having the domain and range in the set of real numbers (or a subset of real numbers).</div></i><font color="#000080">mathematical_transformation</font><br><i>definition_editor</i>: Ryan Brinkman<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: scaling<br><i>definition</i>: Scaling is a mathematical transformation that transforms a single value into another single value; typically, such a transformation may be expressed as a function having the domain and range in the set of real numbers (or a subset of real numbers).<br></div></html><hr>
<a name='item464'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200020<br></b><b><i>normalization_method</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Normalization is a mathematical process that adjusts for differences among data from varying sources in order to create a common basis for comparison in the data, such as microarray probe set expression values. Input is unnormalized (sometimes refered to as raw data) and output is normalised data.</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: www.hud.gov/offices/pih/programs/ph/phecc/definitions.cfm<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: normalization_method<br><i>definition</i>: Normalization is a mathematical process that adjusts for differences among data from varying sources in order to create a common basis for comparison in the data, such as microarray probe set expression values. Input is unnormalized (sometimes refered to as raw data) and output is normalised data.<br></div></html><hr>
<a name='item465'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200024<br></b><b><i>loess_group_normalization</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Loess (or Lowess) group normalization is a transformation of two or more groups of raw data as input, into a normalized output form. This transformation is calculated by fitting a line to local neighbourhood of each data point and aggregating into a curve. Example use to remove intensity dependent bias from dies in microarray experiments.</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: loess_group_normalization<br><i>definition</i>: Loess (or Lowess) group normalization is a transformation of two or more groups of raw data as input, into a normalized output form. This transformation is calculated by fitting a line to local neighbourhood of each data point and aggregating into a curve. Example use to remove intensity dependent bias from dies in microarray experiments.<br></div></html><hr>
<a name='item466'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200029<br></b><b><i>mean_log_normalization</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A mean_log_normalization method is a data transformation that normalizes data based on the mean-log_centering of the data.</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: mean_log_normalization<br><i>definition</i>: A mean_log_normalization method is a data transformation that normalizes data based on the mean-log_centering of the data.<br></div></html><hr>
<a name='item467'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200028<br></b><b><i>quantile_normalization</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A normalization procedure which takes into account data repartition among the n number of quantile defined as parameter</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: quantile_normalization<br><i>definition</i>: A normalization procedure which takes into account data repartition among the n number of quantile defined as parameter<br></div></html><hr>
<a name='item468'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200021<br></b><b><i>linlog_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A linlog transformation is a data transformation whereby combination of a linear and logarithm transformations are used with the aim of removing the intensity-dependence from the variance in microarray data processing. Low intensities are linearly scaled while high intensities are logarithmically scaled. The transition point is determined by minimizing the absolute deviation of the variance of log ratios. Input data is raw microarray data, outout is linear or log transformed data depending on relative intensities of the input data.</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: Reference: Cui, Kerr and Churchill. Transformations for microarray data. Statistical Applications in Genetics and Molecular biology. Volume 2, Issue 1 2003. <br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: linlog_transformation<br><i>definition</i>: A linlog transformation is a data transformation whereby combination of a linear and logarithm transformations are used with the aim of removing the intensity-dependence from the variance in microarray data processing. Low intensities are linearly scaled while high intensities are logarithmically scaled. The transition point is determined by minimizing the absolute deviation of the variance of log ratios. Input data is raw microarray data, outout is linear or log transformed data depending on relative intensities of the input data.<br></div></html><hr>
<a name='item469'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200026<br></b><b><i>total_intensity_normalization_single</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Total intensity normalization single performs a transformation of input data, consisting of a single assay's data, so that intensity values are rescaled relative to a subset or feature of this assy.  The output is the normalised data set.</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: total_intensity_normalization_single<br><i>definition</i>: Total intensity normalization single performs a transformation of input data, consisting of a single assay's data, so that intensity values are rescaled relative to a subset or feature of this assy.  The output is the normalised data set.<br></div></html><hr>
<a name='item470'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200022<br></b><b><i>arsinh_transformation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An arsinh transformation is a data transformation used to stabilize the variance of microarray data based on the assumption of a quadratic relationshio between variance and intensity of microarray signals at the original scale. Input data is raw microarray data, Outout data resembles log log tranformed data for high intensity values and contracts to zero for low intensity values.</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: Reference: Cui, Kerr and Churchill. Transformations for microarray data. Statistical Applications in Genetics and Molecular biology. Volume 2, Issue 1 2003. Huber et al., PUBMED : 12169536 <br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: arsinh_transformation<br><i>definition</i>: An arsinh transformation is a data transformation used to stabilize the variance of microarray data based on the assumption of a quadratic relationshio between variance and intensity of microarray signals at the original scale. Input data is raw microarray data, Outout data resembles log log tranformed data for high intensity values and contracts to zero for low intensity values.<br></div></html><hr>
<a name='item471'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200025<br></b><b><i>loess_scaled_group_normalization</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Loess (or Lowess) scaled group normalization is a scale adjustment of input loess group normalized data, the resulting output being the scale normalised data.</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: James Malone<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: loess_scaled_group_normalization<br><i>definition</i>: Loess (or Lowess) scaled group normalization is a scale adjustment of input loess group normalized data, the resulting output being the scale normalised data.<br></div></html><hr>
<a name='item472'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200030<br></b><b><i>median_log_normalization</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A median_log_normalization method is a data transformation that normalizes data based on the median-log_centering of the data.</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: median_log_normalization<br><i>definition</i>: A median_log_normalization method is a data transformation that normalizes data based on the median-log_centering of the data.<br></div></html><hr>
<a name='item473'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200023<br></b><b><i>loess_global_normalization</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Weighted linear regression analysis as  a function of the log10 intensity</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: loess_global_normalization<br><i>definition</i>: Weighted linear regression analysis as  a function of the log10 intensity<br></div></html><hr>
<a name='item474'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200027<br></b><b><i>total_intensity_normalization_paired</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Total intensity normalization paired performs a transformation of input data, consisting of paired assay data, so that intensity values are rescaled relative to one of the assays.  The output is the normalised paired data sets.</div></i><font color="#000080">normalization_method</font><br><i>definition_editor</i>: Helen Parkinson<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: total_intensity_normalization_paired<br><i>definition</i>: Total intensity normalization paired performs a transformation of input data, consisting of paired assay data, so that intensity values are rescaled relative to one of the assays.  The output is the normalised paired data sets.<br></div></html><hr>
<a name='item475'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200061<br></b><b><i>data_imputation_method</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">To be defined</div></i><font color="#000080">data_transformation</font><br><i>definition_editor</i>: none<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: data_imputation_method<br><i>definition</i>: To be defined<br></div></html><hr>
<a name='item476'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200064<br></b><b><i>centroid_mass_spectrun</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A centroid mass spectrum is a data transformation that is generated by drawing a line from the tip of each peak to the base of the graph.  Centroid mass spectra contain discrete peaks of zero width.</div></i><font color="#000080">data_imputation_method</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: centroid_mass_spectrun<br><i>definition</i>: A centroid mass spectrum is a data transformation that is generated by drawing a line from the tip of each peak to the base of the graph.  Centroid mass spectra contain discrete peaks of zero width.<br></div></html><hr>
<a name='item477'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200063<br></b><b><i>characteristic_path_length_calculation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Quantifying subgraph navigability based on shortest-path length averaged over all pairs of subgraph vertices</div></i><font color="#000080">data_imputation_method</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: characteristic_path_length_calculation<br><i>definition</i>: Quantifying subgraph navigability based on shortest-path length averaged over all pairs of subgraph vertices<br></div></html><hr>
<a name='item478'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_200062<br></b><b><i>continuum_mass_spectrum</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A continuum mass spectrum is a data transformation that contains the full profile of the detected signals for a given ion.</div></i><font color="#000080">data_imputation_method</font><br><i>definition_editor</i>: Tina Hernandez-Boussard<br><i>definition_citation</i>: editor<br><i>curation_status</i>: pending_final_vetting<br><i>example_of_usage</i>: example to be added<br><i>preferred_term</i>: continuum_mass_spectrum<br><i>definition</i>: A continuum mass spectrum is a data transformation that contains the full profile of the detected signals for a given ion.<br></div></html><hr>
<a name='item479'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_70<br></b><b><i>assay</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An assay is a protocol application whose input includes material and output includes data definition_source: OBI.</div></i><font color="#000080">protocol_application</font><br><i>editor_note</i>: AltDef: An assay is a protocol application which is a process of measuring. rem1: Imaging is covered by this definition. rem2: material:  used in the context of studying some biological problem, example: hybridization<br><i>branch</i>: protocol_application<br><i>definition</i>: An assay is a protocol application whose input includes material and output includes data definition_source: OBI.<br></div></html><hr>
<a name='item480'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_10000015<br></b><b><i>radioactivity_detection</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An assay in which a material's radioactivity is measured.</div></i><font color="#000080">(<i>and</i>&nbsp;assay&nbsp;(has_input&nbsp;<i>some</i>&nbsp;object)<br>&nbsp;(has_output&nbsp;<i>some</i><br>&nbsp;&nbsp;(<i>and</i>&nbsp;data_object&nbsp;(is_proxy_for&nbsp;<i>some</i>&nbsp;radioactive))))</font><br><i>example_of_usage</i>: Measurment of the radioactivity of a supernatant, as in chromium_release_assay.<br><i>definition</i>: An assay in which a material's radioactivity is measured.<br></div></html><hr>
<a name='item481'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600003<br></b><b><i>clinical_diagnosis</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">a protocol application during which a series of tests are made of a patient  leading to determination of disease state, or condition.</div></i><font color="#000080">assay</font><br><i>Source</i>: <br><i>definition_editor</i>: <br><i>editor_note</i>: (maybe CIO)<br><i>branch</i>: protocol_application<br><i>definition_citation</i>: <br><i>definition</i>: a protocol application during which a series of tests are made of a patient  leading to determination of disease state, or condition.<br></div></html><hr>
<a name='item482'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_9999994<br></b><b><i>chromium_release_assay</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An assay in which radioactive chromium is absorbed by cells and released into supernatant when the cells die. The amount of radioactivity measured in the supernatant is a proxy for the number of cells that have died.</div></i><font color="#000080">(<i>and</i>&nbsp;assay&nbsp;(has_part&nbsp;<i>some</i>&nbsp;cell_chromium_labeling)<br>&nbsp;(has_part&nbsp;<i>some</i>&nbsp;cell_co-culture)&nbsp;(has_part&nbsp;<i>some</i>&nbsp;separating_based_on_size)<br>&nbsp;(has_part&nbsp;<i>some</i>&nbsp;radioactivity_detection))</font><br><i>example_of_usage</i>: from http://cvi.asm.org/cgi/content/full/11/2/255 (and slightly simplified)
Autologous EBV-transformed B-LCL were used as target cells for the influenza virus-specific CTL assays. Equal volumes of target (5 x 10E4 cells/ml) and effector cells were added to triplicate wells of 96-well tissue culture plates, and 1:2 serial dilutions of effectors were made, producing effector-to-target (E:T) ratios of 100:1, 50:1, 25:1, and 12.5:1 (9). Spontaneous release of 51Cr was assessed by the incubation of target cells in medium alone, and maximum release of 51Cr was determined by the incubation of target cells in 0.1% Triton X-100 (Sigma-Aldrich).
After a 4-h incubation of the effector cells with the target cells at 37C in 6% CO2, supernatants were collected following brief centrifugation. The supernatant was collected into filters of supernatant collection system harvesting frames (Molecular Devices Skatron, Sunnyvale, Calif.) and transferred to polystyrene tubes to be counted with the LKB 1272 Clinigamma counter (Wallac).
Percent specific was determined with the following equation: (experimental 51Cr release - spontaneous 51Cr release)/(maximum 51Cr release - spontaneous 51Cr release) x 100.<br><i>definition</i>: An assay in which radioactive chromium is absorbed by cells and released into supernatant when the cells die. The amount of radioactivity measured in the supernatant is a proxy for the number of cells that have died.<br></div></html><hr>
<a name='item483'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600002<br></b><b><i>tumor_grading</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Determination of the grade (severity/stage)  of a tumor sample, used in cancer biology to describe abnormalities/qualities of tumor cells or tissues.Values can be described by terms from NCI Thesaurus.</div></i><font color="#000080">assay</font><br><i>Source</i>: Biomaterial_branch<br><i>definition_editor</i>: Compiled by Helen Parkinson for Transcriptomics thanks to Adam Witney<br><i>branch</i>: protocol_application<br><i>definition_citation</i>: MO<br><i>alternative_term</i>: grading_of_tumor<br><i>definition</i>: Determination of the grade (severity/stage)  of a tumor sample, used in cancer biology to describe abnormalities/qualities of tumor cells or tissues.Values can be described by terms from NCI Thesaurus.<br></div></html><hr>
<a name='item484'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_296<br></b><b><i>enrollment</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">process of identifying a set of biomaterials for further use in an investigation</div></i><font color="#000080">process</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: Enrollment of patients in a study.<br><i>definition</i>: process of identifying a set of biomaterials for further use in an investigation<br></div></html><hr>
<a name='item485'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600004<br></b><b><i>Enrolling Human Subjects</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">enrollment of humans based on eligibility criteria for an investigation</div></i><font color="#000080">enrollment</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>editor_note</i>: criteria come from plan / clinical trial branch<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: enlisting familiy members of HIV patients into a study<br><i>definition</i>: enrollment of humans based on eligibility criteria for an investigation<br></div></html><hr>
<a name='item486'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_66<br></b><b><i>investigation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Investigation is a process in which data are gathered from one or more protocol applicationswith the goal of reaching conclusions . It is guided by the realization of a investigation_design</div></i><font color="#000080">process</font><br><i>editor_note</i>: It s up for discussion if a single protocol can be an investigation<br><i>editor_note</i>: Altdef: A set of studies that are designed to address a related question.  It may include individual studies that are domain specific(?) , microarray, proteomics.<br><i>definition</i>: Investigation is a process in which data are gathered from one or more protocol applicationswith the goal of reaching conclusions . It is guided by the realization of a investigation_design<br></div></html><hr>
<a name='item487'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_356<br></b><b><i>hypothesis_generating_investigation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is an investigation in which data is generated and analyzed with the purpose of generating new hypothesis</div></i><font color="#000080">investigation</font><br><i>definition</i>: is an investigation in which data is generated and analyzed with the purpose of generating new hypothesis<br></div></html><hr>
<a name='item488'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_355<br></b><b><i>hypothesis_driven_investigation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is an investigation with the goal to test one or more hypothesis</div></i><font color="#000080">investigation</font><br><i>definition</i>: is an investigation with the goal to test one or more hypothesis<br></div></html><hr>
<a name='item489'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600008<br></b><b><i>Acquisition</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">the process of obtaining an entity from a nameable source.</div></i><font color="#000080">process</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>branch</i>: protocol_application<br><i>definition</i>: the process of obtaining an entity from a nameable source.<br></div></html><hr>
<a name='item490'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600013<br></b><b><i>Data Collection</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">data collection	 is an aquisition of information from an organization that provides information.</div></i><font color="#000080">Acquisition</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: Gathering all influenza HA sequences from GenBank, Retrieveing HLA allele frequencies in the North American populations from dbMHC<br><i>definition</i>: data collection	 is an aquisition of information from an organization that provides information.<br></div></html><hr>
<a name='item491'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600012<br></b><b><i>Environmental Material Collection</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">environmental_material_collection is an acquisition where an object is taken from an environment and put into a storage container. Roles include, environment, thing collected, container, acquirer.</div></i><font color="#000080">Acquisition</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>definition_editor</i>: Alan Ruttenberg<br><i>editor_note</i>: the researcher is very interested in the location<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: Taking 1 liter of surface ocean water from the San Diego Mission Bay Jetty. Capturing mice living in rural Arkansas<br><i>definition</i>: environmental_material_collection is an acquisition where an object is taken from an environment and put into a storage container. Roles include, environment, thing collected, container, acquirer.<br></div></html><hr>
<a name='item492'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600010<br></b><b><i>Object Acquisition</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">an object acquisition is an acquisition of an object from an investigator, institution or vendor. Participant roles include source, destination, carrier, container, transaction record, payer...</div></i><font color="#000080">Acquisition</font><br><i>Source</i>: IEDB<br><i>definition_editor</i>: Bjoern Peters<br><i>definition_editor</i>: Alan Ruttenberg<br><i>editor_note</i>: Bjoern is concerned that information such as model number and serial number are to be associated with the process. Others contend that these are associated, perhaps as labels, to the instances that are the participants.<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: Acquiring 50 C57BL/6 mice in the investigators animal facilities. Purchasing 1 mg of peptides synthesized by Mimotopes at 80% purity. Getting a gift of purified CD4+ specific antibodies presented by Stephen Schoenberger at LIAI.<br><i>definition</i>: an object acquisition is an acquisition of an object from an investigator, institution or vendor. Participant roles include source, destination, carrier, container, transaction record, payer...<br></div></html><hr>
<a name='item493'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_600015<br></b><b><i>Group Assignment</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">assigning objects to groups</div></i><font color="#000080">process</font><br><i>Source</i>: Philippe<br><i>definition_editor</i>: Philippe<br><i>branch</i>: protocol_application<br><i>example_of_usage</i>: splitting an enrolled population into a treated and controlled sub-group; randomized group assignment<br><i>definition</i>: assigning objects to groups<br></div></html><hr>
<a name='item494'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_354<br></b><b><i>disease_course</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is a process through which a disease manifests itself</div></i><font color="#000080">process</font><br><i>definition</i>: is a process through which a disease manifests itself<br></div></html><hr>
<a name='item495'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_177<br></b><b><i>role_assignment</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">BP: remove; this is a conclusion</div></i><font color="#000080">process</font><br><i>definition</i>: A role assignment is a process to designate the function of an entity. definition_source: OBI.<br></div></html><hr>
<a name='item496'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#ProcessBoundary<br></b><b><i>process_boundary</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A processual entity that is the fiat or bona fide instantaneous temporal process boundary.</div></i><font color="#000080">(<i>and</i>&nbsp;processual_entity<br>&nbsp;(<i>disjoint-with</i>&nbsp;fiat_process_part&nbsp;processual_context&nbsp;process&nbsp;process_aggregate))</font><br></div></html><hr>
<a name='item497'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#FiatProcessPart<br></b><b><i>fiat_process_part</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A processual entity that is part of a process but that does not have bona fide beginnings and endings corresponding to real discontinuities.</div></i><font color="#000080">(<i>and</i>&nbsp;processual_entity<br>&nbsp;(<i>disjoint-with</i>&nbsp;process_boundary&nbsp;processual_context&nbsp;process&nbsp;process_aggregate))</font><br></div></html><hr>
<a name='item498'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_72<br></b><b><i>parameter_value_application</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A parameter value application is part of a protocol application in which the values of the parameter are set. definition_source: OBI.</div></i><font color="#000080">fiat_process_part</font><br><i>example_of_usage</i>: Setting the centrifuge to 1200 RPM.<br><i>definition</i>: A parameter value application is part of a protocol application in which the values of the parameter are set. definition_source: OBI.<br></div></html><hr>
<a name='item499'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_278<br></b><b><i>disease_stage</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">is a fiat_process_part which is part_of a disease_process which is associated with position in the normal progression of the disease</div></i><font color="#000080">fiat_process_part</font><br><i>example_of_usage</i>: Stage II breast cancer, The timepoint of recovery from a disease<br><i>definition</i>: is a fiat_process_part which is part_of a disease_process which is associated with position in the normal progression of the disease<br></div></html><hr>
<a name='item500'><html><div style="width:800"><b>Class: http://obi.sourceforge.net/ontology/OBI.owl#OBI_76<br></b><b><i>unplanned_process_part</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An extraneous process is a process which is external in origin to the investigation that has an impact on the outcome. definition_source: OBI.</div></i><font color="#000080">fiat_process_part</font><br><i>example_of_usage</i>: Earthquake that destroys the lab, an outside investigator discovering an adverse effect of the reagants used<br><i>definition</i>: An extraneous process is a process which is external in origin to the investigation that has an impact on the outcome. definition_source: OBI.<br></div></html><hr>
<a name='item501'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#TemporalRegion<br></b><b><i>temporal_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: An occurrent that is part of time.</div></i><font color="#000080">(<i>and</i>&nbsp;occurrent&nbsp;(<i>or</i>&nbsp;connected_temporal_region&nbsp;scattered_temporal_region)<br>&nbsp;(<i>disjoint-with</i>&nbsp;spatiotemporal_region&nbsp;processual_entity))</font><br></div></html><hr>
<a name='item502'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#ScatteredTemporalRegion<br></b><b><i>scattered_temporal_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A temporal region every point of which is not mediately or immediately connected with every other point of which.</div></i><font color="#000080">(<i>and</i>&nbsp;temporal_region&nbsp;temporal_region)</font><br></div></html><hr>
<a name='item503'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#ConnectedTemporalRegion<br></b><b><i>connected_temporal_region</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A temporal region every point of which is mediately or immediately connected with every other point of which.</div></i><font color="#000080">(<i>and</i>&nbsp;(<i>not</i>&nbsp;scattered_temporal_region)<br>&nbsp;(<i>or</i>&nbsp;temporal_instant&nbsp;temporal_interval)&nbsp;temporal_region)</font><br></div></html><hr>
<a name='item504'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#TemporalInstant<br></b><b><i>temporal_instant</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A connected temporal region comprising a single moment of time.</div></i><font color="#000080">(<i>and</i>&nbsp;connected_temporal_region&nbsp;connected_temporal_region)</font><br></div></html><hr>
<a name='item505'><html><div style="width:800"><b>Class: http://www.ifomis.org/bfo/1.0/span#TemporalInterval<br></b><b><i>temporal_interval</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Definition: A connected temporal region lasting for more than a single moment of time.</div></i><font color="#000080">(<i>and</i>&nbsp;connected_temporal_region&nbsp;(<i>not</i>&nbsp;temporal_instant))</font><br></div></html><hr>
<a name='item506'><html><div style="width:800"><b>Class: http://www.geneontology.org/formats/oboInOwl#ObsoleteClass<br></b><i></i><font color="#000080">Thing</font><br></div></html><hr>
<a name='item507'><html><div style="width:800"><b>Class: http://www.geneontology.org/formats/oboInOwl#Synonym<br></b><i></i><font color="#000080">Thing</font><br></div></html><hr>
<h3><i>Property Descriptions</i></h3><a name='item507'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym<br></b><i></i></div></html><hr>
<a name='item508'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/source<br></b><b><i>Source</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A reference to a resource from which the present resource
         is derived.</div></i></div></html><hr>
<a name='item509'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/type<br></b><b><i>Resource Type</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The nature or genre of the content of the resource.</div></i></div></html><hr>
<a name='item510'><html><div style="width:800"><b>Annotation-Property: http://www.w3.org/2000/01/rdf-schema#label<br></b><i></i></div></html><hr>
<a name='item511'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasSynonymType<br></b><i></i></div></html><hr>
<a name='item512'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/title<br></b><b><i>Title</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A name given to the resource.</div></i></div></html><hr>
<a name='item513'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#maxExclusive<br></b><i></i></div></html><hr>
<a name='item514'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#defaultLanguage<br></b><i></i></div></html><hr>
<a name='item515'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/publisher<br></b><b><i>Publisher</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An entity responsible for making the resource available</div></i></div></html><hr>
<a name='item516'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasDefaultNamespace<br></b><i></i></div></html><hr>
<a name='item517'><html><div style="width:800"><b>Annotation-Property: http://www.w3.org/2000/01/rdf-schema#isDefinedBy<br></b><i></i></div></html><hr>
<a name='item518'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#minLength<br></b><i></i></div></html><hr>
<a name='item519'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#minInclusive<br></b><i></i></div></html><hr>
<a name='item520'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#subclassesDisjoint<br></b><i></i></div></html><hr>
<a name='item521'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/format<br></b><b><i>Format</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The physical or digital manifestation of the resource.</div></i></div></html><hr>
<a name='item522'><html><div style="width:800"><b>Annotation-Property: http://www.w3.org/2002/07/owl#versionInfo<br></b><i></i></div></html><hr>
<a name='item523'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasURI<br></b><i></i></div></html><hr>
<a name='item524'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#totalDigits<br></b><i></i></div></html><hr>
<a name='item525'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#allowedParent<br></b><i></i></div></html><hr>
<a name='item526'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#maxInclusive<br></b><i></i></div></html><hr>
<a name='item527'><html><div style="width:800"><b>Annotation-Property: http://www.w3.org/2000/01/rdf-schema#seeAlso<br></b><i></i></div></html><hr>
<a name='item528'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/identifier<br></b><b><i>Resource Identifier</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An unambiguous reference to the resource within a given context.</div></i></div></html><hr>
<a name='item529'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#readOnly<br></b><i></i></div></html><hr>
<a name='item530'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasAlternativeId<br></b><i></i></div></html><hr>
<a name='item531'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#alternative_term<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An alternative name for a class or property which means the same thing as the preferred_term (semantically equivalent)</div></i></div></html><hr>
<a name='item532'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#definition_citation<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">formal citation, e.g. identifier in external database to indicate / attribute source(s) for the definition. Free text indicate / attribute source(s) for the definition. EXAMPLE: Author Name, URI, MeSH Term C04, PUBMED ID, Wiki uri on 31.01.2007</div></i></div></html><hr>
<a name='item533'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#preferred_term<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The concise, meaningful, and human-friendly name for a class or property preferred by the ontology developers. (US-English)</div></i></div></html><hr>
<a name='item534'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#minExclusive<br></b><i></i></div></html><hr>
<a name='item535'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym<br></b><i></i></div></html><hr>
<a name='item536'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#usedLanguage<br></b><i></i></div></html><hr>
<a name='item537'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/language<br></b><b><i>Language</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A language of the intellectual content of the resource.</div></i></div></html><hr>
<a name='item538'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#branch<br></b><i></i></div></html><hr>
<a name='item539'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/subject<br></b><b><i>Subject and Keywords</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The topic of the content of the resource.</div></i></div></html><hr>
<a name='item540'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#abstract<br></b><i></i></div></html><hr>
<a name='item541'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#probeClass<br></b><i></i></div></html><hr>
<a name='item542'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#fractionDigits<br></b><i></i></div></html><hr>
<a name='item543'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#todoPrefix<br></b><i></i></div></html><hr>
<a name='item544'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasDate<br></b><i></i></div></html><hr>
<a name='item545'><html><div style="width:800"><b>Annotation-Property: http://www.w3.org/2000/01/rdf-schema#comment<br></b><i></i></div></html><hr>
<a name='item546'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/date<br></b><b><i>Date</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A date associated with an event in the life cycle of the
         resource.</div></i></div></html><hr>
<a name='item547'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#maxLength<br></b><i></i></div></html><hr>
<a name='item548'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/coverage<br></b><b><i>Coverage</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The extent or scope of the content of the resource.</div></i></div></html><hr>
<a name='item549'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/relation<br></b><b><i>Relation</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A reference to a related resource.</div></i></div></html><hr>
<a name='item550'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasDbXref<br></b><i></i></div></html><hr>
<a name='item551'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#replacedBy<br></b><i></i></div></html><hr>
<a name='item552'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#savedBy<br></b><i></i></div></html><hr>
<a name='item553'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#pattern<br></b><i></i></div></html><hr>
<a name='item554'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#example_of_usage<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An actual usage of the term.</div></i></div></html><hr>
<a name='item555'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#excludedTest<br></b><i></i></div></html><hr>
<a name='item556'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#inSubset<br></b><i></i></div></html><hr>
<a name='item557'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#length<br></b><i></i></div></html><hr>
<a name='item558'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#definition<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The official OBI definition, explaining the meaning of a class or property. Shall be Aristotelian, formalized and normalized. Can be augmented with colloquial definitions.</div></i></div></html><hr>
<a name='item559'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#editor_note<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An administrative note intended for its editor. It will not be included in the publication version of the ontology, so it should contain nothing necessary for end users to understand the ontology.</div></i></div></html><hr>
<a name='item560'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#alternative_term_citation<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">ormal citation of the source of the alternative_definition, e.g. identifier in external database to indicate / attribute source(s) for the definition. Free text indicate / attribute source(s) for the definition. EXAMPLE: Author Name, URI, MeSH Term C04, PUBMED ID, Wiki uri on 31.01.2007</div></i></div></html><hr>
<a name='item561'><html><div style="width:800"><b>Annotation-Property: http://protege.stanford.edu/plugins/owl/protege#todoProperty<br></b><i></i></div></html><hr>
<a name='item562'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasOBONamespace<br></b><i></i></div></html><hr>
<a name='item563'><html><div style="width:800"><b>Annotation-Property: http://www.owl-ontologies.com/2005/08/07/xsp.owl#base<br></b><i></i></div></html><hr>
<a name='item564'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasDefinition<br></b><i></i></div></html><hr>
<a name='item565'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/description<br></b><b><i>Description</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An account of the content of the resource.</div></i></div></html><hr>
<a name='item566'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/contributor<br></b><b><i>Contributor</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An entity responsible for making contributions to the
         content of the resource.</div></i></div></html><hr>
<a name='item567'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#curation_status<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">The curation status of the term. The allowed values come from an enumerated list of predefined terms. Examples: raw import, definition incomplete, graph position temporary, uncurated, pending final vetting, curation complete. The value raw import indicates that the term is derived directly via import of an entity from an existing knowledge resource. The value definition incomplete indicates that the curators recognize the definition of the class remains incomplete in someway. The value graph position temporary indicates that the class may not be in its final preferred location within the ontology graph.  The value uncurated indicates that the class has been created by curators but has received minimal review.  The value pending final vetting indicates that indicates the term is likely ready for promotion to "curation complete" status, and only requires a final review by  the curators responsible.  The value curation complete indicates that the class has been fully vetted by the curation authorities and to the best of their ability determined to be ready for use.</div></i></div></html><hr>
<a name='item568'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#consider<br></b><i></i></div></html><hr>
<a name='item569'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/creator<br></b><b><i>Creator</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An entity primarily responsible for making the content 
         of the resource.</div></i></div></html><hr>
<a name='item570'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasExactSynonym<br></b><i></i></div></html><hr>
<a name='item571'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasVersion<br></b><i></i></div></html><hr>
<a name='item572'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym<br></b><i></i></div></html><hr>
<a name='item573'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#alternative_term_tag<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">tag to indicate sets of alternative terms Example: toxicogenomics_community, abbreviation</div></i></div></html><hr>
<a name='item574'><html><div style="width:800"><b>Annotation-Property: http://purl.org/dc/elements/1.1/rights<br></b><b><i>Rights Management</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Information about rights held in and over the resource.</div></i></div></html><hr>
<a name='item575'><html><div style="width:800"><b>Annotation-Property: http://www.geneontology.org/formats/oboInOwl#hasSubset<br></b><i></i></div></html><hr>
<a name='item576'><html><div style="width:800"><b>Annotation-Property: http://obi.sourceforge.net/ontology/OBI.owl#definition_editor<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Name of editor entering the definition</div></i></div></html><hr>
<a name='item577'><html><div style="width:800"><b>Datatype-Property: http://protege.stanford.edu/plugins/owl/protege#PAL-NAME<br></b><i></i><b>Domain</b>: PAL-CONSTRAINT<br><b>Range</b>: string<br></div></html><hr>
<a name='item578'><html><div style="width:800"><b>Datatype-Property: http://protege.stanford.edu/plugins/owl/protege#PAL-STATEMENT<br></b><i></i><b>Domain</b>: PAL-CONSTRAINT<br><b>Range</b>: string<br></div></html><hr>
<a name='item579'><html><div style="width:800"><b>Datatype-Property: http://protege.stanford.edu/plugins/owl/protege#PAL-DESCRIPTION<br></b><i></i><b>Domain</b>: PAL-CONSTRAINT<br><b>Range</b>: string<br></div></html><hr>
<a name='item580'><html><div style="width:800"><b>Datatype-Property: http://protege.stanford.edu/plugins/owl/protege#PAL-RANGE<br></b><i></i><b>Domain</b>: PAL-CONSTRAINT<br><b>Range</b>: string<br></div></html><hr>
<a name='item581'><html><div style="width:800"><b>Object-Property: http://protege.stanford.edu/plugins/owl/protege#FROM<br></b><i></i><b>Domain</b>: DIRECTED-BINARY-RELATION<br><b>Range</b>: Thing<br></div></html><hr>
<a name='item582'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#is_realized_as<br></b><i></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: is_realization_of<br><b>Domain</b>: realizable_entity<br><b>Range</b>: process<br></div></html><hr>
<a name='item583'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#relationship<br></b><b><i>relationship</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A relationship between two classes (terms). Relationships between classes are expressed in terms of relations on underlying instances.</div></i><b>Subproperties</b>: part_of, location_of, outputs_data, participates_in, located_in, has_participant, contained_in, adjacent_to, precedes, derived_into, contains, has_part, transformation_of, inputs_data, derives_from, preceded_by<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7ff4<br></div></html><hr>
<a name='item584'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#location_of<br></b><b><i>location_of</i></b><i><br></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: located_in<br><b>Subproperty of</b>: relationship<br><b>Domain</b>: continuant<br><b>Range</b>: continuant<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item585'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#participates_in<br></b><b><i>participates_in</i></b><i><br></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: has_participant<br><b>Subproperty of</b>: relationship<br><b>Subproperties</b>: agent_in, is_input_of, is_output_of, is_reagent_in, is_instrument_for<br><b>Domain</b>: continuant<br><b>Range</b>: occurrent<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item586'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#is_input_of<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Relationship between a participant and a protocol application. The input is, at a minimum, present at the beginning of the protocol application, but not all participants present at the beginning of the protocol applications are input,  only those that are viewed as inputs according to the plan, and hence realize  InputRole.</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: has_input<br><b>Subproperty of</b>: participates_in<br><b>Domain</b>: continuant<br><b>Range</b>: (<i>and</i>&nbsp;occurrent&nbsp;protocol_application)<br><i>definition_editor</i>: Alan Ruttenberg<br><i>example_of_usage</i>: some Autologous EBV(Epstein-Barr virus)-transformed B-LCL (B lymphocyte cell line) is_input_for instance of Chromum Release Assay described at https://wiki.cbil.upenn.edu/obiwiki/index.php/Chromium_Release_assay<br><i>definition</i>: Relationship between a participant and a protocol application. The input is, at a minimum, present at the beginning of the protocol application, but not all participants present at the beginning of the protocol applications are input,  only those that are viewed as inputs according to the plan, and hence realize  InputRole.<br></div></html><hr>
<a name='item587'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#is_reagent_in<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Relationship between a substance and a protocol application in which it participates, where the substance has a ReagentRole</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: utilizes_reagent<br><b>Subproperty of</b>: participates_in<br><b>Domain</b>: (<i>and</i>&nbsp;continuant&nbsp;independent_continuant)<br><b>Range</b>: (<i>and</i>&nbsp;occurrent&nbsp;protocol_application)<br><i>definition_editor</i>: Alan Ruttenberg<br><i>example_of_usage</i>: some Triton X-100 (Sigma-Aldrich) is_reagent_in instance of Chromium Release Assay described at  https://wiki.cbil.upenn.edu/obiwiki/index.php/Chromium_Release_assay<br><i>definition</i>: Relationship between a substance and a protocol application in which it participates, where the substance has a ReagentRole<br></div></html><hr>
<a name='item588'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#is_output_of<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Relationship between a participant and a protocol application. The output is, at a minimum, present at the end of the protocol application, but not all participants present at the end of the protocol applications are input,  only those that are viewed as outputs according to the plan, and hence realize OutputRole.</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: has_output<br><b>Subproperty of</b>: participates_in<br><b>Domain</b>: continuant<br><b>Range</b>: (<i>and</i>&nbsp;occurrent&nbsp;protocol_application)<br><i>definition_editor</i>: Alan Ruttenberg<br><i>definition</i>: Relationship between a participant and a protocol application. The output is, at a minimum, present at the end of the protocol application, but not all participants present at the end of the protocol applications are input,  only those that are viewed as outputs according to the plan, and hence realize OutputRole.<br></div></html><hr>
<a name='item589'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#agent_in<br></b><b><i>agent_in</i></b><i><br></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: has_agent<br><b>Subproperty of</b>: participates_in<br><b>Domain</b>: continuant<br><b>Range</b>: occurrent<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item590'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#is_instrument_for<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Relationship between a instrument and a protocol application in which it  participates</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: utilizes_instrument<br><b>Subproperty of</b>: participates_in<br><b>Domain</b>: (<i>and</i>&nbsp;instrument&nbsp;continuant)<br><b>Range</b>: (<i>and</i>&nbsp;occurrent&nbsp;protocol_application)<br><i>definition_editor</i>: Alan Ruttenberg<br><i>example_of_usage</i>: some LKB 1272 Clinigamma counter is_instrument_for instance of Chromium Release Assay described at https://wiki.cbil.upenn.edu/obiwiki/index.php/Chromium_Release_assay<br><i>definition</i>: Relationship between a instrument and a protocol application in which it  participates<br></div></html><hr>
<a name='item591'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#derives_from<br></b><b><i>derives_from</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Derivation as a relation between instances. The temporal relation of derivation is more complex. Transformation, on the instance level, is just the relation of identity: each adult is identical to some child existing at some earlier time. Derivation on the instance-level is a relation holding between non-identicals. More precisely, it holds between distinct material continuants when one succeeds the other across a temporal divide in such a way that at least a biologically significant portion of the matter of the earlier continuant is inherited by the later. Thus we will have axioms to the effect that from c derives_from c1 we can infer that c and c1 are not identical and that there is some instant of time t such that c1 exists only prior to and c only subsequent to t. We will also be able to infer that the spatial region occupied by c as it begins to exist at t overlaps with the spatial region occupied by c1 as it ceases to exist in the same instant.</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: derived_into<br><b>Subproperty of</b>: relationship<br><b>Domain</b>: continuant<br><b>Range</b>: continuant<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7fe5<br></div></html><hr>
<a name='item592'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#part_of<br></b><b><i>part_of</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Parthood as a relation between instances: The primitive instance-level relation p part_of p1 is illustrated in assertions such as: this instance of rhodopsin mediated phototransduction part_of this instance of visual perception.    This relation satisfies at least the following standard axioms of mereology: reflexivity (for all p, p part_of p); anti-symmetry (for all p, p1, if p part_of p1 and p1 part_of p then p and p1 are identical); and transitivity (for all p, p1, p2, if p part_of p1 and p1 part_of p2, then p part_of p2). Analogous axioms hold also for parthood as a relation between spatial regions.    For parthood as a relation between continuants, these axioms need to be modified to take account of the incorporation of a temporal argument. Thus for example the axiom of transitivity for continuants will assert that if c part_of c1 at t and c1 part_of c2 at t, then also c part_of c2 at t.    Parthood as a relation between classes: To define part_of as a relation between classes we again need to distinguish the two cases of continuants and processes, even though the explicit reference to instants of time now falls away. For continuants, we have C part_of C1 if and only if any instance of C at any time is an instance-level part of some instance of C1 at that time, as for example in: cell nucleus part_ of cell.</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: has_part<br><b>Subproperty of</b>: relationship<br><b>Subproperties</b>: integral_part_of, proper_part_of<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7ff3<br></div></html><hr>
<a name='item593'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#proper_part_of<br></b><b><i>proper_part_of</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">As for part_of, with the additional constraint that subject and object are distinct</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: has_proper_part<br><b>Subproperty of</b>: part_of<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7fef<br></div></html><hr>
<a name='item594'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#integral_part_of<br></b><b><i>integral_part_of</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">C integral_part_of C' if and only if: C part_of C' AND C' has_part C</div></i><b><font color="#000080">Transitive</font></b><br><b>Subproperty of</b>: part_of<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7ff1<br></div></html><hr>
<a name='item595'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#has_participant<br></b><b><i>has_participant</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Has_participant is a primitive instance-level relation between a process, a continuant, and a time at which the continuant participates in some way in the process. The relation obtains, for example, when this particular process of oxygen exchange across this particular alveolar membrane has_participant this particular sample of hemoglobin at this particular time.</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: participates_in<br><b>Subproperty of</b>: relationship<br><b>Subproperties</b>: utilizes_reagent, has_agent, has_output, utilizes_instrument, has_input<br><b>Domain</b>: occurrent<br><b>Range</b>: continuant<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7fe1<br></div></html><hr>
<a name='item596'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#has_output<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Relationship between a  protocol application and a participant in the protocol application. The output is, at a minimum, present at the end of the protocol application, but not all participants present at the end of the protocol applications are input,  only those that are viewed as outputs according to the plan, and hence realize OutputRole.</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: is_output_of<br><b>Subproperty of</b>: has_participant<br><b>Domain</b>: (<i>and</i>&nbsp;occurrent&nbsp;protocol_application)<br><b>Range</b>: continuant<br><i>definition_editor</i>: Alan Ruttenberg<br><i>definition</i>: Relationship between a  protocol application and a participant in the protocol application. The output is, at a minimum, present at the end of the protocol application, but not all participants present at the end of the protocol applications are input,  only those that are viewed as outputs according to the plan, and hence realize OutputRole.<br></div></html><hr>
<a name='item597'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#has_input<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Relationship between a protocol application and a participant in that protocol application. The input is, at a minimum, present at the beginning of the protocol application, but not all participants present at the beginning of the protocol applications are input,  only those that are viewed as inputs according to the plan, and hence realize  InputRole.</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: is_input_of<br><b>Subproperty of</b>: has_participant<br><b>Domain</b>: (<i>and</i>&nbsp;occurrent&nbsp;protocol_application)<br><b>Range</b>: continuant<br><i>definition_editor</i>: Alan Ruttenberg<br><i>example_of_usage</i>: see is_input_of example_of_usage<br><i>definition</i>: Relationship between a protocol application and a participant in that protocol application. The input is, at a minimum, present at the beginning of the protocol application, but not all participants present at the beginning of the protocol applications are input,  only those that are viewed as inputs according to the plan, and hence realize  InputRole.<br></div></html><hr>
<a name='item598'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#utilizes_reagent<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Relationship between a protocol application and a substance that has role reagent that participates in the protocol application</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: is_reagent_in<br><b>Subproperty of</b>: has_participant<br><b>Domain</b>: (<i>and</i>&nbsp;occurrent&nbsp;protocol_application)<br><b>Range</b>: (<i>and</i>&nbsp;continuant&nbsp;independent_continuant)<br><i>definition_editor</i>: Alan Ruttenberg<br><i>example_of_usage</i>: see example_of_usage for is_reagent_in<br><i>definition</i>: Relationship between a protocol application and a substance that has role reagent that participates in the protocol application<br></div></html><hr>
<a name='item599'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#has_agent<br></b><b><i>has_agent</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">As for has_participant, but with the additional condition that the component instance is causally active in the relevant process</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: agent_in<br><b>Subproperty of</b>: has_participant<br><b>Domain</b>: occurrent<br><b>Range</b>: continuant<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7fdf<br></div></html><hr>
<a name='item600'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#utilizes_instrument<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Relationship between protocol application and an intrument in which it  participates</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: is_instrument_for<br><b>Subproperty of</b>: has_participant<br><b>Domain</b>: (<i>and</i>&nbsp;occurrent&nbsp;protocol_application)<br><b>Range</b>: (<i>and</i>&nbsp;instrument&nbsp;continuant)<br><i>definition_editor</i>: Alan Ruttenberg<br><i>example_of_usage</i>: see example_of_usage for is_instrument_for<br><i>definition</i>: Relationship between protocol application and an intrument in which it  participates<br></div></html><hr>
<a name='item601'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#derived_into<br></b><b><i>derived_into</i></b><i><br></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: derives_from<br><b>Subproperty of</b>: relationship<br><b>Domain</b>: continuant<br><b>Range</b>: continuant<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item602'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#preceded_by<br></b><b><i>preceded_by</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">An example is: translation preceded_by transcription; aging preceded_by development (not however death preceded_by aging). Where derives_from links classes of continuants, preceded_by links classes of processes. Clearly, however, these two relations are not independent of each other. Thus if cells of type C1 derive_from cells of type C, then any cell division involving an instance of C1 in a given lineage is preceded_by cellular processes involving an instance of C.    The assertion P preceded_by P1 tells us something about Ps in general: that is, it tells us something about what happened earlier, given what we know about what happened later. Thus it does not provide information pointing in the opposite direction, concerning instances of P1 in general; that is, that each is such as to be succeeded by some instance of P. Note that an assertion to the effect that P preceded_by P1 is rather weak; it tells us little about the relations between the underlying instances in virtue of which the preceded_by relation obtains. Typically we will be interested in stronger relations, for example in the relation immediately_preceded_by, or in relations which combine preceded_by with a condition to the effect that the corresponding instances of P and P1 share participants, or that their participants are connected by relations of derivation, or (as a first step along the road to a treatment of causality) that the one process in some way affects (for example, initiates or regulates) the other.</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: precedes<br><b>Subproperty of</b>: relationship<br><b>Domain</b>: occurrent<br><b>Range</b>: occurrent<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7fe3<br></div></html><hr>
<a name='item603'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#inputs_data<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Within a protocol application which has instrument and data as participants, the data is being used by the instrument.</div></i><b>Subproperty of</b>: relationship<br><b>Domain</b>: instrument<br><b>Range</b>: data_object<br><i>definition_editor</i>: Alan Ruttenberg<br><i>editor_note</i>: See note for outputs_data<br><i>definition</i>: Within a protocol application which has instrument and data as participants, the data is being used by the instrument.<br></div></html><hr>
<a name='item604'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#outputs_data<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Within a protocol application which has instrument and data as participants, the data is being generated by the instrument.</div></i><b>Subproperty of</b>: relationship<br><b>Domain</b>: instrument<br><b>Range</b>: data_object<br><i>definition_editor</i>: Alan Ruttenberg<br><i>editor_note</i>: Ask Instrument branch: Is the data output always an output of the protocol application? Counterexample: Intermediate signal cleaner, where only the cleaned signal is an output of the protocol application.
Coordinate with DataTransformation. Same question for data_input.<br><i>definition</i>: Within a protocol application which has instrument and data as participants, the data is being generated by the instrument.<br></div></html><hr>
<a name='item605'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#has_part<br></b><b><i>has_part</i></b><i><br></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: part_of<br><b>Subproperty of</b>: relationship<br><b>Subproperties</b>: has_integral_part, has_proper_part<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item606'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#has_proper_part<br></b><b><i>has_proper_part</i></b><i><br></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: proper_part_of<br><b>Subproperty of</b>: has_part<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item607'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#has_integral_part<br></b><b><i>has_integral_part</i></b><i><br></i><b><font color="#000080">Transitive</font></b><br><b>Subproperty of</b>: has_part<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item608'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#contains<br></b><b><i>contains</i></b><i><br></i><b>Inverse of</b>: contained_in<br><b>Subproperty of</b>: relationship<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item609'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#transformation_of<br></b><b><i>transformation_of</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">When an embryonic oenocyte (a type of insect cell) is transformed into a larval oenocyte, one and the same continuant entity preserves its identity while instantiating distinct classes at distinct times. The class-level relation transformation_of obtains between continuant classes C and C1 wherever each instance of the class C is such as to have existed at some earlier time as an instance of the distinct class C1 (see Figure 2 in paper). This relation is illustrated first of all at the molecular level of granularity by the relation between mature RNA and the pre-RNA from which it is processed, or between (UV-induced) thymine-dimer and thymine dinucleotide. At coarser levels of granularity it is illustrated by the transformations involved in the creation of red blood cells, for example, from reticulocyte to erythrocyte, and by processes of development, for example, from larva to pupa, or from (post-gastrular) embryo to fetus or from child to adult. It is also manifest in pathological transformations, for example, of normal colon into carcinomatous colon. In each such case, one and the same continuant entity instantiates distinct classes at different times in virtue of phenotypic changes.</div></i><b><font color="#000080">Transitive</font></b><br><b>Subproperty of</b>: relationship<br><b>Domain</b>: continuant<br><b>Range</b>: continuant<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7fe7<br></div></html><hr>
<a name='item610'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#located_in<br></b><b><i>located_in</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Location as a relation between instances: The primitive instance-level relation c located_in r at t reflects the fact that each continuant is at any given time associated with exactly one spatial region, namely its exact location. Following we can use this relation to define a further instance-level location relation - not between a continuant and the region which it exactly occupies, but rather between one continuant and another. c is located in c1, in this sense, whenever the spatial region occupied by c is part_of the spatial region occupied by c1.    Note that this relation comprehends both the relation of exact location between one continuant and another which obtains when r and r1 are identical (for example, when a portion of fluid exactly fills a cavity), as well as those sorts of inexact location relations which obtain, for example, between brain and head or between ovum and uterus</div></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: location_of<br><b>Subproperty of</b>: relationship<br><b>Domain</b>: continuant<br><b>Range</b>: continuant<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7fed<br></div></html><hr>
<a name='item611'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#contained_in<br></b><b><i>contained_in</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Containment obtains in each case between material and immaterial continuants, for instance: lung contained_in thoracic cavity; bladder contained_in pelvic cavity. Hence containment is not a transitive relation.    If c part_of c1 at t then we have also, by our definition and by the axioms of mereology applied to spatial regions, c located_in c1 at t. Thus, many examples of instance-level location relations for continuants are in fact cases of instance-level parthood. For material continuants location and parthood coincide. Containment is location not involving parthood, and arises only where some immaterial continuant is involved. To understand this relation, we first define overlap for continuants as follows:    c1 overlap c2 at t =def for some c, c part_of c1 at t and c part_of c2 at t.    The containment relation on the instance level can then be defined (see definition):</div></i><b>Inverse of</b>: contains<br><b>Subproperty of</b>: relationship<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7feb<br></div></html><hr>
<a name='item612'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#adjacent_to<br></b><b><i>adjacent_to</i></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">Note that adjacent_to as thus defined is not a symmetric relation, in contrast to its instance-level counterpart. For it can be the case that Cs are in general such as to be adjacent to instances of C1 while no analogous statement holds for C1s in general in relation to instances of C. Examples are: nuclear membrane adjacent_to cytoplasm; seminal vesicle adjacent_to urinary bladder; ovary adjacent_to parietal pelvic peritoneum</div></i><b>Subproperty of</b>: relationship<br><b>Domain</b>: Thing<br><b>Range</b>: Thing<br><i>hasOBONamespace</i>: relationship<br><i>hasDefinition</i>: Blank-7fe9<br></div></html><hr>
<a name='item613'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#precedes<br></b><b><i>precedes</i></b><i><br></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: preceded_by<br><b>Subproperty of</b>: relationship<br><b>Domain</b>: occurrent<br><b>Range</b>: occurrent<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item614'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#is_realization_of<br></b><i></i><b><font color="#000080">Transitive</font></b><br><b>Inverse of</b>: is_realized_as<br><b>Domain</b>: process<br><b>Range</b>: realizable_entity<br></div></html><hr>
<a name='item615'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#is_concretization_of<br></b><i></i><b>Inverse of</b>: is_concretized_as<br><b>Domain</b>: concretized_information_entity<br><b>Range</b>: information_entity<br></div></html><hr>
<a name='item616'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#is_proxy_for<br></b><i><div style="margin-left:5px;margin-top:2px;margin-bottom:2px">A relation between continuant instances c1 and c2 where within an experiment/ protocol application,  measurement of c1 is used to determine what a measurement of c2 would be.</div></i><b>Domain</b>: continuant<br><b>Range</b>: continuant<br><i>definition_editor</i>: Alan Ruttenberg<br><i>example_of_usage</i>: position on a gel is_proxy_for mass and charge of molecule in an western blot. Florescent intensity is_proxy_for amount of protein labeled with GFP. Examples:
A260/A280 (of a DNA sample) is_proxy_for DNA-purity. NMR Sample scan is a proxy for sample quality.

Within the assay mentioned here: https://wiki.cbil.upenn.edu/obiwiki/index.php/Chromium_Release_assay
level of radioactivity is_proxy_for level of toxicity<br><i>definition</i>: A relation between continuant instances c1 and c2 where within an experiment/ protocol application,  measurement of c1 is used to determine what a measurement of c2 would be.<br><i>definition</i>: A relation between continuant instances c1 and c2 where within a protocol 
    application, measurement of c1 is related to a what would be the 
    measurement of c2.<br></div></html><hr>
<a name='item617'><html><div style="width:800"><b>Object-Property: http://www.obofoundry.org/ro/ro.owl#transformed_into<br></b><i></i><b>Domain</b>: continuant<br><b>Range</b>: continuant<br><i>hasOBONamespace</i>: relationship<br></div></html><hr>
<a name='item618'><html><div style="width:800"><b>Object-Property: http://protege.stanford.edu/plugins/owl/protege#SLOT-CONSTRAINTS<br></b><i></i><b>Domain</b>: Thing<br><b>Range</b>: Thing<br></div></html><hr>
<a name='item619'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#is_concretized_as<br></b><i></i><b>Inverse of</b>: is_concretization_of<br><b>Domain</b>: information_entity<br><b>Range</b>: concretized_information_entity<br></div></html><hr>
<a name='item620'><html><div style="width:800"><b>Object-Property: http://www.geneontology.org/formats/oboInOwl#ObsoleteProperty<br></b><i></i><b>Domain</b>: Thing<br><b>Range</b>: Thing<br></div></html><hr>
<a name='item621'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#has_role<br></b><i></i><b><font color="#000080">Transitive</font></b><br><b>Domain</b>: independent_continuant<br><b>Range</b>: role<br></div></html><hr>
<a name='item622'><html><div style="width:800"><b>Object-Property: http://protege.stanford.edu/plugins/owl/protege#TO<br></b><i></i><b>Domain</b>: DIRECTED-BINARY-RELATION<br><b>Range</b>: Thing<br></div></html><hr>
<a name='item623'><html><div style="width:800"><b>Object-Property: http://obi.sourceforge.net/ontology/OBI.owl#has_quality<br></b><i></i><b><font color="#000080">Transitive</font></b><br><b>Domain</b>: independent_continuant<br><b>Range</b>: quality<br></div></html><hr>
</html> 